#/usr/bin/perl -w
use strict;
#modify to take observed variation into account....
#this took output generated by addBrackets_Alleles.pl, addBrackets_Alleles.pl used SNPs from the catalog but positions from observed haplotypes...
my$tag;
my$minProbeLength=100;
while(my$line=<>){
	chomp $line;
	if($line=~/^>/){
		$tag=$line;
	}else{
		my$SNPcount=()=$line=~/\[/g;
		#print "$tag\t$SNPcount\t$line\n";
		my@alleles=$line=~/\[\w+]/g;
		#print "@alleles\n";
		my@segments=split '\[\w+]', $line;
		#foreach my$i (@segments){
		#	print "$i\n";
		#}
		foreach my$i (0..$#alleles){
			my$bases=$alleles[$i];
			$bases=~s/\[//;
			$bases=~s/]//;
			#print "$bases\n";
			my@nucleotides=split "", $bases;
			#foreach my$j (0..$#nucleotides){
			#	print "$nucleotides[$j]\t";
			#}
			#print "\n";
			my$leftseq=$segments[$i];
			my$j=$i+1;
			my$rightseq=$segments[$j];
			my$leftlength=length$leftseq;
			my$rightlength=length$rightseq;
			my$leftprobe;
			if($leftlength<5){
				$leftprobe=$leftseq;
			}else{
				$leftprobe=substr($leftseq,-5,5);
			}
			my$rightprobe;
			if($rightlength<5){
				$rightprobe=$rightseq;
			}else{
				$rightprobe=substr($rightseq,0,5);
			}
			my@probes;
			
			foreach my$k (0..$#nucleotides){
				push@probes,$leftprobe.$nucleotides[$k].$rightprobe;
			}
			foreach my$k (0..$#probes){
				if(length$probes[$k]<$minProbeLength){
					$minProbeLength=length$probes[$k];
				}
				my$probeNum=$k+1;
				#print "SNP $i probe$probeNum: $probes[$k]\n";
			}
			my$ID=$tag."_SNP".$j;
			$ID=~s/^>//;
			print "$ID";
			foreach my$l (0..1){
				print "\t$probes[$l]";
			}
			print "\n";
		}
	}
}

#print "minimum probe length: $minProbeLength\n";