{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook details the bioinformatics associated with optimization run 1 of the AC SNP panel, 10/9/2017"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1\n"
     ]
    }
   ],
   "source": [
    "cd /mnt/hgfs/Ubuntu_Shared/OptimizationRun1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!stacks process_radtags \\\n",
    "-o Process_RadTags_Output \\\n",
    "-P \\\n",
    "-p Optimization_Run1_Data \\\n",
    "-i gzfastq \\\n",
    "-y fastq \\\n",
    "--adapter_1 TCTACACGTTCAGAGTTCTACAGTCCGACGATC \\\n",
    "--adapter_2 GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT \\\n",
    "-r \\\n",
    "-e 'sbfI' \\\n",
    "-c \\\n",
    "-q \\\n",
    "-t 70 \\\n",
    "-E phred33 \\\n",
    "--filter_illumina \\\n",
    "2> Process_RadTags_Output/Opt1_output.txt\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "jk, not using stacks at all- going to use Garrett's custom scripts"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "to count unique lines, perl script from Garrett:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1\n"
     ]
    }
   ],
   "source": [
    "cd /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl hashFastq.pl Optimization_Run1_Data/2010AC19105_S24_L001_R1_001.fastq > uniq_seq_count_hash_output_2010AC19105FWD.txt\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "looks like 200,000+ sequences, 45,000+ of which contain the SBF1 cutsite, TGCAGG, but not all of which are at the beginning. looks like unique sequences to me\n",
    "\n",
    "Going to just use the demultiplexed data from the miseq and ignore the undetermined folders at least for now\n",
    "\n",
    "so to count unique lines in each file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "ls data/Unzipped_Data/ > data/Unzipped_Data/filenames.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19003_S9_L001_R1_001.fastq > Data/Hash_Results/2010AC19003_S9_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19003_S9_L001_R2_001.fastq > Data/Hash_Results/2010AC19003_S9_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19006_S10_L001_R1_001.fastq > Data/Hash_Results/2010AC19006_S10_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19006_S10_L001_R2_001.fastq > Data/Hash_Results/2010AC19006_S10_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19008_S11_L001_R1_001.fastq > Data/Hash_Results/2010AC19008_S11_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19008_S11_L001_R2_001.fastq > Data/Hash_Results/2010AC19008_S11_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19015_S12_L001_R1_001.fastq > Data/Hash_Results/2010AC19015_S12_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19015_S12_L001_R2_001.fastq > Data/Hash_Results/2010AC19015_S12_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19041_S17_L001_R1_001.fastq > Data/Hash_Results/2010AC19041_S17_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19041_S17_L001_R2_001.fastq > Data/Hash_Results/2010AC19041_S17_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19047_S13_L001_R1_001.fastq > Data/Hash_Results/2010AC19047_S13_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19047_S13_L001_R2_001.fastq > Data/Hash_Results/2010AC19047_S13_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19049_S14_L001_R1_001.fastq > Data/Hash_Results/2010AC19049_S14_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19049_S14_L001_R2_001.fastq > Data/Hash_Results/2010AC19049_S14_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19050_S15_L001_R1_001.fastq > Data/Hash_Results/2010AC19050_S15_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19050_S15_L001_R2_001.fastq > Data/Hash_Results/2010AC19050_S15_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19057_S16_L001_R1_001.fastq > Data/Hash_Results/2010AC19057_S16_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19057_S16_L001_R2_001.fastq > Data/Hash_Results/2010AC19057_S16_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19091_S18_L001_R1_001.fastq > Data/Hash_Results/2010AC19091_S18_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19091_S18_L001_R2_001.fastq > Data/Hash_Results/2010AC19091_S18_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19096_S21_L001_R1_001.fastq > Data/Hash_Results/2010AC19096_S21_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19096_S21_L001_R2_001.fastq > Data/Hash_Results/2010AC19096_S21_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19098_S22_L001_R1_001.fastq > Data/Hash_Results/2010AC19098_S22_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19098_S22_L001_R2_001.fastq > Data/Hash_Results/2010AC19098_S22_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19099_S19_L001_R1_001.fastq > Data/Hash_Results/2010AC19099_S19_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19099_S19_L001_R2_001.fastq > Data/Hash_Results/2010AC19099_S19_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19101_S23_L001_R1_001.fastq > Data/Hash_Results/2010AC19101_S23_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19101_S23_L001_R2_001.fastq > Data/Hash_Results/2010AC19101_S23_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19102_S20_L001_R1_001.fastq > Data/Hash_Results/2010AC19102_S20_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19102_S20_L001_R2_001.fastq > Data/Hash_Results/2010AC19102_S20_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19105_S24_L001_R1_001.fastq > Data/Hash_Results/2010AC19105_S24_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19105_S24_L001_R2_001.fastq > Data/Hash_Results/2010AC19105_S24_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19559_S3_L001_R1_001.fastq > Data/Hash_Results/2010AC19559_S3_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19559_S3_L001_R2_001.fastq > Data/Hash_Results/2010AC19559_S3_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19565_S4_L001_R1_001.fastq > Data/Hash_Results/2010AC19565_S4_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19565_S4_L001_R2_001.fastq > Data/Hash_Results/2010AC19565_S4_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19685_S5_L001_R1_001.fastq > Data/Hash_Results/2010AC19685_S5_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19685_S5_L001_R2_001.fastq > Data/Hash_Results/2010AC19685_S5_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19689_S6_L001_R1_001.fastq > Data/Hash_Results/2010AC19689_S6_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19689_S6_L001_R2_001.fastq > Data/Hash_Results/2010AC19689_S6_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19909_S1_L001_R1_001.fastq > Data/Hash_Results/2010AC19909_S1_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19909_S1_L001_R2_001.fastq > Data/Hash_Results/2010AC19909_S1_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19938_S7_L001_R1_001.fastq > Data/Hash_Results/2010AC19938_S7_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19938_S7_L001_R2_001.fastq > Data/Hash_Results/2010AC19938_S7_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19940_S8_L001_R1_001.fastq > Data/Hash_Results/2010AC19940_S8_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC19940_S8_L001_R2_001.fastq > Data/Hash_Results/2010AC19940_S8_L001_R2_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC20099_S2_L001_R1_001.fastq > Data/Hash_Results/2010AC20099_S2_L001_R1_001.txt\n",
    "!perl Scripts/hashFastq.pl Data/Unzipped_Data/2010AC20099_S2_L001_R2_001.fastq > Data/Hash_Results/2010AC20099_S2_L001_R2_001.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "test that required programs are working:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java version \"1.8.0_131\"\r\n",
      "Java(TM) SE Runtime Environment (build 1.8.0_131-b11)\r\n",
      "Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)\r\n"
     ]
    }
   ],
   "source": [
    "!java -version"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--------------------------------------------------------------------------------\r\n",
      "The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18\r\n",
      "Copyright (c) 2010-2016 The Broad Institute\r\n",
      "For support and documentation go to https://software.broadinstitute.org/gatk\r\n",
      "[Mon Oct 16 10:50:37 PDT 2017] Executing on Linux 4.8.0-59-generic amd64\r\n",
      "Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11\r\n",
      "--------------------------------------------------------------------------------\r\n",
      "--------------------------------------------------------------------------------\r\n",
      "usage: java -jar GenomeAnalysisTK.jar -T <analysis_type> [-args <arg_file>] [-I <input_file>] [--showFullBamList] [-rbs \r\n",
      "       <read_buffer_size>] [-rf <read_filter>] [-drf <disable_read_filter>] [-L <intervals>] [-XL <excludeIntervals>] [-isr \r\n",
      "       <interval_set_rule>] [-im <interval_merging>] [-ip <interval_padding>] [-R <reference_sequence>] [-ndrs] [-maxRuntime \r\n",
      "       <maxRuntime>] [-maxRuntimeUnits <maxRuntimeUnits>] [-dt <downsampling_type>] [-dfrac <downsample_to_fraction>] [-dcov \r\n",
      "       <downsample_to_coverage>] [-baq <baq>] [-baqGOP <baqGapOpenPenalty>] [-fixNDN] [-fixMisencodedQuals] \r\n",
      "       [-allowPotentiallyMisencodedQuals] [-OQ] [-DBQ <defaultBaseQualities>] [-PF <performanceLog>] [-BQSR <BQSR>] [-qq \r\n",
      "       <quantize_quals>] [-SQQ <static_quantized_quals>] [-DIQ] [-EOQ] [-preserveQ <preserve_qscores_less_than>] \r\n",
      "       [-globalQScorePrior <globalQScorePrior>] [-secondsBetweenProgressUpdates <secondsBetweenProgressUpdates>] [-S \r\n",
      "       <validation_strictness>] [-rpr] [-kpr] [-sample_rename_mapping_file <sample_rename_mapping_file>] [-U <unsafe>] \r\n",
      "       [-disable_auto_index_creation_and_locking_when_reading_rods] [-no_cmdline_in_header] [-sites_only] [-writeFullFormat] \r\n",
      "       [-compress <bam_compression>] [-simplifyBAM] [--disable_bam_indexing] [--generate_md5] [-nt <num_threads>] [-nct \r\n",
      "       <num_cpu_threads_per_data_thread>] [-mte] [-rgbl <read_group_black_list>] [-ped <pedigree>] [-pedString \r\n",
      "       <pedigreeString>] [-pedValidationType <pedigreeValidationType>] [-variant_index_type <variant_index_type>] \r\n",
      "       [-variant_index_parameter <variant_index_parameter>] [-ref_win_stop <reference_window_stop>] [-l <logging_level>] [-log \r\n",
      "       <log_to_file>] [-h] [-version]\r\n",
      "\r\n",
      " -T,--analysis_type <analysis_type>                                                       Name of the tool to run\r\n",
      " -args,--arg_file <arg_file>                                                              Reads arguments from the \r\n",
      "                                                                                          specified file\r\n",
      " -I,--input_file <input_file>                                                             Input file containing sequence \r\n",
      "                                                                                          data (BAM or CRAM)\r\n",
      " --showFullBamList                                                                        Emit list of input BAM/CRAM \r\n",
      "                                                                                          files to log\r\n",
      " -rbs,--read_buffer_size <read_buffer_size>                                               Number of reads per SAM file \r\n",
      "                                                                                          to buffer in memory\r\n",
      " -rf,--read_filter <read_filter>                                                          Filters to apply to reads \r\n",
      "                                                                                          before analysis\r\n",
      " -drf,--disable_read_filter <disable_read_filter>                                         Read filters to disable\r\n",
      " -L,--intervals <intervals>                                                               One or more genomic intervals \r\n",
      "                                                                                          over which to operate\r\n",
      " -XL,--excludeIntervals <excludeIntervals>                                                One or more genomic intervals \r\n",
      "                                                                                          to exclude from processing\r\n",
      " -isr,--interval_set_rule <interval_set_rule>                                             Set merging approach to use \r\n",
      "                                                                                          for combining interval inputs \r\n",
      "                                                                                          (UNION|INTERSECTION)\r\n",
      " -im,--interval_merging <interval_merging>                                                Interval merging rule for \r\n",
      "                                                                                          abutting intervals (ALL|\r\n",
      "                                                                                          OVERLAPPING_ONLY)\r\n",
      " -ip,--interval_padding <interval_padding>                                                Amount of padding (in bp) to \r\n",
      "                                                                                          add to each interval\r\n",
      " -R,--reference_sequence <reference_sequence>                                             Reference sequence file\r\n",
      " -ndrs,--nonDeterministicRandomSeed                                                       Use a non-deterministic random \r\n",
      "                                                                                          seed\r\n",
      " -maxRuntime,--maxRuntime <maxRuntime>                                                    Stop execution cleanly as soon \r\n",
      "                                                                                          as maxRuntime has been reached\r\n",
      " -maxRuntimeUnits,--maxRuntimeUnits <maxRuntimeUnits>                                     Unit of time used by \r\n",
      "                                                                                          maxRuntime (NANOSECONDS|\r\n",
      "                                                                                          MICROSECONDS|MILLISECONDS|\r\n",
      "                                                                                          SECONDS|MINUTES|HOURS|DAYS)\r\n",
      " -dt,--downsampling_type <downsampling_type>                                              Type of read downsampling to \r\n",
      "                                                                                          employ at a given locus (NONE|\r\n",
      "                                                                                          ALL_READS|BY_SAMPLE)\r\n",
      " -dfrac,--downsample_to_fraction <downsample_to_fraction>                                 Fraction of reads to \r\n",
      "                                                                                          downsample to\r\n",
      " -dcov,--downsample_to_coverage <downsample_to_coverage>                                  Target coverage threshold for \r\n",
      "                                                                                          downsampling to coverage\r\n",
      " -baq,--baq <baq>                                                                         Type of BAQ calculation to \r\n",
      "                                                                                          apply in the engine (OFF|\r\n",
      "                                                                                          CALCULATE_AS_NECESSARY|\r\n",
      "                                                                                          RECALCULATE)\r\n",
      " -baqGOP,--baqGapOpenPenalty <baqGapOpenPenalty>                                          BAQ gap open penalty\r\n",
      " -fixNDN,--refactor_NDN_cigar_string                                                      Reduce NDN elements in CIGAR \r\n",
      "                                                                                          string\r\n",
      " -fixMisencodedQuals,--fix_misencoded_quality_scores                                      Fix mis-encoded base quality \r\n",
      "                                                                                          scores\r\n",
      " -allowPotentiallyMisencodedQuals,--allow_potentially_misencoded_quality_scores           Ignore warnings about base \r\n",
      "                                                                                          quality score encoding\r\n",
      " -OQ,--useOriginalQualities                                                               Use the base quality scores \r\n",
      "                                                                                          from the OQ tag\r\n",
      " -DBQ,--defaultBaseQualities <defaultBaseQualities>                                       Assign a default base quality\r\n",
      " -PF,--performanceLog <performanceLog>                                                    Write GATK runtime performance \r\n",
      "                                                                                          log to this file\r\n",
      " -BQSR,--BQSR <BQSR>                                                                      Input covariates table file \r\n",
      "                                                                                          for on-the-fly base quality \r\n",
      "                                                                                          score recalibration\r\n",
      " -qq,--quantize_quals <quantize_quals>                                                    Quantize quality scores to a \r\n",
      "                                                                                          given number of levels (with \r\n",
      "                                                                                          -BQSR)\r\n",
      " -SQQ,--static_quantized_quals <static_quantized_quals>                                   Use static quantized quality \r\n",
      "                                                                                          scores to a given number of \r\n",
      "                                                                                          levels (with -BQSR)\r\n",
      " -DIQ,--disable_indel_quals                                                               Disable printing of base \r\n",
      "                                                                                          insertion and deletion tags \r\n",
      "                                                                                          (with -BQSR)\r\n",
      " -EOQ,--emit_original_quals                                                               Emit the OQ tag with the \r\n",
      "                                                                                          original base qualities (with \r\n",
      "                                                                                          -BQSR)\r\n",
      " -preserveQ,--preserve_qscores_less_than <preserve_qscores_less_than>                     Don't recalibrate bases with \r\n",
      "                                                                                          quality scores less than this \r\n",
      "                                                                                          threshold (with -BQSR)\r\n",
      " -globalQScorePrior,--globalQScorePrior <globalQScorePrior>                               Global Qscore Bayesian prior \r\n",
      "                                                                                          to use for BQSR\r\n",
      " -secondsBetweenProgressUpdates,--secondsBetweenProgressUpdates                           Time interval for process \r\n",
      "<secondsBetweenProgressUpdates>                                                           meter information output (in \r\n",
      "                                                                                          seconds)\r\n",
      " -S,--validation_strictness <validation_strictness>                                       How strict should we be with \r\n",
      "                                                                                          validation (STRICT|LENIENT|\r\n",
      "                                                                                          SILENT)\r\n",
      " -rpr,--remove_program_records                                                            Remove program records from \r\n",
      "                                                                                          the SAM header\r\n",
      " -kpr,--keep_program_records                                                              Keep program records in the \r\n",
      "                                                                                          SAM header\r\n",
      " -sample_rename_mapping_file,--sample_rename_mapping_file <sample_rename_mapping_file>    Rename sample IDs on-the-fly \r\n",
      "                                                                                          at runtime using the provided \r\n",
      "                                                                                          mapping file\r\n",
      " -U,--unsafe <unsafe>                                                                     Enable unsafe operations: \r\n",
      "                                                                                          nothing will be checked at \r\n",
      "                                                                                          runtime (ALLOW_N_CIGAR_READS|\r\n",
      "                                                                                          ALLOW_UNINDEXED_BAM|\r\n",
      "                                                                                          ALLOW_UNSET_BAM_SORT_ORDER|\r\n",
      "                                                                                          NO_READ_ORDER_VERIFICATION|\r\n",
      "                                                                                          ALLOW_SEQ_DICT_INCOMPATIBILITY|\r\n",
      "                                                                                          LENIENT_VCF_PROCESSING|ALL)\r\n",
      "d_locking_when_reading_rods,--disable_auto_index_creation_and_locking_when_reading_rods   Disable both auto-generation \r\n",
      "                                                                                          of index files and index file \r\n",
      "                                                                                          locking\r\n",
      " -no_cmdline_in_header,--no_cmdline_in_header                                             Don't include the command line \r\n",
      "                                                                                          in output VCF headers\r\n",
      " -sites_only,--sites_only                                                                 Output sites-only VCF\r\n",
      " -writeFullFormat,--never_trim_vcf_format_field                                           Always output all the records \r\n",
      "                                                                                          in VCF FORMAT fields, even if \r\n",
      "                                                                                          some are missing\r\n",
      " -compress,--bam_compression <bam_compression>                                            Compression level to use for \r\n",
      "                                                                                          writing BAM files (0 - 9, \r\n",
      "                                                                                          higher is more compressed)\r\n",
      " -simplifyBAM,--simplifyBAM                                                               Strip down read content and \r\n",
      "                                                                                          tags\r\n",
      " --disable_bam_indexing                                                                   Turn off on-the-fly creation \r\n",
      "                                                                                          of indices for output BAM/CRAM \r\n",
      "                                                                                          files\r\n",
      " --generate_md5                                                                           Enable on-the-fly creation of \r\n",
      "                                                                                          md5s for output BAM files.\r\n",
      " -nt,--num_threads <num_threads>                                                          Number of data threads to \r\n",
      "                                                                                          allocate to this analysis\r\n",
      " -nct,--num_cpu_threads_per_data_thread <num_cpu_threads_per_data_thread>                 Number of CPU threads to \r\n",
      "                                                                                          allocate per data thread\r\n",
      " -mte,--monitorThreadEfficiency                                                           Enable threading efficiency \r\n",
      "                                                                                          monitoring\r\n",
      " -rgbl,--read_group_black_list <read_group_black_list>                                    Exclude read groups based on \r\n",
      "                                                                                          tags\r\n",
      " -ped,--pedigree <pedigree>                                                               Pedigree files for samples\r\n",
      " -pedString,--pedigreeString <pedigreeString>                                             Pedigree string for samples\r\n",
      " -pedValidationType,--pedigreeValidationType <pedigreeValidationType>                     Validation strictness for \r\n",
      "                                                                                          pedigree (STRICT|SILENT)\r\n",
      " -variant_index_type,--variant_index_type <variant_index_type>                            Type of IndexCreator to use \r\n",
      "                                                                                          for VCF/BCF indices \r\n",
      "                                                                                          (DYNAMIC_SEEK|DYNAMIC_SIZE|\r\n",
      "                                                                                          LINEAR|INTERVAL)\r\n",
      " -variant_index_parameter,--variant_index_parameter <variant_index_parameter>             Parameter to pass to the \r\n",
      "                                                                                          VCF/BCF IndexCreator\r\n",
      " -ref_win_stop,--reference_window_stop <reference_window_stop>                            Reference window stop\r\n",
      " -l,--logging_level <logging_level>                                                       Set the minimum level of \r\n",
      "                                                                                          logging\r\n",
      " -log,--log_to_file <log_to_file>                                                         Set the logging location\r\n",
      " -h,--help                                                                                Generate the help message\r\n",
      " -version,--version                                                                       Output version information\r\n",
      "\r\n",
      " annotator                       \r\n",
      "   VariantAnnotator              Annotate variant calls with context information\r\n",
      "                                 \r\n",
      " bqsr                            \r\n",
      "   AnalyzeCovariates             Create plots to visualize base recalibration results\r\n",
      "   BaseRecalibrator              Detect systematic errors in base quality scores\r\n",
      "                                 \r\n",
      " cancer                          \r\n",
      "   AssignSomaticStatus           Assigns somatic status to a set of calls\r\n",
      "                                 \r\n",
      " contamination                   \r\n",
      "   AnnotatePopulationAFWalker    Given a input VCF representing a collection of populations, split the input into each \r\n",
      "                                 population, and annotate each record with population allele frequencies\r\n",
      "   ContEst                       Estimate cross-sample contamination\r\n",
      "                                 \r\n",
      " coverage                        \r\n",
      "   CallableLoci                  Collect statistics on callable, uncallable, poorly mapped, and other parts of the \r\n",
      "                                 genome\r\n",
      "   CompareCallableLoci           Compare callability statistics\r\n",
      "   DepthOfCoverage               Assess sequence coverage by a wide array of metrics, partitioned by sample, read group, \r\n",
      "                                 or library\r\n",
      "   GCContentByInterval           Calculates the GC content of the reference sequence for each interval\r\n",
      "                                 \r\n",
      " diagnosetargets                 \r\n",
      "   DiagnoseTargets               Analyze coverage distribution and validate read mates per interval and per sample\r\n",
      "                                 \r\n",
      " diagnostics                     \r\n",
      "   ErrorRatePerCycle             Compute the read error rate per position\r\n",
      "   FindCoveredIntervals          Outputs a list of intervals that are covered to or above a given threshold\r\n",
      "   ReadGroupProperties           Collect statistics about read groups and their properties\r\n",
      "   ReadLengthDistribution        Collect read length statistics\r\n",
      "                                 \r\n",
      " diffengine                      \r\n",
      "   DiffObjects                   A generic engine for comparing tree-structured objects\r\n",
      "                                 \r\n",
      " examples                        \r\n",
      "   GATKPaperGenotyper            Simple Bayesian genotyper used in the original GATK paper\r\n",
      "                                 \r\n",
      " fasta                           \r\n",
      "   FastaAlternateReferenceMaker  Generate an alternative reference sequence over the specified interval\r\n",
      "   FastaReferenceMaker           Create a subset of a FASTA reference sequence\r\n",
      "   FastaStats                    Calculate basic statistics about the reference sequence itself\r\n",
      "                                 \r\n",
      " filters                         \r\n",
      "   VariantFiltration             Filter variant calls based on INFO and FORMAT annotations\r\n",
      "                                 \r\n",
      " genotyper                       \r\n",
      "   UnifiedGenotyper              Call SNPs and indels on a per-locus basis\r\n",
      "                                 \r\n",
      " haplotypecaller                 \r\n",
      "   HaplotypeCaller               Call germline SNPs and indels via local re-assembly of haplotypes\r\n",
      "   HaplotypeResolver             Haplotype-based resolution of variants in separate callsets.\r\n",
      "                                 \r\n",
      " indels                          \r\n",
      "   IndelRealigner                Perform local realignment of reads around indels\r\n",
      "   LeftAlignIndels               Left-align indels within reads in a bam file\r\n",
      "   RealignerTargetCreator        Define intervals to target for local realignment\r\n",
      "                                 \r\n",
      " m2                              \r\n",
      "   MuTect2                       Call somatic SNPs and indels via local re-assembly of haplotypes\r\n",
      "                                 \r\n",
      " missing                         \r\n",
      "   QualifyMissingIntervals       Collect quality metrics for a set of intervals\r\n",
      "                                 \r\n",
      " phasing                         \r\n",
      "   PhaseByTransmission           Compute the most likely genotype combination and phasing for trios and parent/child \r\n",
      "                                 pairs\r\n",
      "   ReadBackedPhasing             Annotate physical phasing information\r\n",
      "                                 \r\n",
      " qc                              \r\n",
      "   CheckPileup                   Compare GATK's internal pileup to a reference Samtools pileup\r\n",
      "   CountBases                    Count the number of bases in a set of reads\r\n",
      "   CountIntervals                Count contiguous regions in an interval list\r\n",
      "   CountLoci                     Count the total number of covered loci\r\n",
      "   CountMales                    Count the number of reads seen from male samples\r\n",
      "   CountReadEvents               Count the number of read events\r\n",
      "   CountReads                    Count the number of reads\r\n",
      "   CountRODs                     Count the number of ROD objects encountered\r\n",
      "   CountRODsByRef                Count the number of ROD objects encountered along the reference\r\n",
      "   CountTerminusEvent            Count the number of reads ending in insertions, deletions or soft-clips\r\n",
      "   ErrorThrowing                 A walker that simply throws errors.\r\n",
      "   FlagStat                      Collect statistics about sequence reads based on their SAM flags\r\n",
      "   Pileup                        Print read alignments in Pileup-style format\r\n",
      "   PrintRODs                     Print out all of the RODs in the input data set\r\n",
      "   QCRef                         Quality control for the reference fasta\r\n",
      "   ReadClippingStats             Collect read clipping statistics\r\n",
      "                                 \r\n",
      " readutils                       \r\n",
      "   ClipReads                     Read clipping based on quality, position or sequence matching\r\n",
      "   PrintReads                    Write out sequence read data (for filtering, merging, subsetting etc)\r\n",
      "   SplitSamFile                  Split a BAM file by sample\r\n",
      "                                 \r\n",
      " rnaseq                          \r\n",
      "   ASEReadCounter                Calculate read counts per allele for allele-specific expression analysis\r\n",
      "   SplitNCigarReads              Splits reads that contain Ns in their CIGAR string\r\n",
      "                                 \r\n",
      " simulatereads                   \r\n",
      "   SimulateReadsForVariants      Generate simulated reads for variants\r\n",
      "                                 \r\n",
      " validationsiteselector          \r\n",
      "   ValidationSiteSelector        Randomly select variant records according to specified options\r\n",
      "                                 \r\n",
      " varianteval                     \r\n",
      "   VariantEval                   General-purpose tool for variant evaluation (% in dbSNP, genotype concordance, Ti/Tv \r\n",
      "                                 ratios, and a lot more)\r\n",
      "                                 \r\n",
      " variantrecalibration            \r\n",
      "   ApplyRecalibration            Apply a score cutoff to filter variants based on a recalibration table\r\n",
      "   VariantRecalibrator           Build a recalibration model to score variant quality for filtering purposes\r\n",
      "                                 \r\n",
      " variantutils                    \r\n",
      "   CalculateGenotypePosteriors   Calculate genotype posterior likelihoods given panel data\r\n",
      "   CombineGVCFs                  Combine per-sample gVCF files produced by HaplotypeCaller into a multi-sample gVCF file\r\n",
      "   CombineVariants               Combine variant records from different sources\r\n",
      "   GenotypeConcordance           Genotype concordance between two callsets\r\n",
      "   GenotypeGVCFs                 Perform joint genotyping on gVCF files produced by HaplotypeCaller\r\n",
      "   LeftAlignAndTrimVariants      Left-align indels in a variant callset\r\n",
      "   RandomlySplitVariants         Randomly split variants into different sets\r\n",
      "   RegenotypeVariants            Regenotypes the variants from a VCF containing PLs or GLs.\r\n",
      "   SelectHeaders                 Selects headers from a VCF source\r\n",
      "   SelectVariants                Select a subset of variants from a larger callset\r\n",
      "   ValidateVariants              Validate a VCF file with an extra strict set of criteria\r\n",
      "   VariantsToAllelicPrimitives   Simplify multi-nucleotide variants (MNPs) into more basic/primitive alleles.\r\n",
      "   VariantsToBinaryPed           Convert VCF to binary pedigree file\r\n",
      "   VariantsToTable               Extract specific fields from a VCF file to a tab-delimited table\r\n",
      "   VariantsToVCF                 Convert variants from other file formats to VCF format\r\n",
      "                                 \r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!java -jar /mnt/hgfs/Ubuntu_Shared/Optimization_Run1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -h"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "Program: samtools (Tools for alignments in the SAM format)\r\n",
      "Version: 1.3.1 (using htslib 1.3.1)\r\n",
      "\r\n",
      "Usage:   samtools <command> [options]\r\n",
      "\r\n",
      "Commands:\r\n",
      "  -- Indexing\r\n",
      "     dict           create a sequence dictionary file\r\n",
      "     faidx          index/extract FASTA\r\n",
      "     index          index alignment\r\n",
      "\r\n",
      "  -- Editing\r\n",
      "     calmd          recalculate MD/NM tags and '=' bases\r\n",
      "     fixmate        fix mate information\r\n",
      "     reheader       replace BAM header\r\n",
      "     rmdup          remove PCR duplicates\r\n",
      "     targetcut      cut fosmid regions (for fosmid pool only)\r\n",
      "     addreplacerg   adds or replaces RG tags\r\n",
      "\r\n",
      "  -- File operations\r\n",
      "     collate        shuffle and group alignments by name\r\n",
      "     cat            concatenate BAMs\r\n",
      "     merge          merge sorted alignments\r\n",
      "     mpileup        multi-way pileup\r\n",
      "     sort           sort alignment file\r\n",
      "     split          splits a file by read group\r\n",
      "     quickcheck     quickly check if SAM/BAM/CRAM file appears intact\r\n",
      "     fastq          converts a BAM to a FASTQ\r\n",
      "     fasta          converts a BAM to a FASTA\r\n",
      "\r\n",
      "  -- Statistics\r\n",
      "     bedcov         read depth per BED region\r\n",
      "     depth          compute the depth\r\n",
      "     flagstat       simple stats\r\n",
      "     idxstats       BAM index stats\r\n",
      "     phase          phase heterozygotes\r\n",
      "     stats          generate stats (former bamcheck)\r\n",
      "\r\n",
      "  -- Viewing\r\n",
      "     flags          explain BAM flags\r\n",
      "     tview          text alignment viewer\r\n",
      "     view           SAM<->BAM<->CRAM conversion\r\n",
      "     depad          convert padded BAM to unpadded BAM\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!samtools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No index, query, or output file specified!\r\n",
      "Bowtie 2 version 2.2.9 by Ben Langmead (langmea@cs.jhu.edu, www.cs.jhu.edu/~langmea)\r\n",
      "Usage: \r\n",
      "  bowtie2 [options]* -x <bt2-idx> {-1 <m1> -2 <m2> | -U <r>} [-S <sam>]\r\n",
      "\r\n",
      "  <bt2-idx>  Index filename prefix (minus trailing .X.bt2).\r\n",
      "             NOTE: Bowtie 1 and Bowtie 2 indexes are not compatible.\r\n",
      "  <m1>       Files with #1 mates, paired with files in <m2>.\r\n",
      "             Could be gzip'ed (extension: .gz) or bzip2'ed (extension: .bz2).\r\n",
      "  <m2>       Files with #2 mates, paired with files in <m1>.\r\n",
      "             Could be gzip'ed (extension: .gz) or bzip2'ed (extension: .bz2).\r\n",
      "  <r>        Files with unpaired reads.\r\n",
      "             Could be gzip'ed (extension: .gz) or bzip2'ed (extension: .bz2).\r\n",
      "  <sam>      File for SAM output (default: stdout)\r\n",
      "\r\n",
      "  <m1>, <m2>, <r> can be comma-separated lists (no whitespace) and can be\r\n",
      "  specified many times.  E.g. '-U file1.fq,file2.fq -U file3.fq'.\r\n",
      "\r\n",
      "Options (defaults in parentheses):\r\n",
      "\r\n",
      " Input:\r\n",
      "  -q                 query input files are FASTQ .fq/.fastq (default)\r\n",
      "  --qseq             query input files are in Illumina's qseq format\r\n",
      "  -f                 query input files are (multi-)FASTA .fa/.mfa\r\n",
      "  -r                 query input files are raw one-sequence-per-line\r\n",
      "  -c                 <m1>, <m2>, <r> are sequences themselves, not files\r\n",
      "  -s/--skip <int>    skip the first <int> reads/pairs in the input (none)\r\n",
      "  -u/--upto <int>    stop after first <int> reads/pairs (no limit)\r\n",
      "  -5/--trim5 <int>   trim <int> bases from 5'/left end of reads (0)\r\n",
      "  -3/--trim3 <int>   trim <int> bases from 3'/right end of reads (0)\r\n",
      "  --phred33          qualities are Phred+33 (default)\r\n",
      "  --phred64          qualities are Phred+64\r\n",
      "  --int-quals        qualities encoded as space-delimited integers\r\n",
      "\r\n",
      " Presets:                 Same as:\r\n",
      "  For --end-to-end:\r\n",
      "   --very-fast            -D 5 -R 1 -N 0 -L 22 -i S,0,2.50\r\n",
      "   --fast                 -D 10 -R 2 -N 0 -L 22 -i S,0,2.50\r\n",
      "   --sensitive            -D 15 -R 2 -N 0 -L 22 -i S,1,1.15 (default)\r\n",
      "   --very-sensitive       -D 20 -R 3 -N 0 -L 20 -i S,1,0.50\r\n",
      "\r\n",
      "  For --local:\r\n",
      "   --very-fast-local      -D 5 -R 1 -N 0 -L 25 -i S,1,2.00\r\n",
      "   --fast-local           -D 10 -R 2 -N 0 -L 22 -i S,1,1.75\r\n",
      "   --sensitive-local      -D 15 -R 2 -N 0 -L 20 -i S,1,0.75 (default)\r\n",
      "   --very-sensitive-local -D 20 -R 3 -N 0 -L 20 -i S,1,0.50\r\n",
      "\r\n",
      " Alignment:\r\n",
      "  -N <int>           max # mismatches in seed alignment; can be 0 or 1 (0)\r\n",
      "  -L <int>           length of seed substrings; must be >3, <32 (22)\r\n",
      "  -i <func>          interval between seed substrings w/r/t read len (S,1,1.15)\r\n",
      "  --n-ceil <func>    func for max # non-A/C/G/Ts permitted in aln (L,0,0.15)\r\n",
      "  --dpad <int>       include <int> extra ref chars on sides of DP table (15)\r\n",
      "  --gbar <int>       disallow gaps within <int> nucs of read extremes (4)\r\n",
      "  --ignore-quals     treat all quality values as 30 on Phred scale (off)\r\n",
      "  --nofw             do not align forward (original) version of read (off)\r\n",
      "  --norc             do not align reverse-complement version of read (off)\r\n",
      "  --no-1mm-upfront   do not allow 1 mismatch alignments before attempting to\r\n",
      "                     scan for the optimal seeded alignments\r\n",
      "  --end-to-end       entire read must align; no clipping (on)\r\n",
      "   OR\r\n",
      "  --local            local alignment; ends might be soft clipped (off)\r\n",
      "\r\n",
      " Scoring:\r\n",
      "  --ma <int>         match bonus (0 for --end-to-end, 2 for --local) \r\n",
      "  --mp <int>         max penalty for mismatch; lower qual = lower penalty (6)\r\n",
      "  --np <int>         penalty for non-A/C/G/Ts in read/ref (1)\r\n",
      "  --rdg <int>,<int>  read gap open, extend penalties (5,3)\r\n",
      "  --rfg <int>,<int>  reference gap open, extend penalties (5,3)\r\n",
      "  --score-min <func> min acceptable alignment score w/r/t read length\r\n",
      "                     (G,20,8 for local, L,-0.6,-0.6 for end-to-end)\r\n",
      "\r\n",
      " Reporting:\r\n",
      "  (default)          look for multiple alignments, report best, with MAPQ\r\n",
      "   OR\r\n",
      "  -k <int>           report up to <int> alns per read; MAPQ not meaningful\r\n",
      "   OR\r\n",
      "  -a/--all           report all alignments; very slow, MAPQ not meaningful\r\n",
      "\r\n",
      " Effort:\r\n",
      "  -D <int>           give up extending after <int> failed extends in a row (15)\r\n",
      "  -R <int>           for reads w/ repetitive seeds, try <int> sets of seeds (2)\r\n",
      "\r\n",
      " Paired-end:\r\n",
      "  -I/--minins <int>  minimum fragment length (0)\r\n",
      "  -X/--maxins <int>  maximum fragment length (500)\r\n",
      "  --fr/--rf/--ff     -1, -2 mates align fw/rev, rev/fw, fw/fw (--fr)\r\n",
      "  --no-mixed         suppress unpaired alignments for paired reads\r\n",
      "  --no-discordant    suppress discordant alignments for paired reads\r\n",
      "  --no-dovetail      not concordant when mates extend past each other\r\n",
      "  --no-contain       not concordant when one mate alignment contains other\r\n",
      "  --no-overlap       not concordant when mates overlap at all\r\n",
      "\r\n",
      " Output:\r\n",
      "  -t/--time          print wall-clock time taken by search phases\r\n",
      "  --un <path>           write unpaired reads that didn't align to <path>\r\n",
      "  --al <path>           write unpaired reads that aligned at least once to <path>\r\n",
      "  --un-conc <path>      write pairs that didn't align concordantly to <path>\r\n",
      "  --al-conc <path>      write pairs that aligned concordantly at least once to <path>\r\n",
      "  (Note: for --un, --al, --un-conc, or --al-conc, add '-gz' to the option name, e.g.\r\n",
      "  --un-gz <path>, to gzip compress output, or add '-bz2' to bzip2 compress output.)\r\n",
      "  --quiet            print nothing to stderr except serious errors\r\n",
      "  --met-file <path>  send metrics to file at <path> (off)\r\n",
      "  --met-stderr       send metrics to stderr (off)\r\n",
      "  --met <int>        report internal counters & metrics every <int> secs (1)\r\n",
      "  --no-unal          suppress SAM records for unaligned reads\r\n",
      "  --no-head          suppress header lines, i.e. lines starting with @\r\n",
      "  --no-sq            suppress @SQ header lines\r\n",
      "  --rg-id <text>     set read group id, reflected in @RG line and RG:Z: opt field\r\n",
      "  --rg <text>        add <text> (\"lab:value\") to @RG line of SAM header.\r\n",
      "                     Note: @RG line only printed when --rg-id is set.\r\n",
      "  --omit-sec-seq     put '*' in SEQ and QUAL fields for secondary alignments.\r\n",
      "\r\n",
      " Performance:\r\n",
      "  -p/--threads <int> number of alignment threads to launch (1)\r\n",
      "  --reorder          force SAM output order to match order of input reads\r\n",
      "  --mm               use memory-mapped I/O for index; many 'bowtie's can share\r\n",
      "\r\n",
      " Other:\r\n",
      "  --qc-filter        filter out reads that are bad according to QSEQ filter\r\n",
      "  --seed <int>       seed for random number generator (0)\r\n",
      "  --non-deterministic seed rand. gen. arbitrarily instead of using read attributes\r\n",
      "  --version          print version information and quit\r\n",
      "  -h/--help          print this usage message\r\n",
      "(ERR): bowtie2-align exited with value 1\r\n"
     ]
    }
   ],
   "source": [
    "!bowtie2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[31mUSAGE: PicardCommandLine \u001b[32m<program name>\u001b[1m\u001b[31m [-h]\r\n",
      "\r\n",
      "\u001b[0m\u001b[1m\u001b[31mAvailable Programs:\r\n",
      "\u001b[0m\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mAlpha Tools:                                     Tools that are currently UNSUPPORTED until further testing and maturation.\u001b[0m\r\n",
      "\u001b[32m    CollectIndependentReplicateMetrics           \u001b[36mEstimates the rate of independent replication of reads within a bam.\u001b[0m\r\n",
      "\u001b[32m    CollectWgsMetricsWithNonZeroCoverage         \u001b[36mCollect metrics about coverage and performance of whole genome sequencing (WGS) experiments.  \u001b[0m\r\n",
      "\u001b[32m    UmiAwareMarkDuplicatesWithMateCigar          \u001b[36mIdentifies duplicate reads using information from read positions and UMIs. \u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mFasta:                                           Tools for manipulating FASTA, or related data.\u001b[0m\r\n",
      "\u001b[32m    CreateSequenceDictionary                     \u001b[36mCreates a sequence dictionary for a reference sequence.  \u001b[0m\r\n",
      "\u001b[32m    ExtractSequences                             \u001b[36mSubsets intervals from a reference sequence to a new FASTA file.\u001b[0m\r\n",
      "\u001b[32m    NonNFastaSize                                \u001b[36mCounts the number of non-N bases in a fasta file.\u001b[0m\r\n",
      "\u001b[32m    NormalizeFasta                               \u001b[36mNormalizes lines of sequence in a FASTA file to be of the same length.\u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mFingerprinting Tools:                            Tools for manipulating fingerprints, or related data.\u001b[0m\r\n",
      "\u001b[32m    CheckFingerprint                             \u001b[36mComputes a fingerprint from the supplied input (SAM/BAM or VCF) file and compares it to the provided genotypes\u001b[0m\r\n",
      "\u001b[32m    CrosscheckReadGroupFingerprints              \u001b[36mChecks if all read groups appear to come from the same individual\u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mIllumina Tools:                                  Tools for manipulating data specific to Illumina sequencers.\u001b[0m\r\n",
      "\u001b[32m    CheckIlluminaDirectory                       \u001b[36mAsserts the validity for specified Illumina basecalling data.  \u001b[0m\r\n",
      "\u001b[32m    CollectIlluminaBasecallingMetrics            \u001b[36mCollects Illumina Basecalling metrics for a sequencing run.  \u001b[0m\r\n",
      "\u001b[32m    CollectIlluminaLaneMetrics                   \u001b[36mCollects Illumina lane metrics for the given BaseCalling analysis directory.  \u001b[0m\r\n",
      "\u001b[32m    ExtractIlluminaBarcodes                      \u001b[36mTool determines the barcode for each read in an Illumina lane.  \u001b[0m\r\n",
      "\u001b[32m    IlluminaBasecallsToFastq                     \u001b[36mGenerate FASTQ file(s) from Illumina basecall read data.  \u001b[0m\r\n",
      "\u001b[32m    IlluminaBasecallsToSam                       \u001b[36mTransforms raw Illumina sequencing data into an unmapped SAM or BAM file.\u001b[0m\r\n",
      "\u001b[32m    MarkIlluminaAdapters                         \u001b[36mReads a SAM or BAM file and rewrites it with new adapter-trimming tags.  \u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mInterval Tools:                                  Tools for manipulating Picard interval lists.\u001b[0m\r\n",
      "\u001b[32m    BedToIntervalList                            \u001b[36mConverts a BED file to a Picard Interval List.  \u001b[0m\r\n",
      "\u001b[32m    IntervalListToBed                            \u001b[36mConverts an Picard IntervalList file to a BED file.\u001b[0m\r\n",
      "\u001b[32m    IntervalListTools                            \u001b[36mManipulates interval lists.  \u001b[0m\r\n",
      "\u001b[32m    LiftOverIntervalList                         \u001b[36mLifts over an interval list from one reference build to another.  \u001b[0m\r\n",
      "\u001b[32m    ScatterIntervalsByNs                         \u001b[36mWrites an interval list based on splitting a reference by Ns.  \u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mMetrics:                                         Tools for reporting metrics on various data types.\u001b[0m\r\n",
      "\u001b[32m    CalculateHsMetrics                           \u001b[36mDEPRECATED: Use CollectHsMetrics instead.\u001b[0m\r\n",
      "\u001b[32m    CollectAlignmentSummaryMetrics               \u001b[36m<b>Produces a summary of alignment metrics from a SAM or BAM file.</b>  \u001b[0m\r\n",
      "\u001b[32m    CollectBaseDistributionByCycle               \u001b[36mChart the nucleotide distribution per cycle in a SAM or BAM file\u001b[0m\r\n",
      "\u001b[32m    CollectGcBiasMetrics                         \u001b[36mCollect metrics regarding GC bias. \u001b[0m\r\n",
      "\u001b[32m    CollectHiSeqXPfFailMetrics                   \u001b[36mClassify PF-Failing reads in a HiSeqX Illumina Basecalling directory into various categories.\u001b[0m\r\n",
      "\u001b[32m    CollectHsMetrics                             \u001b[36mCollects hybrid-selection (HS) metrics for a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    CollectInsertSizeMetrics                     \u001b[36mCollect metrics about the insert size distribution of a paired-end library.\u001b[0m\r\n",
      "\u001b[32m    CollectJumpingLibraryMetrics                 \u001b[36mCollect jumping library metrics. \u001b[0m\r\n",
      "\u001b[32m    CollectMultipleMetrics                       \u001b[36mCollect multiple classes of metrics.  \u001b[0m\r\n",
      "\u001b[32m    CollectOxoGMetrics                           \u001b[36mCollect metrics to assess oxidative artifacts.\u001b[0m\r\n",
      "\u001b[32m    CollectQualityYieldMetrics                   \u001b[36mCollect metrics about reads that pass quality thresholds and Illumina-specific filters.  \u001b[0m\r\n",
      "\u001b[32m    CollectRawWgsMetrics                         \u001b[36mCollect whole genome sequencing-related metrics.  \u001b[0m\r\n",
      "\u001b[32m    CollectRnaSeqMetrics                         \u001b[36mProduces RNA alignment metrics for a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    CollectRrbsMetrics                           \u001b[36m<b>Collects metrics from reduced representation bisulfite sequencing (Rrbs) data.</b>  \u001b[0m\r\n",
      "\u001b[32m    CollectSequencingArtifactMetrics             \u001b[36mCollect metrics to quantify single-base sequencing artifacts.  \u001b[0m\r\n",
      "\u001b[32m    CollectTargetedPcrMetrics                    \u001b[36mCalculate PCR-related metrics from targeted sequencing data. \u001b[0m\r\n",
      "\u001b[32m    CollectVariantCallingMetrics                 \u001b[36mCollects per-sample and aggregate (spanning all samples) metrics from the provided VCF file\u001b[0m\r\n",
      "\u001b[32m    CollectWgsMetrics                            \u001b[36mCollect metrics about coverage and performance of whole genome sequencing (WGS) experiments.\u001b[0m\r\n",
      "\u001b[32m    CollectWgsMetricsFromQuerySorted             \u001b[36mWrites sequencing-related metrics for a SAM or BAM file\u001b[0m\r\n",
      "\u001b[32m    CollectWgsMetricsFromSampledSites            \u001b[36mWrites whole genome sequencing-related metrics for a SAM or BAM file\u001b[0m\r\n",
      "\u001b[32m    CompareMetrics                               \u001b[36mCompare two metrics files.\u001b[0m\r\n",
      "\u001b[32m    ConvertSequencingArtifactToOxoG              \u001b[36mExtract OxoG metrics from generalized artifacts metrics.  \u001b[0m\r\n",
      "\u001b[32m    EstimateLibraryComplexity                    \u001b[36mEstimates the numbers of unique molecules in a sequencing library.  \u001b[0m\r\n",
      "\u001b[32m    MeanQualityByCycle                           \u001b[36mCollect mean quality by cycle.\u001b[0m\r\n",
      "\u001b[32m    QualityScoreDistribution                     \u001b[36mChart the distribution of quality scores.  \u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mMiscellaneous Tools:                             A set of miscellaneous tools.                \u001b[0m\r\n",
      "\u001b[32m    BaitDesigner                                 \u001b[36m<b>Designs oligonucleotide baits for hybrid selection reactions.</b>  \u001b[0m\r\n",
      "\u001b[32m    FifoBuffer                                   \u001b[36mFIFO buffer used to buffer input and output streams with a customizable buffer size \u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mSAM/BAM:                                         Tools for manipulating SAM, BAM, or related data.\u001b[0m\r\n",
      "\u001b[32m    AddCommentsToBam                             \u001b[36mAdds comments to the header of a BAM file.\u001b[0m\r\n",
      "\u001b[32m    AddOrReplaceReadGroups                       \u001b[36mReplace read groups in a BAM file.\u001b[0m\r\n",
      "\u001b[32m    BamIndexStats                                \u001b[36mGenerate index statistics from a BAM file\u001b[0m\r\n",
      "\u001b[32m    BamToBfq                                     \u001b[36mCreate BFQ files from a BAM file for use by the maq aligner.  \u001b[0m\r\n",
      "\u001b[32m    BuildBamIndex                                \u001b[36mGenerates a BAM index \".bai\" file.  \u001b[0m\r\n",
      "\u001b[32m    CalculateReadGroupChecksum                   \u001b[36mCreates a hash code based on the read groups (RG).  \u001b[0m\r\n",
      "\u001b[32m    CheckTerminatorBlock                         \u001b[36mAsserts the provided gzip file's (e.g., BAM) last block is well-formed; RC 100 otherwise\u001b[0m\r\n",
      "\u001b[32m    CleanSam                                     \u001b[36mCleans the provided SAM/BAM, soft-clipping beyond-end-of-reference alignments and setting MAPQ to 0 for unmapped reads\u001b[0m\r\n",
      "\u001b[32m    CompareSAMs                                  \u001b[36mCompare two input \".sam\" or \".bam\" files.  \u001b[0m\r\n",
      "\u001b[32m    DownsampleSam                                \u001b[36mDownsample a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    FastqToSam                                   \u001b[36mConverts a FASTQ file to an unaligned BAM or SAM file.  \u001b[0m\r\n",
      "\u001b[32m    FilterSamReads                               \u001b[36mSubset read data from a SAM or BAM file\u001b[0m\r\n",
      "\u001b[32m    FixMateInformation                           \u001b[36mVerify mate-pair information between mates and fix if needed.\u001b[0m\r\n",
      "\u001b[32m    GatherBamFiles                               \u001b[36mConcatenate one or more BAM files as efficiently as possible\u001b[0m\r\n",
      "\u001b[32m    MarkDuplicates                               \u001b[36mIdentifies duplicate reads.  \u001b[0m\r\n",
      "\u001b[32m    MarkDuplicatesWithMateCigar                  \u001b[36mIdentifies duplicate reads, accounting for mate CIGAR.  \u001b[0m\r\n",
      "\u001b[32m    MergeBamAlignment                            \u001b[36mMerge alignment data from a SAM or BAM with data in an unmapped BAM file.  \u001b[0m\r\n",
      "\u001b[32m    MergeSamFiles                                \u001b[36mMerges multiple SAM and/or BAM files into a single file.  \u001b[0m\r\n",
      "\u001b[32m    PositionBasedDownsampleSam                   \u001b[36mDownsample a SAM or BAM file to retain a subset of the reads based on the reads location in each tile in the flowcell.\u001b[0m\r\n",
      "\u001b[32m    ReorderSam                                   \u001b[36mReorders reads in a SAM or BAM file to match ordering in reference\u001b[0m\r\n",
      "\u001b[32m    ReplaceSamHeader                             \u001b[36mReplaces the SAMFileHeader in a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    RevertOriginalBaseQualitiesAndAddMateCigar   \u001b[36mReverts the original base qualities and adds the mate cigar tag to read-group BAMs\u001b[0m\r\n",
      "\u001b[32m    RevertSam                                    \u001b[36mReverts SAM or BAM files to a previous state.  \u001b[0m\r\n",
      "\u001b[32m    SamFormatConverter                           \u001b[36mConvert a BAM file to a SAM file, or a SAM to a BAM\u001b[0m\r\n",
      "\u001b[32m    SamToFastq                                   \u001b[36mConverts a SAM or BAM file to FASTQ.  \u001b[0m\r\n",
      "\u001b[32m    SetNmAndUqTags                               \u001b[36mDEPRECATED: Use SetNmMdAndUqTags instead.\u001b[0m\r\n",
      "\u001b[32m    SetNmMdAndUqTags                             \u001b[36mFixes the NM, MD, and UQ tags in a SAM file.  \u001b[0m\r\n",
      "\u001b[32m    SortSam                                      \u001b[36mSorts a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    SplitSamByLibrary                            \u001b[36mSplits a SAM or BAM file into individual files by library\u001b[0m\r\n",
      "\u001b[32m    ValidateSamFile                              \u001b[36mValidates a SAM or BAM file.  \u001b[0m\r\n",
      "\u001b[32m    ViewSam                                      \u001b[36mPrints a SAM or BAM file to the screen\u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mUnit Testing:                                    Unit testing                                 \u001b[0m\r\n",
      "\u001b[32m    SimpleMarkDuplicatesWithMateCigar            \u001b[36mExamines aligned records in the supplied SAM or BAM file to locate duplicate molecules.\u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\u001b[0m\u001b[31mVCF/BCF:                                         Tools for manipulating VCF, BCF, or related data.\u001b[0m\r\n",
      "\u001b[32m    FilterVcf                                    \u001b[36mHard filters a VCF.\u001b[0m\r\n",
      "\u001b[32m    FindMendelianViolations                      \u001b[36mFinds mendelian violations of all types within a VCF\u001b[0m\r\n",
      "\u001b[32m    FixVcfHeader                                 \u001b[36mReplaces or fixes a VCF header.\u001b[0m\r\n",
      "\u001b[32m    GatherVcfs                                   \u001b[36mGathers multiple VCF files from a scatter operation into a single VCF file\u001b[0m\r\n",
      "\u001b[32m    GenotypeConcordance                          \u001b[36mEvaluate genotype concordance between callsets.\u001b[0m\r\n",
      "\u001b[32m    LiftoverVcf                                  \u001b[36mLifts over a VCF file from one reference build to another.  \u001b[0m\r\n",
      "\u001b[32m    MakeSitesOnlyVcf                             \u001b[36mCreates a VCF bereft of genotype information from an input VCF or BCF\u001b[0m\r\n",
      "\u001b[32m    MergeVcfs                                    \u001b[36mMerges multiple VCF or BCF files into one VCF file or BCF\u001b[0m\r\n",
      "\u001b[32m    RenameSampleInVcf                            \u001b[36mRenames a sample within a VCF or BCF.  \u001b[0m\r\n",
      "\u001b[32m    SortVcf                                      \u001b[36mSorts one or more VCF files.  \u001b[0m\r\n",
      "\u001b[32m    SplitVcfs                                    \u001b[36mSplits SNPs and INDELs into separate files.  \u001b[0m\r\n",
      "\u001b[32m    UpdateVcfSequenceDictionary                  \u001b[36mTakes a VCF and a second file that contains a sequence dictionary and updates the VCF with the new sequence dictionary.\u001b[0m\r\n",
      "\u001b[32m    VcfFormatConverter                           \u001b[36mConverts VCF to BCF or BCF to VCF.  \u001b[0m\r\n",
      "\u001b[32m    VcfToIntervalList                            \u001b[36mConverts a VCF or BCF file to a Picard Interval List.\u001b[0m\r\n",
      "\r\n",
      "\u001b[37m--------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!java -jar /mnt/hgfs/Ubuntu_Shared/Optimization_Run1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar -h"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "This is perl 5, version 22, subversion 2 (v5.22.2) built for x86_64-linux-gnu-thread-multi\r\n",
      "(with 67 registered patches, see perl -V for more detail)\r\n",
      "\r\n",
      "Copyright 1987-2015, Larry Wall\r\n",
      "\r\n",
      "Perl may be copied only under the terms of either the Artistic License or the\r\n",
      "GNU General Public License, which may be found in the Perl 5 source kit.\r\n",
      "\r\n",
      "Complete documentation for Perl, including FAQ lists, should be found on\r\n",
      "this system using \"man perl\" or \"perldoc perl\".  If you have access to the\r\n",
      "Internet, point your browser at http://www.perl.org/, the Perl Home Page.\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!perl -verson"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data\n"
     ]
    }
   ],
   "source": [
    "cd Data/Unzipped_Data/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bowtie2-build 233_Amplicons.fa GATKRefTrim\n",
      "Building a SMALL index\n",
      "bwa index -p GATKRefTrim 233_Amplicons.fa\n",
      "[bwa_index] Pack FASTA... 0.00 sec\n",
      "[bwa_index] Construct BWT for the packed sequence...\n",
      "[bwa_index] 0.00 seconds elapse.\n",
      "[bwa_index] Update BWT... 0.00 sec\n",
      "[bwa_index] Pack forward-only FASTA... 0.00 sec\n",
      "[bwa_index] Construct SA from BWT and Occ... 0.00 sec\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa index -p GATKRefTrim 233_Amplicons.fa\n",
      "[main] Real time: 0.040 sec; CPU: 0.012 sec\n",
      "samtools faidx 233_Amplicons.fa\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar CreateSequenceDictionary REFERENCE=233_Amplicons.fa OUTPUT=233_Amplicons.dict\n",
      "[Thu Oct 19 15:24:22 PDT 2017] picard.sam.CreateSequenceDictionary REFERENCE=233_Amplicons.fa OUTPUT=233_Amplicons.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:24:22 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "[Thu Oct 19 15:24:23 PDT 2017] picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.\n",
      "Runtime.totalMemory()=331874304\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19003_S9_L001_R1_001 --rg SM:2010AC19003_S9_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19003_S9_L001_R1_001.fastq -S 2010AC19003_S9_L001_R1_001.sam 2> 2010AC19003_S9_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19003_S9_L001_R1_001.sam OUTPUT=2010AC19003_S9_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:25:42 PDT 2017] picard.sam.SortSam INPUT=2010AC19003_S9_L001_R1_001.sam OUTPUT=2010AC19003_S9_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:25:42 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:26:10\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:26:20 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.63 minutes.\n",
      "Runtime.totalMemory()=2143289344\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19003_S9_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19003_S9_L001_R1_001.bam.list\n",
      "INFO  15:26:22,073 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:22,075 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:26:22,076 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:26:22,076 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:26:22,076 HelpFormatter - [Thu Oct 19 15:26:22 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:26:22,076 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:26:22,080 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19003_S9_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19003_S9_L001_R1_001.bam.list \n",
      "INFO  15:26:22,083 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:26:22,083 HelpFormatter - Date/Time: 2017/10/19 15:26:22 \n",
      "INFO  15:26:22,084 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:22,084 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:22,089 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:26:22,232 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:26:22,239 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:26:22,277 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:26:22,519 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:26:22,544 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:26:22,544 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:26:22,545 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:26:22,545 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:26:27,209 ProgressMeter -            done     16100.0     4.0 s       4.8 m       99.5%     4.0 s       0.0 s \n",
      "INFO  15:26:27,210 ProgressMeter - Total runtime 4.67 secs, 0.08 min, 0.00 hours \n",
      "INFO  15:26:27,210 MicroScheduler - 310 reads were filtered out during the traversal out of approximately 699489 total reads (0.04%) \n",
      "INFO  15:26:27,210 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:26:27,211 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:26:27,211 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:26:27,211 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:26:27,211 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:26:27,212 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:26:27,212 MicroScheduler -   -> 310 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:26:27,212 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:26:27,212 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:26:27,213 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R1_001.bam -targetIntervals 2010AC19003_S9_L001_R1_001.bam.list -o 2010AC19003_S9_L001_R1_001.realigned.bam\n",
      "INFO  15:26:28,999 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:29,002 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:26:29,002 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:26:29,002 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:26:29,002 HelpFormatter - [Thu Oct 19 15:26:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:26:29,002 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:26:29,006 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R1_001.bam -targetIntervals 2010AC19003_S9_L001_R1_001.bam.list -o 2010AC19003_S9_L001_R1_001.realigned.bam \n",
      "INFO  15:26:29,008 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:26:29,009 HelpFormatter - Date/Time: 2017/10/19 15:26:28 \n",
      "INFO  15:26:29,009 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:29,010 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:26:29,100 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:26:29,192 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:26:29,198 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:26:29,238 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:26:29,398 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:26:29,402 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:26:29,403 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:26:29,403 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:26:29,403 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:26:29,429 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:26:29,430 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:26:31,130 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-54 because there are too many reads. \n",
      "INFO  15:26:37,498 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-77 because there are too many reads. \n",
      "INFO  15:26:47,867 ProgressMeter -            done    931392.0    18.0 s      19.0 s      100.0%    18.0 s       0.0 s \n",
      "INFO  15:26:47,867 ProgressMeter - Total runtime 18.46 secs, 0.31 min, 0.01 hours \n",
      "INFO  15:26:47,870 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 931392 total reads (0.00%) \n",
      "INFO  15:26:47,870 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:26:47,870 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19003_S9_L001_R2_001 --rg SM:2010AC19003_S9_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19003_S9_L001_R2_001.fastq -S 2010AC19003_S9_L001_R2_001.sam 2> 2010AC19003_S9_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19003_S9_L001_R2_001.sam OUTPUT=2010AC19003_S9_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:27:59 PDT 2017] picard.sam.SortSam INPUT=2010AC19003_S9_L001_R2_001.sam OUTPUT=2010AC19003_S9_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:27:59 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:28:11\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:28:20 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.36 minutes.\n",
      "Runtime.totalMemory()=1771044864\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19003_S9_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19003_S9_L001_R2_001.bam.list\n",
      "INFO  15:28:22,433 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:22,435 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:28:22,435 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:28:22,435 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:28:22,436 HelpFormatter - [Thu Oct 19 15:28:22 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:28:22,436 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:28:22,439 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19003_S9_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19003_S9_L001_R2_001.bam.list \n",
      "INFO  15:28:22,442 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:28:22,442 HelpFormatter - Date/Time: 2017/10/19 15:28:22 \n",
      "INFO  15:28:22,442 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:22,443 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:22,448 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:28:22,582 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:28:22,588 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:28:22,632 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:28:22,786 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:28:22,816 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:28:22,816 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:28:22,817 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:28:22,817 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:28:27,112 ProgressMeter -            done     16100.0     4.0 s       4.4 m       99.5%     4.0 s       0.0 s \n",
      "INFO  15:28:27,112 ProgressMeter - Total runtime 4.30 secs, 0.07 min, 0.00 hours \n",
      "INFO  15:28:27,113 MicroScheduler - 354 reads were filtered out during the traversal out of approximately 706820 total reads (0.05%) \n",
      "INFO  15:28:27,113 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:28:27,113 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:28:27,114 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:28:27,114 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:28:27,114 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:28:27,114 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:28:27,114 MicroScheduler -   -> 354 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:28:27,115 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:28:27,115 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:28:27,115 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R2_001.bam -targetIntervals 2010AC19003_S9_L001_R2_001.bam.list -o 2010AC19003_S9_L001_R2_001.realigned.bam\n",
      "INFO  15:28:28,855 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:28,857 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:28:28,858 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:28:28,858 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:28:28,858 HelpFormatter - [Thu Oct 19 15:28:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:28:28,858 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:28:28,861 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R2_001.bam -targetIntervals 2010AC19003_S9_L001_R2_001.bam.list -o 2010AC19003_S9_L001_R2_001.realigned.bam \n",
      "INFO  15:28:28,864 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:28:28,865 HelpFormatter - Date/Time: 2017/10/19 15:28:28 \n",
      "INFO  15:28:28,865 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:28,865 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:28:28,942 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:28:29,033 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:28:29,040 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:28:29,095 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:28:29,253 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:28:29,257 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:28:29,257 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:28:29,258 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:28:29,258 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:28:29,282 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:28:29,284 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:28:30,862 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-73 because there are too many reads. \n",
      "INFO  15:28:36,721 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  15:28:49,868 ProgressMeter -            done    931392.0    20.0 s      22.0 s      100.0%    20.0 s       0.0 s \n",
      "INFO  15:28:49,868 ProgressMeter - Total runtime 20.61 secs, 0.34 min, 0.01 hours \n",
      "INFO  15:28:49,871 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 931392 total reads (0.00%) \n",
      "INFO  15:28:49,872 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:28:49,872 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19006_S10_L001_R1_001 --rg SM:2010AC19006_S10_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19006_S10_L001_R1_001.fastq -S 2010AC19006_S10_L001_R1_001.sam 2> 2010AC19006_S10_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19006_S10_L001_R1_001.sam OUTPUT=2010AC19006_S10_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:29:09 PDT 2017] picard.sam.SortSam INPUT=2010AC19006_S10_L001_R1_001.sam OUTPUT=2010AC19006_S10_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:29:09 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:29:12\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:29:14 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.08 minutes.\n",
      "Runtime.totalMemory()=988282880\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19006_S10_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19006_S10_L001_R1_001.bam.list\n",
      "INFO  15:29:16,559 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:16,562 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:29:16,562 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:29:16,562 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:29:16,562 HelpFormatter - [Thu Oct 19 15:29:16 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:29:16,562 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:29:16,565 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19006_S10_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19006_S10_L001_R1_001.bam.list \n",
      "INFO  15:29:16,569 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:29:16,569 HelpFormatter - Date/Time: 2017/10/19 15:29:16 \n",
      "INFO  15:29:16,569 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:16,569 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:16,574 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:29:16,713 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:29:16,719 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:29:16,758 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:29:16,933 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:29:16,959 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:29:16,959 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:29:16,960 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:29:16,960 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:29:19,294 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:29:19,295 ProgressMeter - Total runtime 2.34 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:29:19,295 MicroScheduler - 7 reads were filtered out during the traversal out of approximately 177639 total reads (0.00%) \n",
      "INFO  15:29:19,295 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:29:19,296 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:29:19,296 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:29:19,296 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:29:19,296 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:29:19,297 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:29:19,297 MicroScheduler -   -> 7 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:29:19,297 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:29:19,297 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:29:19,298 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R1_001.bam -targetIntervals 2010AC19006_S10_L001_R1_001.bam.list -o 2010AC19006_S10_L001_R1_001.realigned.bam\n",
      "INFO  15:29:21,020 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:21,023 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:29:21,023 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:29:21,023 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:29:21,023 HelpFormatter - [Thu Oct 19 15:29:20 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:29:21,024 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:29:21,027 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R1_001.bam -targetIntervals 2010AC19006_S10_L001_R1_001.bam.list -o 2010AC19006_S10_L001_R1_001.realigned.bam \n",
      "INFO  15:29:21,030 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:29:21,030 HelpFormatter - Date/Time: 2017/10/19 15:29:21 \n",
      "INFO  15:29:21,030 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:21,031 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:21,110 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:29:21,199 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:29:21,205 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:29:21,245 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:29:21,398 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:29:21,402 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:29:21,402 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:29:21,403 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:29:21,403 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:29:21,427 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:29:21,428 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:29:21,940 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-59 because there are too many reads. \n",
      "INFO  15:29:23,635 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:10-68 because there are too many reads. \n",
      "INFO  15:29:26,886 ProgressMeter -            done    264331.0     5.0 s      20.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  15:29:26,887 ProgressMeter - Total runtime 5.48 secs, 0.09 min, 0.00 hours \n",
      "INFO  15:29:26,890 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 264331 total reads (0.00%) \n",
      "INFO  15:29:26,890 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:29:26,890 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19006_S10_L001_R2_001 --rg SM:2010AC19006_S10_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19006_S10_L001_R2_001.fastq -S 2010AC19006_S10_L001_R2_001.sam 2> 2010AC19006_S10_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19006_S10_L001_R2_001.sam OUTPUT=2010AC19006_S10_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:29:45 PDT 2017] picard.sam.SortSam INPUT=2010AC19006_S10_L001_R2_001.sam OUTPUT=2010AC19006_S10_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:29:45 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:29:47\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:29:50 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=990380032\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19006_S10_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19006_S10_L001_R2_001.bam.list\n",
      "INFO  15:29:52,193 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:52,195 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:29:52,195 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:29:52,195 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:29:52,195 HelpFormatter - [Thu Oct 19 15:29:52 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:29:52,196 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:29:52,199 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19006_S10_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19006_S10_L001_R2_001.bam.list \n",
      "INFO  15:29:52,202 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:29:52,202 HelpFormatter - Date/Time: 2017/10/19 15:29:52 \n",
      "INFO  15:29:52,203 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:52,203 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:52,208 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:29:52,346 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:29:52,352 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:29:52,392 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:29:52,569 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:29:52,595 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:29:52,596 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:29:52,596 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:29:52,596 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:29:54,862 ProgressMeter -            done     16100.0     2.0 s       2.3 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:29:54,862 ProgressMeter - Total runtime 2.27 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:29:54,863 MicroScheduler - 81 reads were filtered out during the traversal out of approximately 181967 total reads (0.04%) \n",
      "INFO  15:29:54,863 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:29:54,863 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:29:54,864 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:29:54,864 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:29:54,864 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:29:54,864 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:29:54,864 MicroScheduler -   -> 81 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:29:54,865 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:29:54,865 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:29:54,865 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R2_001.bam -targetIntervals 2010AC19006_S10_L001_R2_001.bam.list -o 2010AC19006_S10_L001_R2_001.realigned.bam\n",
      "INFO  15:29:56,555 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:56,557 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:29:56,557 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:29:56,557 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:29:56,557 HelpFormatter - [Thu Oct 19 15:29:56 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:29:56,557 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:29:56,560 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R2_001.bam -targetIntervals 2010AC19006_S10_L001_R2_001.bam.list -o 2010AC19006_S10_L001_R2_001.realigned.bam \n",
      "INFO  15:29:56,563 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:29:56,563 HelpFormatter - Date/Time: 2017/10/19 15:29:56 \n",
      "INFO  15:29:56,564 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:56,564 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:29:56,641 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:29:56,733 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:29:56,740 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:29:56,782 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:29:56,938 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:29:56,941 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:29:56,942 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:29:56,942 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:29:56,942 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:29:56,967 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:29:56,968 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:29:57,563 IndelRealigner - Not attempting realignment in interval One_RAD56448:14-57 because there are too many reads. \n",
      "INFO  15:29:59,429 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:21-75 because there are too many reads. \n",
      "INFO  15:30:03,018 ProgressMeter -            done    264331.0     6.0 s      22.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  15:30:03,019 ProgressMeter - Total runtime 6.08 secs, 0.10 min, 0.00 hours \n",
      "INFO  15:30:03,022 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 264331 total reads (0.00%) \n",
      "INFO  15:30:03,022 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:30:03,022 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19008_S11_L001_R1_001 --rg SM:2010AC19008_S11_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19008_S11_L001_R1_001.fastq -S 2010AC19008_S11_L001_R1_001.sam 2> 2010AC19008_S11_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19008_S11_L001_R1_001.sam OUTPUT=2010AC19008_S11_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:32:06 PDT 2017] picard.sam.SortSam INPUT=2010AC19008_S11_L001_R1_001.sam OUTPUT=2010AC19008_S11_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:32:06 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:32:35\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:32:45 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.66 minutes.\n",
      "Runtime.totalMemory()=2153775104\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19008_S11_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19008_S11_L001_R1_001.bam.list\n",
      "INFO  15:32:47,460 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:47,462 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:32:47,463 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:32:47,463 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:32:47,463 HelpFormatter - [Thu Oct 19 15:32:47 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:32:47,463 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:32:47,467 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19008_S11_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19008_S11_L001_R1_001.bam.list \n",
      "INFO  15:32:47,469 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:32:47,470 HelpFormatter - Date/Time: 2017/10/19 15:32:47 \n",
      "INFO  15:32:47,470 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:47,470 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:47,475 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:32:47,644 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:32:47,650 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:32:47,690 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:32:47,860 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:32:47,885 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:32:47,886 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:32:47,886 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:32:47,886 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:32:53,777 ProgressMeter -            done     16100.0     5.0 s       6.1 m       99.5%     5.0 s       0.0 s \n",
      "INFO  15:32:53,777 ProgressMeter - Total runtime 5.89 secs, 0.10 min, 0.00 hours \n",
      "INFO  15:32:53,778 MicroScheduler - 114 reads were filtered out during the traversal out of approximately 998690 total reads (0.01%) \n",
      "INFO  15:32:53,778 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:32:53,778 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:32:53,778 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:32:53,779 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:32:53,779 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:32:53,779 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:32:53,779 MicroScheduler -   -> 114 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:32:53,779 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:32:53,780 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:32:53,780 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R1_001.bam -targetIntervals 2010AC19008_S11_L001_R1_001.bam.list -o 2010AC19008_S11_L001_R1_001.realigned.bam\n",
      "INFO  15:32:55,536 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:55,538 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:32:55,538 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:32:55,539 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:32:55,539 HelpFormatter - [Thu Oct 19 15:32:55 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:32:55,539 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:32:55,542 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R1_001.bam -targetIntervals 2010AC19008_S11_L001_R1_001.bam.list -o 2010AC19008_S11_L001_R1_001.realigned.bam \n",
      "INFO  15:32:55,545 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:32:55,546 HelpFormatter - Date/Time: 2017/10/19 15:32:55 \n",
      "INFO  15:32:55,546 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:55,546 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:32:55,623 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:32:55,710 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:32:55,715 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:32:55,771 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.06 \n",
      "INFO  15:32:55,938 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:32:55,942 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:32:55,943 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:32:55,943 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:32:55,943 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:32:55,970 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:32:55,971 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:32:57,511 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-82 because there are too many reads. \n",
      "INFO  15:33:05,534 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-78 because there are too many reads. \n",
      "INFO  15:33:27,906 ProgressMeter -            done   1301783.0    31.0 s      24.0 s      100.0%    31.0 s       0.0 s \n",
      "INFO  15:33:27,906 ProgressMeter - Total runtime 31.96 secs, 0.53 min, 0.01 hours \n",
      "INFO  15:33:27,909 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1301783 total reads (0.00%) \n",
      "INFO  15:33:27,909 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:33:27,910 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19008_S11_L001_R2_001 --rg SM:2010AC19008_S11_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19008_S11_L001_R2_001.fastq -S 2010AC19008_S11_L001_R2_001.sam 2> 2010AC19008_S11_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19008_S11_L001_R2_001.sam OUTPUT=2010AC19008_S11_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:35:10 PDT 2017] picard.sam.SortSam INPUT=2010AC19008_S11_L001_R2_001.sam OUTPUT=2010AC19008_S11_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:35:10 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:35:23\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:35:34 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.41 minutes.\n",
      "Runtime.totalMemory()=2105540608\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19008_S11_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19008_S11_L001_R2_001.bam.list\n",
      "INFO  15:35:36,598 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:36,600 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:35:36,600 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:35:36,601 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:35:36,601 HelpFormatter - [Thu Oct 19 15:35:36 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:35:36,601 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:35:36,604 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19008_S11_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19008_S11_L001_R2_001.bam.list \n",
      "INFO  15:35:36,607 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:35:36,607 HelpFormatter - Date/Time: 2017/10/19 15:35:36 \n",
      "INFO  15:35:36,607 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:36,608 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:36,612 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:35:36,752 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:35:36,758 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:35:36,808 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:35:36,975 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:35:37,001 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:35:37,002 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:35:37,002 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:35:37,002 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:35:42,986 ProgressMeter -            done     16100.0     5.0 s       6.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  15:35:42,986 ProgressMeter - Total runtime 5.98 secs, 0.10 min, 0.00 hours \n",
      "INFO  15:35:42,987 MicroScheduler - 521 reads were filtered out during the traversal out of approximately 1004290 total reads (0.05%) \n",
      "INFO  15:35:42,987 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:35:42,987 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:35:42,988 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:35:42,988 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:35:42,988 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:35:42,988 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:35:42,988 MicroScheduler -   -> 521 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:35:42,989 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:35:42,989 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:35:42,989 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R2_001.bam -targetIntervals 2010AC19008_S11_L001_R2_001.bam.list -o 2010AC19008_S11_L001_R2_001.realigned.bam\n",
      "INFO  15:35:44,703 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:44,705 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:35:44,705 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:35:44,705 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:35:44,706 HelpFormatter - [Thu Oct 19 15:35:44 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:35:44,706 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:35:44,709 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R2_001.bam -targetIntervals 2010AC19008_S11_L001_R2_001.bam.list -o 2010AC19008_S11_L001_R2_001.realigned.bam \n",
      "INFO  15:35:44,712 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:35:44,712 HelpFormatter - Date/Time: 2017/10/19 15:35:44 \n",
      "INFO  15:35:44,712 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:44,713 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:35:44,787 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:35:44,885 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:35:44,891 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:35:44,939 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:35:45,099 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:35:45,103 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:35:45,103 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:35:45,104 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:35:45,104 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:35:45,131 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:35:45,132 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:35:46,783 IndelRealigner - Not attempting realignment in interval One_RAD56448:26-68 because there are too many reads. \n",
      "INFO  15:35:53,066 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-71 because there are too many reads. \n",
      "INFO  15:36:13,399 ProgressMeter -            done   1301783.0    28.0 s      21.0 s      100.0%    28.0 s       0.0 s \n",
      "INFO  15:36:13,399 ProgressMeter - Total runtime 28.30 secs, 0.47 min, 0.01 hours \n",
      "INFO  15:36:13,402 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1301783 total reads (0.00%) \n",
      "INFO  15:36:13,402 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:36:13,403 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19015_S12_L001_R1_001 --rg SM:2010AC19015_S12_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19015_S12_L001_R1_001.fastq -S 2010AC19015_S12_L001_R1_001.sam 2> 2010AC19015_S12_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19015_S12_L001_R1_001.sam OUTPUT=2010AC19015_S12_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:36:32 PDT 2017] picard.sam.SortSam INPUT=2010AC19015_S12_L001_R1_001.sam OUTPUT=2010AC19015_S12_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:36:32 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:36:34\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:36:37 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=978845696\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19015_S12_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19015_S12_L001_R1_001.bam.list\n",
      "INFO  15:36:39,115 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:39,117 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:36:39,117 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:36:39,118 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:36:39,118 HelpFormatter - [Thu Oct 19 15:36:39 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:36:39,118 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:36:39,121 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19015_S12_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19015_S12_L001_R1_001.bam.list \n",
      "INFO  15:36:39,124 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:36:39,125 HelpFormatter - Date/Time: 2017/10/19 15:36:39 \n",
      "INFO  15:36:39,125 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:39,125 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:39,130 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:36:39,278 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:36:39,284 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:36:39,324 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:36:39,492 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:36:39,519 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:36:39,519 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:36:39,520 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:36:39,520 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:36:41,812 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:36:41,813 ProgressMeter - Total runtime 2.29 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:36:41,814 MicroScheduler - 25 reads were filtered out during the traversal out of approximately 175816 total reads (0.01%) \n",
      "INFO  15:36:41,814 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:36:41,814 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:36:41,814 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:36:41,814 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:36:41,815 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:36:41,815 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:36:41,815 MicroScheduler -   -> 25 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:36:41,815 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:36:41,816 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:36:41,816 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R1_001.bam -targetIntervals 2010AC19015_S12_L001_R1_001.bam.list -o 2010AC19015_S12_L001_R1_001.realigned.bam\n",
      "INFO  15:36:43,573 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:43,575 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:36:43,575 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:36:43,576 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:36:43,576 HelpFormatter - [Thu Oct 19 15:36:43 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:36:43,576 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:36:43,579 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R1_001.bam -targetIntervals 2010AC19015_S12_L001_R1_001.bam.list -o 2010AC19015_S12_L001_R1_001.realigned.bam \n",
      "INFO  15:36:43,582 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:36:43,583 HelpFormatter - Date/Time: 2017/10/19 15:36:43 \n",
      "INFO  15:36:43,583 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:43,583 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:36:43,663 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:36:43,763 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:36:43,769 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:36:43,823 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:36:43,987 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:36:43,991 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:36:43,991 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:36:43,992 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:36:43,992 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:36:44,018 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:36:44,019 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:36:44,689 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-68 because there are too many reads. \n",
      "INFO  15:36:46,404 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:7-72 because there are too many reads. \n",
      "INFO  15:36:49,213 ProgressMeter -            done    252023.0     5.0 s      20.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  15:36:49,214 ProgressMeter - Total runtime 5.22 secs, 0.09 min, 0.00 hours \n",
      "INFO  15:36:49,217 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 252023 total reads (0.00%) \n",
      "INFO  15:36:49,218 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:36:49,218 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19015_S12_L001_R2_001 --rg SM:2010AC19015_S12_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19015_S12_L001_R2_001.fastq -S 2010AC19015_S12_L001_R2_001.sam 2> 2010AC19015_S12_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19015_S12_L001_R2_001.sam OUTPUT=2010AC19015_S12_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:37:07 PDT 2017] picard.sam.SortSam INPUT=2010AC19015_S12_L001_R2_001.sam OUTPUT=2010AC19015_S12_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:37:07 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:37:09\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:37:12 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=976748544\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19015_S12_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19015_S12_L001_R2_001.bam.list\n",
      "INFO  15:37:13,956 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:13,958 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:37:13,958 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:37:13,958 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:37:13,959 HelpFormatter - [Thu Oct 19 15:37:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:37:13,959 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:37:13,962 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19015_S12_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19015_S12_L001_R2_001.bam.list \n",
      "INFO  15:37:13,965 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:37:13,966 HelpFormatter - Date/Time: 2017/10/19 15:37:13 \n",
      "INFO  15:37:13,966 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:13,966 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:13,971 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:37:14,114 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:37:14,120 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:37:14,162 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:37:14,323 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:37:14,349 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:37:14,350 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:37:14,350 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:37:14,350 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:37:16,704 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:37:16,704 ProgressMeter - Total runtime 2.35 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:37:16,705 MicroScheduler - 73 reads were filtered out during the traversal out of approximately 180427 total reads (0.04%) \n",
      "INFO  15:37:16,705 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:37:16,705 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:37:16,706 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:37:16,706 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:37:16,706 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:37:16,706 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:37:16,707 MicroScheduler -   -> 73 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:37:16,707 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:37:16,707 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:37:16,707 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R2_001.bam -targetIntervals 2010AC19015_S12_L001_R2_001.bam.list -o 2010AC19015_S12_L001_R2_001.realigned.bam\n",
      "INFO  15:37:18,447 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:18,449 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:37:18,450 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:37:18,450 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:37:18,450 HelpFormatter - [Thu Oct 19 15:37:18 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:37:18,450 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:37:18,453 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R2_001.bam -targetIntervals 2010AC19015_S12_L001_R2_001.bam.list -o 2010AC19015_S12_L001_R2_001.realigned.bam \n",
      "INFO  15:37:18,456 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:37:18,456 HelpFormatter - Date/Time: 2017/10/19 15:37:18 \n",
      "INFO  15:37:18,457 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:18,457 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:37:18,539 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:37:18,635 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:37:18,641 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:37:18,692 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:37:18,851 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:37:18,855 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:37:18,855 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:37:18,856 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:37:18,856 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:37:18,888 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:37:18,889 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:37:19,466 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-81 because there are too many reads. \n",
      "INFO  15:37:21,104 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:16-69 because there are too many reads. \n",
      "INFO  15:37:24,203 ProgressMeter -            done    252023.0     5.0 s      21.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  15:37:24,204 ProgressMeter - Total runtime 5.35 secs, 0.09 min, 0.00 hours \n",
      "INFO  15:37:24,206 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 252023 total reads (0.00%) \n",
      "INFO  15:37:24,207 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:37:24,207 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19041_S17_L001_R1_001 --rg SM:2010AC19041_S17_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19041_S17_L001_R1_001.fastq -S 2010AC19041_S17_L001_R1_001.sam 2> 2010AC19041_S17_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19041_S17_L001_R1_001.sam OUTPUT=2010AC19041_S17_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:37:53 PDT 2017] picard.sam.SortSam INPUT=2010AC19041_S17_L001_R1_001.sam OUTPUT=2010AC19041_S17_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:37:53 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:37:55\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:37:58 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1006108672\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19041_S17_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19041_S17_L001_R1_001.bam.list\n",
      "INFO  15:38:00,448 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:00,450 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:38:00,450 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:38:00,450 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:38:00,451 HelpFormatter - [Thu Oct 19 15:38:00 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:38:00,451 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:38:00,454 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19041_S17_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19041_S17_L001_R1_001.bam.list \n",
      "INFO  15:38:00,457 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:38:00,457 HelpFormatter - Date/Time: 2017/10/19 15:38:00 \n",
      "INFO  15:38:00,458 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:00,458 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:00,463 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:38:00,609 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:38:00,615 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:38:00,655 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:38:00,823 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:38:00,853 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:38:00,853 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:38:00,854 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:38:00,854 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:38:03,063 ProgressMeter -            done     16100.0     2.0 s       2.3 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:38:03,064 ProgressMeter - Total runtime 2.21 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:38:03,064 MicroScheduler - 10 reads were filtered out during the traversal out of approximately 220971 total reads (0.00%) \n",
      "INFO  15:38:03,065 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:38:03,065 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:38:03,065 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:38:03,066 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:38:03,066 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:38:03,066 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:38:03,066 MicroScheduler -   -> 10 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:38:03,067 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:38:03,067 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:38:03,067 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R1_001.bam -targetIntervals 2010AC19041_S17_L001_R1_001.bam.list -o 2010AC19041_S17_L001_R1_001.realigned.bam\n",
      "INFO  15:38:04,827 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:04,829 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:38:04,829 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:38:04,830 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:38:04,830 HelpFormatter - [Thu Oct 19 15:38:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:38:04,830 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:38:04,833 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R1_001.bam -targetIntervals 2010AC19041_S17_L001_R1_001.bam.list -o 2010AC19041_S17_L001_R1_001.realigned.bam \n",
      "INFO  15:38:04,836 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:38:04,837 HelpFormatter - Date/Time: 2017/10/19 15:38:04 \n",
      "INFO  15:38:04,837 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:04,837 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:04,915 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:38:05,009 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:38:05,015 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:38:05,064 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:38:05,229 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:38:05,233 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:38:05,233 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:38:05,233 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:38:05,233 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:38:05,277 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:38:05,279 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:38:10,953 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  15:38:14,289 ProgressMeter -            done    288358.0     9.0 s      31.0 s      100.0%     9.0 s       0.0 s \n",
      "INFO  15:38:14,289 ProgressMeter - Total runtime 9.06 secs, 0.15 min, 0.00 hours \n",
      "INFO  15:38:14,292 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 288358 total reads (0.00%) \n",
      "INFO  15:38:14,293 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:38:14,293 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19041_S17_L001_R2_001 --rg SM:2010AC19041_S17_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19041_S17_L001_R2_001.fastq -S 2010AC19041_S17_L001_R2_001.sam 2> 2010AC19041_S17_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19041_S17_L001_R2_001.sam OUTPUT=2010AC19041_S17_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:38:38 PDT 2017] picard.sam.SortSam INPUT=2010AC19041_S17_L001_R2_001.sam OUTPUT=2010AC19041_S17_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:38:38 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:38:40\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:38:43 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1010827264\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19041_S17_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19041_S17_L001_R2_001.bam.list\n",
      "INFO  15:38:45,418 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:45,420 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:38:45,421 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:38:45,421 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:38:45,421 HelpFormatter - [Thu Oct 19 15:38:45 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:38:45,421 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:38:45,424 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19041_S17_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19041_S17_L001_R2_001.bam.list \n",
      "INFO  15:38:45,428 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:38:45,428 HelpFormatter - Date/Time: 2017/10/19 15:38:45 \n",
      "INFO  15:38:45,429 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:45,429 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:45,434 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:38:45,573 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:38:45,579 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:38:45,621 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:38:45,779 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:38:45,804 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:38:45,805 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:38:45,805 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:38:45,805 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:38:48,139 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:38:48,140 ProgressMeter - Total runtime 2.34 secs, 0.04 min, 0.00 hours \n",
      "INFO  15:38:48,141 MicroScheduler - 117 reads were filtered out during the traversal out of approximately 219068 total reads (0.05%) \n",
      "INFO  15:38:48,141 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:38:48,141 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:38:48,141 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:38:48,142 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:38:48,142 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:38:48,142 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:38:48,142 MicroScheduler -   -> 117 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:38:48,143 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:38:48,143 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:38:48,143 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R2_001.bam -targetIntervals 2010AC19041_S17_L001_R2_001.bam.list -o 2010AC19041_S17_L001_R2_001.realigned.bam\n",
      "INFO  15:38:49,920 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:49,922 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:38:49,923 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:38:49,923 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:38:49,923 HelpFormatter - [Thu Oct 19 15:38:49 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:38:49,923 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:38:49,926 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R2_001.bam -targetIntervals 2010AC19041_S17_L001_R2_001.bam.list -o 2010AC19041_S17_L001_R2_001.realigned.bam \n",
      "INFO  15:38:49,929 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:38:49,930 HelpFormatter - Date/Time: 2017/10/19 15:38:49 \n",
      "INFO  15:38:49,930 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:49,930 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:38:50,013 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:38:50,102 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:38:50,108 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:38:50,149 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:38:50,309 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:38:50,313 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:38:50,313 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:38:50,314 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:38:50,314 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:38:50,339 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:38:50,340 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:38:57,629 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-77 because there are too many reads. \n",
      "INFO  15:39:01,503 ProgressMeter -            done    288358.0    11.0 s      38.0 s      100.0%    11.0 s       0.0 s \n",
      "INFO  15:39:01,504 ProgressMeter - Total runtime 11.19 secs, 0.19 min, 0.00 hours \n",
      "INFO  15:39:01,506 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 288358 total reads (0.00%) \n",
      "INFO  15:39:01,507 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:39:01,507 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19047_S13_L001_R1_001 --rg SM:2010AC19047_S13_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19047_S13_L001_R1_001.fastq -S 2010AC19047_S13_L001_R1_001.sam 2> 2010AC19047_S13_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19047_S13_L001_R1_001.sam OUTPUT=2010AC19047_S13_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:41:01 PDT 2017] picard.sam.SortSam INPUT=2010AC19047_S13_L001_R1_001.sam OUTPUT=2010AC19047_S13_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:41:01 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:41:30\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:41:43 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.70 minutes.\n",
      "Runtime.totalMemory()=2167930880\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19047_S13_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19047_S13_L001_R1_001.bam.list\n",
      "INFO  15:41:44,710 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:44,712 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:41:44,712 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:41:44,712 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:41:44,712 HelpFormatter - [Thu Oct 19 15:41:44 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:41:44,713 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:41:44,716 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19047_S13_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19047_S13_L001_R1_001.bam.list \n",
      "INFO  15:41:44,719 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:41:44,719 HelpFormatter - Date/Time: 2017/10/19 15:41:44 \n",
      "INFO  15:41:44,720 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:44,720 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:44,726 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:41:44,862 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:41:44,868 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:41:44,911 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:41:45,069 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:41:45,095 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:41:45,096 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:41:45,096 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:41:45,096 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:41:51,659 ProgressMeter -            done     16100.0     6.0 s       6.8 m       99.5%     6.0 s       0.0 s \n",
      "INFO  15:41:51,659 ProgressMeter - Total runtime 6.56 secs, 0.11 min, 0.00 hours \n",
      "INFO  15:41:51,660 MicroScheduler - 31 reads were filtered out during the traversal out of approximately 1047921 total reads (0.00%) \n",
      "INFO  15:41:51,661 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:41:51,661 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:41:51,661 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:41:51,662 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:41:51,663 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:41:51,663 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:41:51,663 MicroScheduler -   -> 31 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:41:51,663 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:41:51,664 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:41:51,664 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R1_001.bam -targetIntervals 2010AC19047_S13_L001_R1_001.bam.list -o 2010AC19047_S13_L001_R1_001.realigned.bam\n",
      "INFO  15:41:53,576 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:53,578 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:41:53,579 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:41:53,579 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:41:53,579 HelpFormatter - [Thu Oct 19 15:41:53 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:41:53,579 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:41:53,583 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R1_001.bam -targetIntervals 2010AC19047_S13_L001_R1_001.bam.list -o 2010AC19047_S13_L001_R1_001.realigned.bam \n",
      "INFO  15:41:53,586 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:41:53,587 HelpFormatter - Date/Time: 2017/10/19 15:41:53 \n",
      "INFO  15:41:53,587 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:53,587 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:41:53,693 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:41:53,797 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:41:53,804 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:41:53,851 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:41:54,014 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:41:54,017 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:41:54,018 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:41:54,018 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:41:54,019 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:41:54,063 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:41:54,064 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:41:55,896 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-76 because there are too many reads. \n",
      "INFO  15:42:06,264 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-75 because there are too many reads. \n",
      "INFO  15:42:13,634 IndelRealigner - Not attempting realignment in interval 59386_x1_FemaleLG_13_FemalePos_2.44_MaleLG_13_MalePos_34.91_duplicated_no:7-15 because there are too many reads. \n",
      "INFO  15:42:26,695 ProgressMeter -            done   1465140.0    32.0 s      22.0 s      100.0%    32.0 s       0.0 s \n",
      "INFO  15:42:26,696 ProgressMeter - Total runtime 32.68 secs, 0.54 min, 0.01 hours \n",
      "INFO  15:42:26,698 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1465140 total reads (0.00%) \n",
      "INFO  15:42:26,699 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:42:26,699 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19047_S13_L001_R2_001 --rg SM:2010AC19047_S13_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19047_S13_L001_R2_001.fastq -S 2010AC19047_S13_L001_R2_001.sam 2> 2010AC19047_S13_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19047_S13_L001_R2_001.sam OUTPUT=2010AC19047_S13_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:44:17 PDT 2017] picard.sam.SortSam INPUT=2010AC19047_S13_L001_R2_001.sam OUTPUT=2010AC19047_S13_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:44:17 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:44:35\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:44:50 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.54 minutes.\n",
      "Runtime.totalMemory()=2189950976\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19047_S13_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19047_S13_L001_R2_001.bam.list\n",
      "INFO  15:44:51,957 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:44:51,959 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:44:51,960 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:44:51,960 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:44:51,960 HelpFormatter - [Thu Oct 19 15:44:51 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:44:51,960 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:44:51,964 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19047_S13_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19047_S13_L001_R2_001.bam.list \n",
      "INFO  15:44:51,967 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:44:51,967 HelpFormatter - Date/Time: 2017/10/19 15:44:51 \n",
      "INFO  15:44:51,967 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:44:51,967 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:44:51,972 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:44:52,116 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:44:52,123 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:44:52,163 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:44:52,332 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:44:52,357 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:44:52,357 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:44:52,358 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:44:52,358 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:44:58,632 ProgressMeter -            done     16100.0     6.0 s       6.5 m       99.5%     6.0 s       0.0 s \n",
      "INFO  15:44:58,633 ProgressMeter - Total runtime 6.28 secs, 0.10 min, 0.00 hours \n",
      "INFO  15:44:58,633 MicroScheduler - 506 reads were filtered out during the traversal out of approximately 1052273 total reads (0.05%) \n",
      "INFO  15:44:58,634 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:44:58,634 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:44:58,634 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:44:58,634 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:44:58,635 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:44:58,635 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:44:58,635 MicroScheduler -   -> 506 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:44:58,636 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:44:58,636 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:44:58,636 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R2_001.bam -targetIntervals 2010AC19047_S13_L001_R2_001.bam.list -o 2010AC19047_S13_L001_R2_001.realigned.bam\n",
      "INFO  15:45:00,461 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:45:00,463 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:45:00,463 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:45:00,464 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:45:00,464 HelpFormatter - [Thu Oct 19 15:45:00 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:45:00,464 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:45:00,467 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R2_001.bam -targetIntervals 2010AC19047_S13_L001_R2_001.bam.list -o 2010AC19047_S13_L001_R2_001.realigned.bam \n",
      "INFO  15:45:00,470 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:45:00,470 HelpFormatter - Date/Time: 2017/10/19 15:45:00 \n",
      "INFO  15:45:00,471 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:45:00,471 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:45:00,553 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:45:00,645 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:45:00,651 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:45:00,699 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:45:00,858 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:45:00,862 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:45:00,862 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:45:00,862 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:45:00,863 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:45:00,896 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:45:00,897 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:45:03,132 IndelRealigner - Not attempting realignment in interval One_RAD56448:26-82 because there are too many reads. \n",
      "INFO  15:45:11,164 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:15-63 because there are too many reads. \n",
      "INFO  15:45:20,102 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:14-57 because there are too many reads. \n",
      "INFO  15:45:27,876 ProgressMeter -            done   1465140.0    27.0 s      18.0 s      100.0%    27.0 s       0.0 s \n",
      "INFO  15:45:27,877 ProgressMeter - Total runtime 27.01 secs, 0.45 min, 0.01 hours \n",
      "INFO  15:45:27,880 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1465140 total reads (0.00%) \n",
      "INFO  15:45:27,880 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:45:27,880 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19049_S14_L001_R1_001 --rg SM:2010AC19049_S14_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19049_S14_L001_R1_001.fastq -S 2010AC19049_S14_L001_R1_001.sam 2> 2010AC19049_S14_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19049_S14_L001_R1_001.sam OUTPUT=2010AC19049_S14_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:46:05 PDT 2017] picard.sam.SortSam INPUT=2010AC19049_S14_L001_R1_001.sam OUTPUT=2010AC19049_S14_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:46:05 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:46:09\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:46:13 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.12 minutes.\n",
      "Runtime.totalMemory()=1188036608\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19049_S14_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19049_S14_L001_R1_001.bam.list\n",
      "INFO  15:46:15,286 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:15,289 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:46:15,289 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:46:15,289 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:46:15,289 HelpFormatter - [Thu Oct 19 15:46:15 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:46:15,289 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:46:15,293 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19049_S14_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19049_S14_L001_R1_001.bam.list \n",
      "INFO  15:46:15,296 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:46:15,296 HelpFormatter - Date/Time: 2017/10/19 15:46:15 \n",
      "INFO  15:46:15,296 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:15,296 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:15,301 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:46:15,447 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:46:15,453 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:46:15,497 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:46:15,683 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:46:15,717 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:46:15,718 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:46:15,718 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:46:15,719 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:46:18,500 ProgressMeter -            done     16100.0     2.0 s       2.9 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:46:18,500 ProgressMeter - Total runtime 2.78 secs, 0.05 min, 0.00 hours \n",
      "INFO  15:46:18,501 MicroScheduler - 13 reads were filtered out during the traversal out of approximately 289723 total reads (0.00%) \n",
      "INFO  15:46:18,501 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:46:18,502 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:46:18,502 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:46:18,502 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:46:18,502 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:46:18,502 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:46:18,503 MicroScheduler -   -> 13 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:46:18,504 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:46:18,504 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:46:18,504 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R1_001.bam -targetIntervals 2010AC19049_S14_L001_R1_001.bam.list -o 2010AC19049_S14_L001_R1_001.realigned.bam\n",
      "INFO  15:46:20,416 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:20,418 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:46:20,418 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:46:20,418 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:46:20,419 HelpFormatter - [Thu Oct 19 15:46:20 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:46:20,419 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:46:20,422 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R1_001.bam -targetIntervals 2010AC19049_S14_L001_R1_001.bam.list -o 2010AC19049_S14_L001_R1_001.realigned.bam \n",
      "INFO  15:46:20,425 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:46:20,425 HelpFormatter - Date/Time: 2017/10/19 15:46:20 \n",
      "INFO  15:46:20,425 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:20,426 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:46:20,503 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:46:20,596 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:46:20,602 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:46:20,648 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:46:20,811 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:46:20,816 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:46:20,816 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:46:20,817 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:46:20,817 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:46:20,844 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:46:20,846 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:46:31,282 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-75 because there are too many reads. \n",
      "INFO  15:46:35,507 ProgressMeter -            done    387460.0    14.0 s      37.0 s      100.0%    14.0 s       0.0 s \n",
      "INFO  15:46:35,507 ProgressMeter - Total runtime 14.69 secs, 0.24 min, 0.00 hours \n",
      "INFO  15:46:35,510 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 387460 total reads (0.00%) \n",
      "INFO  15:46:35,510 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:46:35,511 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19049_S14_L001_R2_001 --rg SM:2010AC19049_S14_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19049_S14_L001_R2_001.fastq -S 2010AC19049_S14_L001_R2_001.sam 2> 2010AC19049_S14_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19049_S14_L001_R2_001.sam OUTPUT=2010AC19049_S14_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:47:06 PDT 2017] picard.sam.SortSam INPUT=2010AC19049_S14_L001_R2_001.sam OUTPUT=2010AC19049_S14_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:47:06 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:47:10\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:47:14 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.13 minutes.\n",
      "Runtime.totalMemory()=1191706624\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19049_S14_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19049_S14_L001_R2_001.bam.list\n",
      "INFO  15:47:16,380 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:16,383 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:47:16,383 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:47:16,383 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:47:16,383 HelpFormatter - [Thu Oct 19 15:47:16 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:47:16,384 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:47:16,387 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19049_S14_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19049_S14_L001_R2_001.bam.list \n",
      "INFO  15:47:16,390 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:47:16,391 HelpFormatter - Date/Time: 2017/10/19 15:47:16 \n",
      "INFO  15:47:16,391 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:16,391 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:16,396 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:47:16,528 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:47:16,535 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:47:16,574 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:47:16,751 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:47:16,780 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:47:16,781 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:47:16,781 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:47:16,781 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:47:19,541 ProgressMeter -            done     16100.0     2.0 s       2.9 m       99.5%     2.0 s       0.0 s \n",
      "INFO  15:47:19,541 ProgressMeter - Total runtime 2.76 secs, 0.05 min, 0.00 hours \n",
      "INFO  15:47:19,542 MicroScheduler - 163 reads were filtered out during the traversal out of approximately 289601 total reads (0.06%) \n",
      "INFO  15:47:19,542 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:47:19,543 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:47:19,543 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:47:19,543 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:47:19,543 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:47:19,544 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:47:19,544 MicroScheduler -   -> 163 reads (0.06% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:47:19,545 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:47:19,545 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:47:19,545 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R2_001.bam -targetIntervals 2010AC19049_S14_L001_R2_001.bam.list -o 2010AC19049_S14_L001_R2_001.realigned.bam\n",
      "INFO  15:47:21,392 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:21,395 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:47:21,395 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:47:21,395 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:47:21,395 HelpFormatter - [Thu Oct 19 15:47:21 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:47:21,395 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:47:21,399 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R2_001.bam -targetIntervals 2010AC19049_S14_L001_R2_001.bam.list -o 2010AC19049_S14_L001_R2_001.realigned.bam \n",
      "INFO  15:47:21,402 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:47:21,402 HelpFormatter - Date/Time: 2017/10/19 15:47:21 \n",
      "INFO  15:47:21,402 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:21,403 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:47:21,492 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:47:21,601 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:47:21,607 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:47:21,680 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.07 \n",
      "INFO  15:47:21,900 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:47:21,904 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:47:21,905 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:47:21,905 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:47:21,905 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:47:21,939 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:47:21,941 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:47:22,572 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-68 because there are too many reads. \n",
      "INFO  15:47:25,254 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-73 because there are too many reads. \n",
      "INFO  15:47:30,235 ProgressMeter -            done    387460.0     8.0 s      21.0 s      100.0%     8.0 s       0.0 s \n",
      "INFO  15:47:30,235 ProgressMeter - Total runtime 8.33 secs, 0.14 min, 0.00 hours \n",
      "INFO  15:47:30,238 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 387460 total reads (0.00%) \n",
      "INFO  15:47:30,238 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:47:30,239 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19050_S15_L001_R1_001 --rg SM:2010AC19050_S15_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19050_S15_L001_R1_001.fastq -S 2010AC19050_S15_L001_R1_001.sam 2> 2010AC19050_S15_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19050_S15_L001_R1_001.sam OUTPUT=2010AC19050_S15_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:48:32 PDT 2017] picard.sam.SortSam INPUT=2010AC19050_S15_L001_R1_001.sam OUTPUT=2010AC19050_S15_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:48:32 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:48:40\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:48:46 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.23 minutes.\n",
      "Runtime.totalMemory()=1579155456\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19050_S15_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19050_S15_L001_R1_001.bam.list\n",
      "INFO  15:48:48,388 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:48,390 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:48:48,390 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:48:48,390 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:48:48,390 HelpFormatter - [Thu Oct 19 15:48:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:48:48,391 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:48:48,394 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19050_S15_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19050_S15_L001_R1_001.bam.list \n",
      "INFO  15:48:48,397 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:48:48,397 HelpFormatter - Date/Time: 2017/10/19 15:48:48 \n",
      "INFO  15:48:48,397 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:48,397 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:48,402 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:48:48,554 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:48:48,560 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:48:48,600 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:48:48,755 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:48:48,783 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:48:48,784 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:48:48,784 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:48:48,785 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:48:52,749 ProgressMeter -            done     16100.0     3.0 s       4.1 m       99.5%     3.0 s       0.0 s \n",
      "INFO  15:48:52,750 ProgressMeter - Total runtime 3.97 secs, 0.07 min, 0.00 hours \n",
      "INFO  15:48:52,750 MicroScheduler - 104 reads were filtered out during the traversal out of approximately 516024 total reads (0.02%) \n",
      "INFO  15:48:52,750 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:48:52,751 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:48:52,751 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:48:52,751 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:48:52,751 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:48:52,752 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:48:52,752 MicroScheduler -   -> 104 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:48:52,752 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:48:52,752 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:48:52,753 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R1_001.bam -targetIntervals 2010AC19050_S15_L001_R1_001.bam.list -o 2010AC19050_S15_L001_R1_001.realigned.bam\n",
      "INFO  15:48:54,490 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:54,492 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:48:54,493 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:48:54,493 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:48:54,493 HelpFormatter - [Thu Oct 19 15:48:54 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:48:54,493 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:48:54,496 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R1_001.bam -targetIntervals 2010AC19050_S15_L001_R1_001.bam.list -o 2010AC19050_S15_L001_R1_001.realigned.bam \n",
      "INFO  15:48:54,500 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:48:54,500 HelpFormatter - Date/Time: 2017/10/19 15:48:54 \n",
      "INFO  15:48:54,500 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:54,501 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:48:54,577 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:48:54,662 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:48:54,669 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:48:54,715 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:48:54,871 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:48:54,875 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:48:54,875 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:48:54,876 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:48:54,876 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:48:54,906 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:48:54,908 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:48:55,769 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-57 because there are too many reads. \n",
      "INFO  15:49:00,747 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  15:49:08,293 ProgressMeter -            done    689484.0    13.0 s      19.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  15:49:08,293 ProgressMeter - Total runtime 13.42 secs, 0.22 min, 0.00 hours \n",
      "INFO  15:49:08,296 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 689484 total reads (0.00%) \n",
      "INFO  15:49:08,296 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:49:08,297 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19050_S15_L001_R2_001 --rg SM:2010AC19050_S15_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19050_S15_L001_R2_001.fastq -S 2010AC19050_S15_L001_R2_001.sam 2> 2010AC19050_S15_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19050_S15_L001_R2_001.sam OUTPUT=2010AC19050_S15_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:50:03 PDT 2017] picard.sam.SortSam INPUT=2010AC19050_S15_L001_R2_001.sam OUTPUT=2010AC19050_S15_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:50:03 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:50:13\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:50:19 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.28 minutes.\n",
      "Runtime.totalMemory()=1630535680\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19050_S15_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19050_S15_L001_R2_001.bam.list\n",
      "INFO  15:50:21,632 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:21,635 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:50:21,635 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:50:21,635 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:50:21,636 HelpFormatter - [Thu Oct 19 15:50:21 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:50:21,636 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:50:21,639 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19050_S15_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19050_S15_L001_R2_001.bam.list \n",
      "INFO  15:50:21,642 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:50:21,642 HelpFormatter - Date/Time: 2017/10/19 15:50:21 \n",
      "INFO  15:50:21,643 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:21,643 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:21,648 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:50:21,784 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:50:21,790 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:50:21,829 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:50:22,006 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:50:22,037 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:50:22,037 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:50:22,038 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:50:22,038 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:50:25,855 ProgressMeter -            done     16100.0     3.0 s       4.0 m       99.5%     3.0 s       0.0 s \n",
      "INFO  15:50:25,856 ProgressMeter - Total runtime 3.82 secs, 0.06 min, 0.00 hours \n",
      "INFO  15:50:25,856 MicroScheduler - 234 reads were filtered out during the traversal out of approximately 516701 total reads (0.05%) \n",
      "INFO  15:50:25,857 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:50:25,857 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:50:25,857 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:50:25,857 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:50:25,857 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:50:25,858 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:50:25,858 MicroScheduler -   -> 234 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:50:25,858 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:50:25,858 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:50:25,859 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R2_001.bam -targetIntervals 2010AC19050_S15_L001_R2_001.bam.list -o 2010AC19050_S15_L001_R2_001.realigned.bam\n",
      "INFO  15:50:27,632 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:27,634 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:50:27,635 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:50:27,635 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:50:27,635 HelpFormatter - [Thu Oct 19 15:50:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:50:27,635 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:50:27,638 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R2_001.bam -targetIntervals 2010AC19050_S15_L001_R2_001.bam.list -o 2010AC19050_S15_L001_R2_001.realigned.bam \n",
      "INFO  15:50:27,641 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:50:27,642 HelpFormatter - Date/Time: 2017/10/19 15:50:27 \n",
      "INFO  15:50:27,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:27,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:50:27,722 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:50:27,807 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:50:27,813 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:50:27,857 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:50:28,026 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:50:28,030 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:50:28,030 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:50:28,031 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:50:28,031 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:50:28,064 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:50:28,065 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:50:28,906 IndelRealigner - Not attempting realignment in interval One_RAD56448:27-61 because there are too many reads. \n",
      "INFO  15:50:33,602 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-63 because there are too many reads. \n",
      "INFO  15:50:43,272 ProgressMeter -            done    689484.0    15.0 s      22.0 s      100.0%    15.0 s       0.0 s \n",
      "INFO  15:50:43,272 ProgressMeter - Total runtime 15.24 secs, 0.25 min, 0.00 hours \n",
      "INFO  15:50:43,275 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 689484 total reads (0.00%) \n",
      "INFO  15:50:43,276 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:50:43,276 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19057_S16_L001_R1_001 --rg SM:2010AC19057_S16_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19057_S16_L001_R1_001.fastq -S 2010AC19057_S16_L001_R1_001.sam 2> 2010AC19057_S16_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19057_S16_L001_R1_001.sam OUTPUT=2010AC19057_S16_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:51:56 PDT 2017] picard.sam.SortSam INPUT=2010AC19057_S16_L001_R1_001.sam OUTPUT=2010AC19057_S16_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:51:56 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:52:11\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:52:17 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=1848115200\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19057_S16_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19057_S16_L001_R1_001.bam.list\n",
      "INFO  15:52:19,273 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:19,275 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:52:19,275 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:52:19,275 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:52:19,276 HelpFormatter - [Thu Oct 19 15:52:19 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:52:19,276 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:52:19,279 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19057_S16_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19057_S16_L001_R1_001.bam.list \n",
      "INFO  15:52:19,282 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:52:19,282 HelpFormatter - Date/Time: 2017/10/19 15:52:19 \n",
      "INFO  15:52:19,282 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:19,283 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:19,288 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:52:19,446 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:52:19,452 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:52:19,498 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:52:19,663 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:52:19,691 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:52:19,692 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:52:19,692 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:52:19,692 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:52:23,477 ProgressMeter -            done     16100.0     3.0 s       3.9 m       99.5%     3.0 s       0.0 s \n",
      "INFO  15:52:23,478 ProgressMeter - Total runtime 3.79 secs, 0.06 min, 0.00 hours \n",
      "INFO  15:52:23,478 MicroScheduler - 70 reads were filtered out during the traversal out of approximately 579035 total reads (0.01%) \n",
      "INFO  15:52:23,478 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:52:23,479 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:52:23,479 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:52:23,479 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:52:23,479 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:52:23,479 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:52:23,480 MicroScheduler -   -> 70 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:52:23,480 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:52:23,480 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:52:23,480 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R1_001.bam -targetIntervals 2010AC19057_S16_L001_R1_001.bam.list -o 2010AC19057_S16_L001_R1_001.realigned.bam\n",
      "INFO  15:52:25,227 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:25,229 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:52:25,229 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:52:25,229 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:52:25,230 HelpFormatter - [Thu Oct 19 15:52:25 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:52:25,230 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:52:25,233 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R1_001.bam -targetIntervals 2010AC19057_S16_L001_R1_001.bam.list -o 2010AC19057_S16_L001_R1_001.realigned.bam \n",
      "INFO  15:52:25,236 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:52:25,236 HelpFormatter - Date/Time: 2017/10/19 15:52:25 \n",
      "INFO  15:52:25,237 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:25,237 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:52:25,327 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:52:25,412 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:52:25,418 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:52:25,488 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.07 \n",
      "INFO  15:52:25,661 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:52:25,665 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:52:25,666 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:52:25,666 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:52:25,667 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:52:25,700 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:52:25,701 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:52:26,327 IndelRealigner - Not attempting realignment in interval One_RAD56448:20-57 because there are too many reads. \n",
      "INFO  15:52:31,711 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  15:52:40,694 ProgressMeter -            done    712444.0    15.0 s      21.0 s      100.0%    15.0 s       0.0 s \n",
      "INFO  15:52:40,694 ProgressMeter - Total runtime 15.03 secs, 0.25 min, 0.00 hours \n",
      "INFO  15:52:40,697 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 712444 total reads (0.00%) \n",
      "INFO  15:52:40,697 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:52:40,697 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19057_S16_L001_R2_001 --rg SM:2010AC19057_S16_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19057_S16_L001_R2_001.fastq -S 2010AC19057_S16_L001_R2_001.sam 2> 2010AC19057_S16_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19057_S16_L001_R2_001.sam OUTPUT=2010AC19057_S16_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:53:41 PDT 2017] picard.sam.SortSam INPUT=2010AC19057_S16_L001_R2_001.sam OUTPUT=2010AC19057_S16_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:53:41 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:53:54\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:54:02 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=1913126912\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19057_S16_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19057_S16_L001_R2_001.bam.list\n",
      "INFO  15:54:04,200 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:04,203 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:54:04,203 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:54:04,203 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:54:04,203 HelpFormatter - [Thu Oct 19 15:54:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:54:04,203 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:54:04,207 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19057_S16_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19057_S16_L001_R2_001.bam.list \n",
      "INFO  15:54:04,210 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:54:04,210 HelpFormatter - Date/Time: 2017/10/19 15:54:04 \n",
      "INFO  15:54:04,211 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:04,211 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:04,216 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:54:04,370 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:54:04,376 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:54:04,416 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:54:04,588 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:54:04,620 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:54:04,621 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:54:04,621 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:54:04,621 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:54:08,421 ProgressMeter -            done     16100.0     3.0 s       3.9 m       99.5%     3.0 s       0.0 s \n",
      "INFO  15:54:08,422 ProgressMeter - Total runtime 3.80 secs, 0.06 min, 0.00 hours \n",
      "INFO  15:54:08,422 MicroScheduler - 341 reads were filtered out during the traversal out of approximately 573545 total reads (0.06%) \n",
      "INFO  15:54:08,423 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:54:08,423 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:54:08,423 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:54:08,424 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:54:08,424 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:54:08,424 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:54:08,425 MicroScheduler -   -> 341 reads (0.06% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:54:08,425 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:54:08,425 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:54:08,425 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R2_001.bam -targetIntervals 2010AC19057_S16_L001_R2_001.bam.list -o 2010AC19057_S16_L001_R2_001.realigned.bam\n",
      "INFO  15:54:10,189 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:10,192 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:54:10,192 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:54:10,192 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:54:10,192 HelpFormatter - [Thu Oct 19 15:54:10 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:54:10,192 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:54:10,196 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R2_001.bam -targetIntervals 2010AC19057_S16_L001_R2_001.bam.list -o 2010AC19057_S16_L001_R2_001.realigned.bam \n",
      "INFO  15:54:10,199 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:54:10,199 HelpFormatter - Date/Time: 2017/10/19 15:54:10 \n",
      "INFO  15:54:10,199 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:10,199 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:54:10,276 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:54:10,362 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:54:10,368 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:54:10,411 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:54:10,571 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:54:10,575 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:54:10,576 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:54:10,576 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:54:10,576 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:54:10,608 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:54:10,609 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:54:11,244 IndelRealigner - Not attempting realignment in interval One_RAD56448:26-65 because there are too many reads. \n",
      "INFO  15:54:15,923 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-78 because there are too many reads. \n",
      "INFO  15:54:25,261 ProgressMeter -            done    712444.0    14.0 s      20.0 s      100.0%    14.0 s       0.0 s \n",
      "INFO  15:54:25,261 ProgressMeter - Total runtime 14.69 secs, 0.24 min, 0.00 hours \n",
      "INFO  15:54:25,264 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 712444 total reads (0.00%) \n",
      "INFO  15:54:25,264 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:54:25,264 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19091_S18_L001_R1_001 --rg SM:2010AC19091_S18_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19091_S18_L001_R1_001.fastq -S 2010AC19091_S18_L001_R1_001.sam 2> 2010AC19091_S18_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19091_S18_L001_R1_001.sam OUTPUT=2010AC19091_S18_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:56:03 PDT 2017] picard.sam.SortSam INPUT=2010AC19091_S18_L001_R1_001.sam OUTPUT=2010AC19091_S18_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:56:03 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:56:26\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:56:35 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.54 minutes.\n",
      "Runtime.totalMemory()=2305818624\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19091_S18_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19091_S18_L001_R1_001.bam.list\n",
      "INFO  15:56:37,664 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:37,666 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:56:37,666 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:56:37,667 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:56:37,667 HelpFormatter - [Thu Oct 19 15:56:37 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:56:37,667 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:56:37,670 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19091_S18_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19091_S18_L001_R1_001.bam.list \n",
      "INFO  15:56:37,673 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:56:37,673 HelpFormatter - Date/Time: 2017/10/19 15:56:37 \n",
      "INFO  15:56:37,674 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:37,674 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:37,679 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:56:37,815 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:56:37,821 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:56:37,860 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:56:38,026 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:56:38,051 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:56:38,052 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:56:38,052 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:56:38,052 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:56:42,842 ProgressMeter -            done     16100.0     4.0 s       5.0 m       99.5%     4.0 s       0.0 s \n",
      "INFO  15:56:42,842 ProgressMeter - Total runtime 4.79 secs, 0.08 min, 0.00 hours \n",
      "INFO  15:56:42,843 MicroScheduler - 215 reads were filtered out during the traversal out of approximately 803745 total reads (0.03%) \n",
      "INFO  15:56:42,843 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:56:42,843 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:56:42,843 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:56:42,844 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:56:42,844 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:56:42,844 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:56:42,844 MicroScheduler -   -> 215 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:56:42,844 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:56:42,845 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:56:42,845 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R1_001.bam -targetIntervals 2010AC19091_S18_L001_R1_001.bam.list -o 2010AC19091_S18_L001_R1_001.realigned.bam\n",
      "INFO  15:56:44,583 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:44,586 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:56:44,586 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:56:44,586 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:56:44,586 HelpFormatter - [Thu Oct 19 15:56:44 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:56:44,587 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:56:44,590 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R1_001.bam -targetIntervals 2010AC19091_S18_L001_R1_001.bam.list -o 2010AC19091_S18_L001_R1_001.realigned.bam \n",
      "INFO  15:56:44,593 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:56:44,593 HelpFormatter - Date/Time: 2017/10/19 15:56:44 \n",
      "INFO  15:56:44,593 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:44,594 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:56:44,682 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:56:44,765 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:56:44,771 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:56:44,823 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:56:44,976 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:56:44,980 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:56:44,981 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:56:44,981 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:56:44,981 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:56:45,007 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:56:45,008 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:56:45,960 IndelRealigner - Not attempting realignment in interval One_RAD56448:4-81 because there are too many reads. \n",
      "INFO  15:56:54,694 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  15:57:06,569 ProgressMeter -            done    989427.0    21.0 s      21.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  15:57:06,569 ProgressMeter - Total runtime 21.59 secs, 0.36 min, 0.01 hours \n",
      "INFO  15:57:06,572 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 989427 total reads (0.00%) \n",
      "INFO  15:57:06,572 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:57:06,573 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19091_S18_L001_R2_001 --rg SM:2010AC19091_S18_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19091_S18_L001_R2_001.fastq -S 2010AC19091_S18_L001_R2_001.sam 2> 2010AC19091_S18_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19091_S18_L001_R2_001.sam OUTPUT=2010AC19091_S18_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 15:58:28 PDT 2017] picard.sam.SortSam INPUT=2010AC19091_S18_L001_R2_001.sam OUTPUT=2010AC19091_S18_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 15:58:28 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 15:58:44\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 15:58:55 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.44 minutes.\n",
      "Runtime.totalMemory()=2276982784\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19091_S18_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19091_S18_L001_R2_001.bam.list\n",
      "INFO  15:58:57,255 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:58:57,257 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:58:57,257 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:58:57,257 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:58:57,258 HelpFormatter - [Thu Oct 19 15:58:57 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:58:57,258 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:58:57,261 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19091_S18_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19091_S18_L001_R2_001.bam.list \n",
      "INFO  15:58:57,264 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:58:57,264 HelpFormatter - Date/Time: 2017/10/19 15:58:57 \n",
      "INFO  15:58:57,264 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:58:57,265 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:58:57,270 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:58:57,403 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  15:58:57,409 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:58:57,448 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  15:58:57,602 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:58:57,627 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:58:57,627 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:58:57,628 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:58:57,628 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  15:59:02,709 ProgressMeter -            done     16100.0     5.0 s       5.3 m       99.5%     5.0 s       0.0 s \n",
      "INFO  15:59:02,710 ProgressMeter - Total runtime 5.08 secs, 0.08 min, 0.00 hours \n",
      "INFO  15:59:02,710 MicroScheduler - 349 reads were filtered out during the traversal out of approximately 798995 total reads (0.04%) \n",
      "INFO  15:59:02,711 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:59:02,711 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  15:59:02,712 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  15:59:02,712 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  15:59:02,712 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  15:59:02,712 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  15:59:02,713 MicroScheduler -   -> 349 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  15:59:02,713 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  15:59:02,713 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  15:59:02,713 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R2_001.bam -targetIntervals 2010AC19091_S18_L001_R2_001.bam.list -o 2010AC19091_S18_L001_R2_001.realigned.bam\n",
      "INFO  15:59:04,478 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:59:04,480 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  15:59:04,480 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  15:59:04,480 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  15:59:04,480 HelpFormatter - [Thu Oct 19 15:59:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  15:59:04,481 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  15:59:04,484 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R2_001.bam -targetIntervals 2010AC19091_S18_L001_R2_001.bam.list -o 2010AC19091_S18_L001_R2_001.realigned.bam \n",
      "INFO  15:59:04,487 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  15:59:04,488 HelpFormatter - Date/Time: 2017/10/19 15:59:04 \n",
      "INFO  15:59:04,488 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:59:04,488 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  15:59:04,567 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  15:59:04,652 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  15:59:04,658 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  15:59:04,704 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  15:59:04,863 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  15:59:04,867 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  15:59:04,867 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  15:59:04,868 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  15:59:04,868 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  15:59:04,892 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  15:59:04,893 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  15:59:05,861 IndelRealigner - Not attempting realignment in interval One_RAD56448:16-68 because there are too many reads. \n",
      "INFO  15:59:12,161 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  15:59:27,871 ProgressMeter -            done    989427.0    23.0 s      23.0 s      100.0%    23.0 s       0.0 s \n",
      "INFO  15:59:27,871 ProgressMeter - Total runtime 23.00 secs, 0.38 min, 0.01 hours \n",
      "INFO  15:59:27,874 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 989427 total reads (0.00%) \n",
      "INFO  15:59:27,874 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  15:59:27,875 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19096_S21_L001_R1_001 --rg SM:2010AC19096_S21_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19096_S21_L001_R1_001.fastq -S 2010AC19096_S21_L001_R1_001.sam 2> 2010AC19096_S21_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19096_S21_L001_R1_001.sam OUTPUT=2010AC19096_S21_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:00:37 PDT 2017] picard.sam.SortSam INPUT=2010AC19096_S21_L001_R1_001.sam OUTPUT=2010AC19096_S21_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:00:37 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:00:46\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:00:52 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.25 minutes.\n",
      "Runtime.totalMemory()=1798832128\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19096_S21_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19096_S21_L001_R1_001.bam.list\n",
      "INFO  16:00:53,807 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:00:53,810 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:00:53,810 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:00:53,810 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:00:53,810 HelpFormatter - [Thu Oct 19 16:00:53 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:00:53,810 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:00:53,814 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19096_S21_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19096_S21_L001_R1_001.bam.list \n",
      "INFO  16:00:53,817 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:00:53,817 HelpFormatter - Date/Time: 2017/10/19 16:00:53 \n",
      "INFO  16:00:53,817 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:00:53,818 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:00:53,823 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:00:53,966 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:00:53,972 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:00:54,011 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:00:54,197 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:00:54,222 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:00:54,223 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:00:54,223 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:00:54,224 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:00:58,241 ProgressMeter -            done     16100.0     4.0 s       4.2 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:00:58,242 ProgressMeter - Total runtime 4.02 secs, 0.07 min, 0.00 hours \n",
      "INFO  16:00:58,242 MicroScheduler - 105 reads were filtered out during the traversal out of approximately 575798 total reads (0.02%) \n",
      "INFO  16:00:58,243 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:00:58,243 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:00:58,243 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:00:58,243 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:00:58,244 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:00:58,244 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:00:58,244 MicroScheduler -   -> 105 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:00:58,244 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:00:58,245 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:00:58,245 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R1_001.bam -targetIntervals 2010AC19096_S21_L001_R1_001.bam.list -o 2010AC19096_S21_L001_R1_001.realigned.bam\n",
      "INFO  16:01:00,008 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:01:00,010 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:01:00,010 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:01:00,011 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:01:00,011 HelpFormatter - [Thu Oct 19 16:00:59 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:01:00,011 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:01:00,014 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R1_001.bam -targetIntervals 2010AC19096_S21_L001_R1_001.bam.list -o 2010AC19096_S21_L001_R1_001.realigned.bam \n",
      "INFO  16:01:00,017 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:01:00,017 HelpFormatter - Date/Time: 2017/10/19 16:01:00 \n",
      "INFO  16:01:00,017 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:01:00,018 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:01:00,099 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:01:00,186 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:01:00,192 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:01:00,235 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:01:00,397 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:01:00,402 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:01:00,402 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:01:00,403 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:01:00,403 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:01:00,434 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:01:00,435 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:01:01,193 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-57 because there are too many reads. \n",
      "INFO  16:01:06,131 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-75 because there are too many reads. \n",
      "INFO  16:01:14,244 ProgressMeter -            done    735422.0    13.0 s      18.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  16:01:14,244 ProgressMeter - Total runtime 13.84 secs, 0.23 min, 0.00 hours \n",
      "INFO  16:01:14,247 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 735422 total reads (0.00%) \n",
      "INFO  16:01:14,247 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:01:14,247 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19096_S21_L001_R2_001 --rg SM:2010AC19096_S21_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19096_S21_L001_R2_001.fastq -S 2010AC19096_S21_L001_R2_001.sam 2> 2010AC19096_S21_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19096_S21_L001_R2_001.sam OUTPUT=2010AC19096_S21_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:02:13 PDT 2017] picard.sam.SortSam INPUT=2010AC19096_S21_L001_R2_001.sam OUTPUT=2010AC19096_S21_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:02:13 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:02:24\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:02:31 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.29 minutes.\n",
      "Runtime.totalMemory()=1786773504\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19096_S21_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19096_S21_L001_R2_001.bam.list\n",
      "INFO  16:02:32,869 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:32,872 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:02:32,872 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:02:32,872 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:02:32,872 HelpFormatter - [Thu Oct 19 16:02:32 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:02:32,873 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:02:32,876 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19096_S21_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19096_S21_L001_R2_001.bam.list \n",
      "INFO  16:02:32,879 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:02:32,880 HelpFormatter - Date/Time: 2017/10/19 16:02:32 \n",
      "INFO  16:02:32,880 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:32,880 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:32,885 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:02:33,009 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:02:33,015 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:02:33,057 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:02:33,228 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:02:33,254 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:02:33,254 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:02:33,255 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:02:33,255 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:02:37,478 ProgressMeter -            done     16100.0     4.0 s       4.4 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:02:37,478 ProgressMeter - Total runtime 4.22 secs, 0.07 min, 0.00 hours \n",
      "INFO  16:02:37,478 MicroScheduler - 270 reads were filtered out during the traversal out of approximately 577425 total reads (0.05%) \n",
      "INFO  16:02:37,479 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:02:37,479 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:02:37,479 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:02:37,479 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:02:37,480 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:02:37,480 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:02:37,480 MicroScheduler -   -> 270 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:02:37,480 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:02:37,481 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:02:37,481 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R2_001.bam -targetIntervals 2010AC19096_S21_L001_R2_001.bam.list -o 2010AC19096_S21_L001_R2_001.realigned.bam\n",
      "INFO  16:02:39,200 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:39,202 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:02:39,203 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:02:39,203 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:02:39,203 HelpFormatter - [Thu Oct 19 16:02:39 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:02:39,203 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:02:39,206 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R2_001.bam -targetIntervals 2010AC19096_S21_L001_R2_001.bam.list -o 2010AC19096_S21_L001_R2_001.realigned.bam \n",
      "INFO  16:02:39,209 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:02:39,210 HelpFormatter - Date/Time: 2017/10/19 16:02:39 \n",
      "INFO  16:02:39,210 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:39,210 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:02:39,290 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:02:39,377 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:02:39,384 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:02:39,426 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:02:39,587 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:02:39,591 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:02:39,591 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:02:39,592 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:02:39,592 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:02:39,618 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:02:39,620 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:02:40,430 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-56 because there are too many reads. \n",
      "INFO  16:02:45,250 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:25-78 because there are too many reads. \n",
      "INFO  16:02:56,378 ProgressMeter -            done    735422.0    16.0 s      22.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  16:02:56,378 ProgressMeter - Total runtime 16.79 secs, 0.28 min, 0.00 hours \n",
      "INFO  16:02:56,381 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 735422 total reads (0.00%) \n",
      "INFO  16:02:56,381 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:02:56,382 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19098_S22_L001_R1_001 --rg SM:2010AC19098_S22_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19098_S22_L001_R1_001.fastq -S 2010AC19098_S22_L001_R1_001.sam 2> 2010AC19098_S22_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19098_S22_L001_R1_001.sam OUTPUT=2010AC19098_S22_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:04:05 PDT 2017] picard.sam.SortSam INPUT=2010AC19098_S22_L001_R1_001.sam OUTPUT=2010AC19098_S22_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:04:05 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:04:20\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:04:28 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.38 minutes.\n",
      "Runtime.totalMemory()=1816657920\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19098_S22_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19098_S22_L001_R1_001.bam.list\n",
      "INFO  16:04:29,898 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:29,900 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:04:29,900 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:04:29,900 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:04:29,900 HelpFormatter - [Thu Oct 19 16:04:29 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:04:29,900 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:04:29,904 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19098_S22_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19098_S22_L001_R1_001.bam.list \n",
      "INFO  16:04:29,906 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:04:29,907 HelpFormatter - Date/Time: 2017/10/19 16:04:29 \n",
      "INFO  16:04:29,907 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:29,907 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:29,912 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:04:30,052 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:04:30,058 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:04:30,095 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:04:30,264 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:04:30,290 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:04:30,290 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:04:30,291 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:04:30,291 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:04:34,939 ProgressMeter -            done     16100.0     4.0 s       4.8 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:04:34,940 ProgressMeter - Total runtime 4.65 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:04:34,940 MicroScheduler - 93 reads were filtered out during the traversal out of approximately 643150 total reads (0.01%) \n",
      "INFO  16:04:34,941 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:04:34,941 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:04:34,941 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:04:34,941 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:04:34,942 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:04:34,942 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:04:34,942 MicroScheduler -   -> 93 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:04:34,942 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:04:34,943 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:04:34,943 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R1_001.bam -targetIntervals 2010AC19098_S22_L001_R1_001.bam.list -o 2010AC19098_S22_L001_R1_001.realigned.bam\n",
      "INFO  16:04:36,696 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:36,698 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:04:36,699 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:04:36,699 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:04:36,699 HelpFormatter - [Thu Oct 19 16:04:36 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:04:36,699 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:04:36,702 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R1_001.bam -targetIntervals 2010AC19098_S22_L001_R1_001.bam.list -o 2010AC19098_S22_L001_R1_001.realigned.bam \n",
      "INFO  16:04:36,705 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:04:36,706 HelpFormatter - Date/Time: 2017/10/19 16:04:36 \n",
      "INFO  16:04:36,706 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:36,706 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:04:36,783 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:04:36,868 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:04:36,874 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:04:36,917 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:04:37,069 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:04:37,073 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:04:37,074 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:04:37,074 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:04:37,074 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:04:37,098 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:04:37,100 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:04:38,013 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-57 because there are too many reads. \n",
      "INFO  16:04:44,813 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  16:04:53,993 ProgressMeter -            done    821460.0    16.0 s      20.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  16:04:53,993 ProgressMeter - Total runtime 16.92 secs, 0.28 min, 0.00 hours \n",
      "INFO  16:04:53,996 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 821460 total reads (0.00%) \n",
      "INFO  16:04:53,996 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:04:53,996 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19098_S22_L001_R2_001 --rg SM:2010AC19098_S22_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19098_S22_L001_R2_001.fastq -S 2010AC19098_S22_L001_R2_001.sam 2> 2010AC19098_S22_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19098_S22_L001_R2_001.sam OUTPUT=2010AC19098_S22_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:05:57 PDT 2017] picard.sam.SortSam INPUT=2010AC19098_S22_L001_R2_001.sam OUTPUT=2010AC19098_S22_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:05:57 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:06:11\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:06:19 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.38 minutes.\n",
      "Runtime.totalMemory()=1875378176\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19098_S22_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19098_S22_L001_R2_001.bam.list\n",
      "INFO  16:06:21,235 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:21,237 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:06:21,238 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:06:21,238 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:06:21,238 HelpFormatter - [Thu Oct 19 16:06:21 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:06:21,238 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:06:21,242 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19098_S22_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19098_S22_L001_R2_001.bam.list \n",
      "INFO  16:06:21,245 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:06:21,245 HelpFormatter - Date/Time: 2017/10/19 16:06:21 \n",
      "INFO  16:06:21,245 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:21,245 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:21,251 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:06:21,387 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:06:21,393 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:06:21,432 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:06:21,610 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:06:21,635 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:06:21,636 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:06:21,636 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:06:21,637 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:06:26,340 ProgressMeter -            done     16100.0     4.0 s       4.9 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:06:26,341 ProgressMeter - Total runtime 4.70 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:06:26,341 MicroScheduler - 295 reads were filtered out during the traversal out of approximately 652522 total reads (0.05%) \n",
      "INFO  16:06:26,341 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:06:26,342 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:06:26,342 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:06:26,342 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:06:26,342 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:06:26,343 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:06:26,343 MicroScheduler -   -> 295 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:06:26,343 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:06:26,343 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:06:26,344 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R2_001.bam -targetIntervals 2010AC19098_S22_L001_R2_001.bam.list -o 2010AC19098_S22_L001_R2_001.realigned.bam\n",
      "INFO  16:06:28,116 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:28,118 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:06:28,119 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:06:28,119 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:06:28,119 HelpFormatter - [Thu Oct 19 16:06:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:06:28,119 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:06:28,122 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R2_001.bam -targetIntervals 2010AC19098_S22_L001_R2_001.bam.list -o 2010AC19098_S22_L001_R2_001.realigned.bam \n",
      "INFO  16:06:28,125 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:06:28,126 HelpFormatter - Date/Time: 2017/10/19 16:06:28 \n",
      "INFO  16:06:28,126 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:28,126 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:06:28,219 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:06:28,303 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:06:28,309 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:06:28,352 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:06:28,506 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:06:28,510 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:06:28,510 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:06:28,511 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:06:28,511 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:06:28,536 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:06:28,538 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:06:30,053 IndelRealigner - Not attempting realignment in interval One_RAD56448:14-61 because there are too many reads. \n",
      "INFO  16:06:34,423 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-61 because there are too many reads. \n",
      "INFO  16:06:48,024 ProgressMeter -            done    821460.0    19.0 s      23.0 s      100.0%    19.0 s       0.0 s \n",
      "INFO  16:06:48,024 ProgressMeter - Total runtime 19.51 secs, 0.33 min, 0.01 hours \n",
      "INFO  16:06:48,027 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 821460 total reads (0.00%) \n",
      "INFO  16:06:48,027 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:06:48,028 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19099_S19_L001_R1_001 --rg SM:2010AC19099_S19_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19099_S19_L001_R1_001.fastq -S 2010AC19099_S19_L001_R1_001.sam 2> 2010AC19099_S19_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19099_S19_L001_R1_001.sam OUTPUT=2010AC19099_S19_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:08:35 PDT 2017] picard.sam.SortSam INPUT=2010AC19099_S19_L001_R1_001.sam OUTPUT=2010AC19099_S19_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:08:35 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:08:59\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:09:07 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.53 minutes.\n",
      "Runtime.totalMemory()=2312110080\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19099_S19_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19099_S19_L001_R1_001.bam.list\n",
      "INFO  16:09:09,269 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:09,271 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:09:09,271 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:09:09,272 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:09:09,272 HelpFormatter - [Thu Oct 19 16:09:09 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:09:09,272 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:09:09,275 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19099_S19_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19099_S19_L001_R1_001.bam.list \n",
      "INFO  16:09:09,278 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:09:09,279 HelpFormatter - Date/Time: 2017/10/19 16:09:09 \n",
      "INFO  16:09:09,279 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:09,279 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:09,284 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:09:09,426 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:09:09,432 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:09:09,471 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:09:09,643 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:09:09,672 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:09:09,673 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:09:09,673 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:09:09,673 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:09:14,956 ProgressMeter -            done     16100.0     5.0 s       5.5 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:09:14,957 ProgressMeter - Total runtime 5.28 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:09:14,958 MicroScheduler - 302 reads were filtered out during the traversal out of approximately 885833 total reads (0.03%) \n",
      "INFO  16:09:14,958 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:09:14,958 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:09:14,959 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:09:14,959 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:09:14,959 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:09:14,959 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:09:14,960 MicroScheduler -   -> 302 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:09:14,960 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:09:14,960 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:09:14,960 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R1_001.bam -targetIntervals 2010AC19099_S19_L001_R1_001.bam.list -o 2010AC19099_S19_L001_R1_001.realigned.bam\n",
      "INFO  16:09:16,706 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:16,708 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:09:16,708 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:09:16,709 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:09:16,709 HelpFormatter - [Thu Oct 19 16:09:16 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:09:16,709 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:09:16,712 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R1_001.bam -targetIntervals 2010AC19099_S19_L001_R1_001.bam.list -o 2010AC19099_S19_L001_R1_001.realigned.bam \n",
      "INFO  16:09:16,715 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:09:16,716 HelpFormatter - Date/Time: 2017/10/19 16:09:16 \n",
      "INFO  16:09:16,716 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:16,716 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:09:16,794 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:09:16,884 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:09:16,889 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:09:16,934 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:09:17,091 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:09:17,094 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:09:17,095 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:09:17,095 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:09:17,095 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:09:17,126 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:09:17,127 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:09:18,132 IndelRealigner - Not attempting realignment in interval One_RAD56448:12-56 because there are too many reads. \n",
      "INFO  16:09:26,196 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-75 because there are too many reads. \n",
      "INFO  16:09:39,000 ProgressMeter -            done   1106797.0    21.0 s      19.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  16:09:39,000 ProgressMeter - Total runtime 21.91 secs, 0.37 min, 0.01 hours \n",
      "INFO  16:09:39,003 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1106797 total reads (0.00%) \n",
      "INFO  16:09:39,003 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:09:39,003 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19099_S19_L001_R2_001 --rg SM:2010AC19099_S19_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19099_S19_L001_R2_001.fastq -S 2010AC19099_S19_L001_R2_001.sam 2> 2010AC19099_S19_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19099_S19_L001_R2_001.sam OUTPUT=2010AC19099_S19_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:11:09 PDT 2017] picard.sam.SortSam INPUT=2010AC19099_S19_L001_R2_001.sam OUTPUT=2010AC19099_S19_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:11:09 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:11:30\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:11:39 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.49 minutes.\n",
      "Runtime.totalMemory()=2332557312\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19099_S19_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19099_S19_L001_R2_001.bam.list\n",
      "INFO  16:11:41,115 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:41,117 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:11:41,117 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:11:41,117 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:11:41,117 HelpFormatter - [Thu Oct 19 16:11:41 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:11:41,117 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:11:41,121 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19099_S19_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19099_S19_L001_R2_001.bam.list \n",
      "INFO  16:11:41,124 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:11:41,124 HelpFormatter - Date/Time: 2017/10/19 16:11:41 \n",
      "INFO  16:11:41,124 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:41,125 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:41,129 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:11:41,270 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:11:41,276 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:11:41,315 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:11:41,471 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:11:41,498 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:11:41,499 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:11:41,499 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:11:41,499 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:11:46,566 ProgressMeter -            done     16100.0     5.0 s       5.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:11:46,566 ProgressMeter - Total runtime 5.07 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:11:46,567 MicroScheduler - 456 reads were filtered out during the traversal out of approximately 883139 total reads (0.05%) \n",
      "INFO  16:11:46,567 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:11:46,567 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:11:46,568 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:11:46,568 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:11:46,568 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:11:46,568 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:11:46,569 MicroScheduler -   -> 456 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:11:46,569 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:11:46,569 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:11:46,569 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R2_001.bam -targetIntervals 2010AC19099_S19_L001_R2_001.bam.list -o 2010AC19099_S19_L001_R2_001.realigned.bam\n",
      "INFO  16:11:48,371 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:48,373 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:11:48,373 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:11:48,374 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:11:48,374 HelpFormatter - [Thu Oct 19 16:11:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:11:48,374 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:11:48,377 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R2_001.bam -targetIntervals 2010AC19099_S19_L001_R2_001.bam.list -o 2010AC19099_S19_L001_R2_001.realigned.bam \n",
      "INFO  16:11:48,380 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:11:48,380 HelpFormatter - Date/Time: 2017/10/19 16:11:48 \n",
      "INFO  16:11:48,381 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:48,381 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:11:48,461 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:11:48,556 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:11:48,562 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:11:48,609 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:11:48,771 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:11:48,775 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:11:48,776 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:11:48,776 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:11:48,776 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:11:48,804 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:11:48,805 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:11:50,421 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-61 because there are too many reads. \n",
      "INFO  16:11:55,980 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:18-78 because there are too many reads. \n",
      "INFO  16:12:13,232 ProgressMeter -            done   1106797.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  16:12:13,233 ProgressMeter - Total runtime 24.46 secs, 0.41 min, 0.01 hours \n",
      "INFO  16:12:13,235 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1106797 total reads (0.00%) \n",
      "INFO  16:12:13,235 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:12:13,236 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19101_S23_L001_R1_001 --rg SM:2010AC19101_S23_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19101_S23_L001_R1_001.fastq -S 2010AC19101_S23_L001_R1_001.sam 2> 2010AC19101_S23_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19101_S23_L001_R1_001.sam OUTPUT=2010AC19101_S23_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:14:09 PDT 2017] picard.sam.SortSam INPUT=2010AC19101_S23_L001_R1_001.sam OUTPUT=2010AC19101_S23_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:14:09 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:14:33\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:14:43 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.56 minutes.\n",
      "Runtime.totalMemory()=2313158656\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19101_S23_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19101_S23_L001_R1_001.bam.list\n",
      "INFO  16:14:45,620 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:45,623 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:14:45,623 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:14:45,623 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:14:45,623 HelpFormatter - [Thu Oct 19 16:14:45 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:14:45,623 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:14:45,627 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19101_S23_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19101_S23_L001_R1_001.bam.list \n",
      "INFO  16:14:45,630 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:14:45,630 HelpFormatter - Date/Time: 2017/10/19 16:14:45 \n",
      "INFO  16:14:45,630 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:45,630 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:45,636 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:14:45,785 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:14:45,792 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:14:45,831 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:14:45,992 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:14:46,018 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:14:46,019 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:14:46,019 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:14:46,019 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:14:51,982 ProgressMeter -            done     16100.0     5.0 s       6.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:14:51,983 ProgressMeter - Total runtime 5.96 secs, 0.10 min, 0.00 hours \n",
      "INFO  16:14:51,985 MicroScheduler - 176 reads were filtered out during the traversal out of approximately 1035426 total reads (0.02%) \n",
      "INFO  16:14:51,986 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:14:51,986 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:14:51,986 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:14:51,987 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:14:51,987 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:14:51,987 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:14:51,987 MicroScheduler -   -> 176 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:14:51,987 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:14:51,988 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:14:51,988 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R1_001.bam -targetIntervals 2010AC19101_S23_L001_R1_001.bam.list -o 2010AC19101_S23_L001_R1_001.realigned.bam\n",
      "INFO  16:14:53,750 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:53,753 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:14:53,753 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:14:53,753 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:14:53,753 HelpFormatter - [Thu Oct 19 16:14:53 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:14:53,753 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:14:53,756 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R1_001.bam -targetIntervals 2010AC19101_S23_L001_R1_001.bam.list -o 2010AC19101_S23_L001_R1_001.realigned.bam \n",
      "INFO  16:14:53,759 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:14:53,760 HelpFormatter - Date/Time: 2017/10/19 16:14:53 \n",
      "INFO  16:14:53,760 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:53,760 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:14:53,836 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:14:53,923 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:14:53,929 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:14:53,973 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:14:54,129 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:14:54,133 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:14:54,134 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:14:54,134 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:14:54,135 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:14:54,165 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:14:54,166 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:14:55,813 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-57 because there are too many reads. \n",
      "INFO  16:15:05,826 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  16:15:25,299 ProgressMeter - 9801_x1_FemaleLG_28_FemalePos_166.1_MaleLG_NA_MalePos_NA_duplicated_no:76   1033899.0    31.0 s      30.0 s      100.0%    31.0 s       0.0 s \n",
      "INFO  16:15:26,187 ProgressMeter -            done   1239009.0    32.0 s      25.0 s      100.0%    32.0 s       0.0 s \n",
      "INFO  16:15:26,187 ProgressMeter - Total runtime 32.05 secs, 0.53 min, 0.01 hours \n",
      "INFO  16:15:26,190 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1239009 total reads (0.00%) \n",
      "INFO  16:15:26,191 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:15:26,191 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19101_S23_L001_R2_001 --rg SM:2010AC19101_S23_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19101_S23_L001_R2_001.fastq -S 2010AC19101_S23_L001_R2_001.sam 2> 2010AC19101_S23_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19101_S23_L001_R2_001.sam OUTPUT=2010AC19101_S23_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:17:08 PDT 2017] picard.sam.SortSam INPUT=2010AC19101_S23_L001_R2_001.sam OUTPUT=2010AC19101_S23_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:17:08 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:17:22\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:17:32 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.41 minutes.\n",
      "Runtime.totalMemory()=2193096704\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19101_S23_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19101_S23_L001_R2_001.bam.list\n",
      "INFO  16:17:34,617 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:34,619 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:17:34,619 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:17:34,619 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:17:34,620 HelpFormatter - [Thu Oct 19 16:17:34 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:17:34,620 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:17:34,623 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19101_S23_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19101_S23_L001_R2_001.bam.list \n",
      "INFO  16:17:34,626 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:17:34,626 HelpFormatter - Date/Time: 2017/10/19 16:17:34 \n",
      "INFO  16:17:34,627 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:34,627 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:34,632 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:17:34,772 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:17:34,778 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:17:34,817 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:17:34,969 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:17:34,996 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:17:34,997 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:17:34,997 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:17:34,997 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:17:40,910 ProgressMeter -            done     16100.0     5.0 s       6.1 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:17:40,911 ProgressMeter - Total runtime 5.91 secs, 0.10 min, 0.00 hours \n",
      "INFO  16:17:40,911 MicroScheduler - 597 reads were filtered out during the traversal out of approximately 1044135 total reads (0.06%) \n",
      "INFO  16:17:40,912 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:17:40,912 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:17:40,912 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:17:40,912 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:17:40,912 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:17:40,913 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:17:40,913 MicroScheduler -   -> 597 reads (0.06% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:17:40,913 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:17:40,914 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:17:40,914 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R2_001.bam -targetIntervals 2010AC19101_S23_L001_R2_001.bam.list -o 2010AC19101_S23_L001_R2_001.realigned.bam\n",
      "INFO  16:17:42,674 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:42,676 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:17:42,677 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:17:42,677 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:17:42,677 HelpFormatter - [Thu Oct 19 16:17:42 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:17:42,677 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:17:42,680 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R2_001.bam -targetIntervals 2010AC19101_S23_L001_R2_001.bam.list -o 2010AC19101_S23_L001_R2_001.realigned.bam \n",
      "INFO  16:17:42,683 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:17:42,684 HelpFormatter - Date/Time: 2017/10/19 16:17:42 \n",
      "INFO  16:17:42,684 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:42,684 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:17:42,763 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:17:42,849 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:17:42,855 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:17:42,898 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:17:43,059 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:17:43,062 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:17:43,063 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:17:43,063 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:17:43,063 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:17:43,088 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:17:43,090 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:17:44,735 IndelRealigner - Not attempting realignment in interval One_RAD56448:27-59 because there are too many reads. \n",
      "INFO  16:17:51,642 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:25-78 because there are too many reads. \n",
      "INFO  16:18:00,892 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:6-69 because there are too many reads. \n",
      "INFO  16:18:05,267 ProgressMeter -            done   1239009.0    22.0 s      17.0 s      100.0%    22.0 s       0.0 s \n",
      "INFO  16:18:05,267 ProgressMeter - Total runtime 22.20 secs, 0.37 min, 0.01 hours \n",
      "INFO  16:18:05,270 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1239009 total reads (0.00%) \n",
      "INFO  16:18:05,270 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:18:05,271 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19102_S20_L001_R1_001 --rg SM:2010AC19102_S20_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19102_S20_L001_R1_001.fastq -S 2010AC19102_S20_L001_R1_001.sam 2> 2010AC19102_S20_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19102_S20_L001_R1_001.sam OUTPUT=2010AC19102_S20_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:19:59 PDT 2017] picard.sam.SortSam INPUT=2010AC19102_S20_L001_R1_001.sam OUTPUT=2010AC19102_S20_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:19:59 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:20:15\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:20:24 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.42 minutes.\n",
      "Runtime.totalMemory()=2125987840\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19102_S20_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19102_S20_L001_R1_001.bam.list\n",
      "INFO  16:20:26,018 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:26,020 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:20:26,021 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:20:26,021 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:20:26,021 HelpFormatter - [Thu Oct 19 16:20:25 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:20:26,021 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:20:26,025 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19102_S20_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19102_S20_L001_R1_001.bam.list \n",
      "INFO  16:20:26,027 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:20:26,028 HelpFormatter - Date/Time: 2017/10/19 16:20:26 \n",
      "INFO  16:20:26,028 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:26,028 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:26,033 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:20:26,171 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:20:26,177 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:20:26,221 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:20:26,385 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:20:26,417 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:20:26,418 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:20:26,419 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:20:26,419 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:20:31,986 ProgressMeter -            done     16100.0     5.0 s       5.8 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:20:31,986 ProgressMeter - Total runtime 5.57 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:20:31,987 MicroScheduler - 59 reads were filtered out during the traversal out of approximately 924949 total reads (0.01%) \n",
      "INFO  16:20:31,987 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:20:31,988 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:20:31,988 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:20:31,988 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:20:31,989 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:20:31,989 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:20:31,989 MicroScheduler -   -> 59 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:20:31,989 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:20:31,989 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:20:31,990 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R1_001.bam -targetIntervals 2010AC19102_S20_L001_R1_001.bam.list -o 2010AC19102_S20_L001_R1_001.realigned.bam\n",
      "INFO  16:20:33,771 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:33,773 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:20:33,773 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:20:33,773 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:20:33,774 HelpFormatter - [Thu Oct 19 16:20:33 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:20:33,774 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:20:33,777 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R1_001.bam -targetIntervals 2010AC19102_S20_L001_R1_001.bam.list -o 2010AC19102_S20_L001_R1_001.realigned.bam \n",
      "INFO  16:20:33,780 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:20:33,780 HelpFormatter - Date/Time: 2017/10/19 16:20:33 \n",
      "INFO  16:20:33,780 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:33,781 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:20:33,860 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:20:33,944 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:20:33,950 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:20:33,995 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:20:34,157 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:20:34,161 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:20:34,162 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:20:34,163 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:20:34,163 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:20:34,191 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:20:34,193 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:20:35,186 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-59 because there are too many reads. \n",
      "INFO  16:20:44,346 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-78 because there are too many reads. \n",
      "INFO  16:20:59,198 ProgressMeter -            done   1126561.0    25.0 s      22.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  16:20:59,199 ProgressMeter - Total runtime 25.04 secs, 0.42 min, 0.01 hours \n",
      "INFO  16:20:59,201 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1126561 total reads (0.00%) \n",
      "INFO  16:20:59,202 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:20:59,202 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19102_S20_L001_R2_001 --rg SM:2010AC19102_S20_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19102_S20_L001_R2_001.fastq -S 2010AC19102_S20_L001_R2_001.sam 2> 2010AC19102_S20_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19102_S20_L001_R2_001.sam OUTPUT=2010AC19102_S20_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:22:34 PDT 2017] picard.sam.SortSam INPUT=2010AC19102_S20_L001_R2_001.sam OUTPUT=2010AC19102_S20_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:22:34 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:22:52\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:23:01 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.45 minutes.\n",
      "Runtime.totalMemory()=2386558976\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19102_S20_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19102_S20_L001_R2_001.bam.list\n",
      "INFO  16:23:03,377 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:03,379 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:23:03,380 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:23:03,380 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:23:03,380 HelpFormatter - [Thu Oct 19 16:23:03 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:23:03,380 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:23:03,383 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19102_S20_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19102_S20_L001_R2_001.bam.list \n",
      "INFO  16:23:03,386 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:23:03,387 HelpFormatter - Date/Time: 2017/10/19 16:23:03 \n",
      "INFO  16:23:03,387 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:03,387 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:03,392 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:23:03,547 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:23:03,553 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:23:03,594 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:23:03,750 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:23:03,776 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:23:03,777 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:23:03,777 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:23:03,778 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:23:08,931 ProgressMeter -            done     16100.0     5.0 s       5.3 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:23:08,931 ProgressMeter - Total runtime 5.15 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:23:08,932 MicroScheduler - 471 reads were filtered out during the traversal out of approximately 922146 total reads (0.05%) \n",
      "INFO  16:23:08,932 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:23:08,932 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:23:08,933 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:23:08,933 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:23:08,933 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:23:08,933 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:23:08,934 MicroScheduler -   -> 471 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:23:08,934 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:23:08,934 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:23:08,934 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R2_001.bam -targetIntervals 2010AC19102_S20_L001_R2_001.bam.list -o 2010AC19102_S20_L001_R2_001.realigned.bam\n",
      "INFO  16:23:10,687 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:10,689 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:23:10,689 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:23:10,689 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:23:10,690 HelpFormatter - [Thu Oct 19 16:23:10 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:23:10,690 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:23:10,693 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R2_001.bam -targetIntervals 2010AC19102_S20_L001_R2_001.bam.list -o 2010AC19102_S20_L001_R2_001.realigned.bam \n",
      "INFO  16:23:10,696 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:23:10,696 HelpFormatter - Date/Time: 2017/10/19 16:23:10 \n",
      "INFO  16:23:10,696 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:10,696 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:23:10,773 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:23:10,858 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:23:10,864 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:23:10,908 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:23:11,062 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:23:11,066 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:23:11,066 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:23:11,067 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:23:11,067 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:23:11,099 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:23:11,100 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:23:12,107 IndelRealigner - Not attempting realignment in interval One_RAD56448:20-66 because there are too many reads. \n",
      "INFO  16:23:18,569 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-75 because there are too many reads. \n",
      "INFO  16:23:36,906 ProgressMeter -            done   1126561.0    25.0 s      22.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  16:23:36,906 ProgressMeter - Total runtime 25.84 secs, 0.43 min, 0.01 hours \n",
      "INFO  16:23:36,909 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1126561 total reads (0.00%) \n",
      "INFO  16:23:36,909 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:23:36,909 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19105_S24_L001_R1_001 --rg SM:2010AC19105_S24_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19105_S24_L001_R1_001.fastq -S 2010AC19105_S24_L001_R1_001.sam 2> 2010AC19105_S24_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19105_S24_L001_R1_001.sam OUTPUT=2010AC19105_S24_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:26:24 PDT 2017] picard.sam.SortSam INPUT=2010AC19105_S24_L001_R1_001.sam OUTPUT=2010AC19105_S24_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:26:24 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:26:53\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:27:07 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.73 minutes.\n",
      "Runtime.totalMemory()=2420113408\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19105_S24_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19105_S24_L001_R1_001.bam.list\n",
      "INFO  16:27:09,711 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:09,713 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:27:09,713 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:27:09,713 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:27:09,714 HelpFormatter - [Thu Oct 19 16:27:09 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:27:09,714 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:27:09,717 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19105_S24_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19105_S24_L001_R1_001.bam.list \n",
      "INFO  16:27:09,720 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:27:09,721 HelpFormatter - Date/Time: 2017/10/19 16:27:09 \n",
      "INFO  16:27:09,721 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:09,721 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:09,726 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:27:09,871 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:27:09,877 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:27:09,915 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:27:10,081 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:27:10,109 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:27:10,109 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:27:10,110 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:27:10,110 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:27:18,334 ProgressMeter -            done     16100.0     8.0 s       8.5 m       99.5%     8.0 s       0.0 s \n",
      "INFO  16:27:18,335 ProgressMeter - Total runtime 8.23 secs, 0.14 min, 0.00 hours \n",
      "INFO  16:27:18,335 MicroScheduler - 328 reads were filtered out during the traversal out of approximately 1644950 total reads (0.02%) \n",
      "INFO  16:27:18,335 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:27:18,336 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:27:18,336 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:27:18,336 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:27:18,336 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:27:18,337 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:27:18,337 MicroScheduler -   -> 328 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:27:18,337 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:27:18,337 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:27:18,337 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R1_001.bam -targetIntervals 2010AC19105_S24_L001_R1_001.bam.list -o 2010AC19105_S24_L001_R1_001.realigned.bam\n",
      "INFO  16:27:20,149 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:20,151 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:27:20,152 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:27:20,152 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:27:20,152 HelpFormatter - [Thu Oct 19 16:27:20 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:27:20,152 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:27:20,155 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R1_001.bam -targetIntervals 2010AC19105_S24_L001_R1_001.bam.list -o 2010AC19105_S24_L001_R1_001.realigned.bam \n",
      "INFO  16:27:20,158 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:27:20,159 HelpFormatter - Date/Time: 2017/10/19 16:27:20 \n",
      "INFO  16:27:20,159 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:20,159 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:27:20,236 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:27:20,323 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:27:20,329 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:27:20,379 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:27:20,540 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:27:20,544 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:27:20,544 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:27:20,545 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:27:20,545 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:27:20,578 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:27:20,579 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:27:23,236 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-84 because there are too many reads. \n",
      "INFO  16:27:28,914 IndelRealigner - Not attempting realignment in interval CM007957.1_1049506:6-58 because there are too many reads. \n",
      "INFO  16:27:50,718 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-77 because there are too many reads. \n",
      "INFO  16:27:50,729 ProgressMeter - 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:83    625304.0    30.0 s      48.0 s       45.5%    65.0 s      35.0 s \n",
      "INFO  16:28:01,282 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:10-69 because there are too many reads. \n",
      "INFO  16:28:10,443 ProgressMeter -            done   2115329.0    49.0 s      23.0 s      100.0%    49.0 s       0.0 s \n",
      "INFO  16:28:10,443 ProgressMeter - Total runtime 49.90 secs, 0.83 min, 0.01 hours \n",
      "INFO  16:28:10,446 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 2115329 total reads (0.00%) \n",
      "INFO  16:28:10,446 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:28:10,446 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19105_S24_L001_R2_001 --rg SM:2010AC19105_S24_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19105_S24_L001_R2_001.fastq -S 2010AC19105_S24_L001_R2_001.sam 2> 2010AC19105_S24_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19105_S24_L001_R2_001.sam OUTPUT=2010AC19105_S24_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:30:48 PDT 2017] picard.sam.SortSam INPUT=2010AC19105_S24_L001_R2_001.sam OUTPUT=2010AC19105_S24_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:30:48 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "Ignoring SAM validation error due to lenient parsing:\n",
      "Error parsing text SAM file. RNAME '43875_x1_FemaleLG_3_FemAlePos_23.39_MaleLG_NA_MalePos_NA_duplicated_no' not found in any SQ record; File 2010AC19105_S24_L001_R2_001.sam; Line 734048\n",
      "Line: M02215:12:000000000-BB9C2:1:1114:10782:2745\t16\t43875_x1_FemaleLG_3_FemAlePos_23.39_MaleLG_NA_MalePos_NA_duplicated_no\t1\t36\t20S56M\t*\t0\t0\tAGTTCTACAGTCCGACGATCCCCACTGACACACAGAGGAGGTGAGAAACAGTGTGAAAGACCTGCAAATGTCACCC\tFGGFFGGGGFCGGGGGEEGGGFGEGFGFFE<FGGGFGGFGGGGGGGGGGEFFGGGFGFGGGFGGGGGGGGGCCCCC\tAS:i:105\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:28G27\tYT:Z:UU\tRG:Z:S_2010AC19105_S24_L001_R2_001\n",
      "Ignoring SAM validation error due to lenient parsing:\n",
      "Error parsing text SAM file. Reference sequence not found in sequence dictionary.; File 2010AC19105_S24_L001_R2_001.sam; Line 734048\n",
      "Line: M02215:12:000000000-BB9C2:1:1114:10782:2745\t16\t43875_x1_FemaleLG_3_FemAlePos_23.39_MaleLG_NA_MalePos_NA_duplicated_no\t1\t36\t20S56M\t*\t0\t0\tAGTTCTACAGTCCGACGATCCCCACTGACACACAGAGGAGGTGAGAAACAGTGTGAAAGACCTGCAAATGTCACCC\tFGGFFGGGGFCGGGGGEEGGGFGEGFGFFE<FGGGFGGFGGGGGGGGGGEFFGGGFGFGGGFGGGGGGGGGCCCCC\tAS:i:105\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:28G27\tYT:Z:UU\tRG:Z:S_2010AC19105_S24_L001_R2_001\n",
      "[Thu Oct 19 16:30:56 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.13 minutes.\n",
      "Runtime.totalMemory()=1458569216\n",
      "To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp\n",
      "Exception in thread \"main\" java.lang.IllegalArgumentException: Reference index for '43875_x1_FemaleLG_3_FemAlePos_23.39_MaleLG_NA_MalePos_NA_duplicated_no' not found in sequence dictionary.\n",
      "\tat htsjdk.samtools.SAMRecord.resolveIndexFromName(SAMRecord.java:533)\n",
      "\tat htsjdk.samtools.SAMRecord.setHeaderStrict(SAMRecord.java:1624)\n",
      "\tat htsjdk.samtools.SAMFileWriterImpl.addAlignment(SAMFileWriterImpl.java:185)\n",
      "\tat picard.sam.SortSam.doWork(SortSam.java:100)\n",
      "\tat picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:205)\n",
      "\tat picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:94)\n",
      "\tat picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:104)\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19105_S24_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19105_S24_L001_R2_001.bam.list\n",
      "INFO  16:30:58,204 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:30:58,206 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:30:58,206 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:30:58,207 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:30:58,207 HelpFormatter - [Thu Oct 19 16:30:58 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:30:58,207 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:30:58,210 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19105_S24_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19105_S24_L001_R2_001.bam.list \n",
      "INFO  16:30:58,213 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:30:58,214 HelpFormatter - Date/Time: 2017/10/19 16:30:58 \n",
      "INFO  16:30:58,214 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:30:58,214 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:30:58,219 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:30:58,359 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:30:58,366 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: SAM/BAM/CRAM file 2010AC19105_S24_L001_R2_001.bam is malformed. Please see https://software.broadinstitute.org/gatk/documentation/article?id=1317for more information. Error details: Cannot use index file with textual SAM file\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R2_001.bam -targetIntervals 2010AC19105_S24_L001_R2_001.bam.list -o 2010AC19105_S24_L001_R2_001.realigned.bam\n",
      "INFO  16:31:00,093 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:31:00,095 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:31:00,096 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:31:00,096 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:31:00,096 HelpFormatter - [Thu Oct 19 16:31:00 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:31:00,096 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:31:00,099 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R2_001.bam -targetIntervals 2010AC19105_S24_L001_R2_001.bam.list -o 2010AC19105_S24_L001_R2_001.realigned.bam \n",
      "INFO  16:31:00,102 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:31:00,103 HelpFormatter - Date/Time: 2017/10/19 16:31:00 \n",
      "INFO  16:31:00,103 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:31:00,103 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:31:00,177 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:31:00,263 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:31:00,269 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: SAM/BAM/CRAM file 2010AC19105_S24_L001_R2_001.bam is malformed. Please see https://software.broadinstitute.org/gatk/documentation/article?id=1317for more information. Error details: Cannot use index file with textual SAM file\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19559_S3_L001_R1_001 --rg SM:2010AC19559_S3_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19559_S3_L001_R1_001.fastq -S 2010AC19559_S3_L001_R1_001.sam 2> 2010AC19559_S3_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19559_S3_L001_R1_001.sam OUTPUT=2010AC19559_S3_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:32:40 PDT 2017] picard.sam.SortSam INPUT=2010AC19559_S3_L001_R1_001.sam OUTPUT=2010AC19559_S3_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:32:40 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:32:54\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:33:02 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.37 minutes.\n",
      "Runtime.totalMemory()=2019033088\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19559_S3_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19559_S3_L001_R1_001.bam.list\n",
      "INFO  16:33:04,685 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:04,687 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:33:04,687 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:33:04,687 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:33:04,688 HelpFormatter - [Thu Oct 19 16:33:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:33:04,688 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:33:04,691 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19559_S3_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19559_S3_L001_R1_001.bam.list \n",
      "INFO  16:33:04,694 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:33:04,694 HelpFormatter - Date/Time: 2017/10/19 16:33:04 \n",
      "INFO  16:33:04,694 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:04,694 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:04,699 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:33:04,843 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:33:04,849 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:33:04,889 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:33:05,055 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:33:05,088 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:33:05,089 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:33:05,089 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:33:05,089 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:33:10,336 ProgressMeter -            done     16100.0     5.0 s       5.4 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:33:10,337 ProgressMeter - Total runtime 5.25 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:33:10,337 MicroScheduler - 217 reads were filtered out during the traversal out of approximately 821526 total reads (0.03%) \n",
      "INFO  16:33:10,337 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:33:10,338 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:33:10,338 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:33:10,338 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:33:10,338 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:33:10,339 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:33:10,339 MicroScheduler -   -> 217 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:33:10,339 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:33:10,339 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:33:10,340 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R1_001.bam -targetIntervals 2010AC19559_S3_L001_R1_001.bam.list -o 2010AC19559_S3_L001_R1_001.realigned.bam\n",
      "INFO  16:33:12,073 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:12,075 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:33:12,075 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:33:12,075 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:33:12,076 HelpFormatter - [Thu Oct 19 16:33:12 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:33:12,076 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:33:12,079 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R1_001.bam -targetIntervals 2010AC19559_S3_L001_R1_001.bam.list -o 2010AC19559_S3_L001_R1_001.realigned.bam \n",
      "INFO  16:33:12,082 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:33:12,082 HelpFormatter - Date/Time: 2017/10/19 16:33:12 \n",
      "INFO  16:33:12,082 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:12,083 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:33:12,160 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:33:12,245 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:33:12,251 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:33:12,293 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:33:12,450 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:33:12,454 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:33:12,455 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:33:12,455 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:33:12,456 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:33:12,484 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:33:12,486 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:33:14,180 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-73 because there are too many reads. \n",
      "INFO  16:33:20,954 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-77 because there are too many reads. \n",
      "INFO  16:33:34,671 ProgressMeter -            done   1133993.0    22.0 s      19.0 s      100.0%    22.0 s       0.0 s \n",
      "INFO  16:33:34,671 ProgressMeter - Total runtime 22.22 secs, 0.37 min, 0.01 hours \n",
      "INFO  16:33:34,674 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1133993 total reads (0.00%) \n",
      "INFO  16:33:34,674 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:33:34,674 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19559_S3_L001_R2_001 --rg SM:2010AC19559_S3_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19559_S3_L001_R2_001.fastq -S 2010AC19559_S3_L001_R2_001.sam 2> 2010AC19559_S3_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19559_S3_L001_R2_001.sam OUTPUT=2010AC19559_S3_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:35:03 PDT 2017] picard.sam.SortSam INPUT=2010AC19559_S3_L001_R2_001.sam OUTPUT=2010AC19559_S3_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:35:03 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:35:15\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:35:25 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.37 minutes.\n",
      "Runtime.totalMemory()=2076704768\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19559_S3_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19559_S3_L001_R2_001.bam.list\n",
      "INFO  16:35:27,564 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:27,566 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:35:27,567 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:35:27,567 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:35:27,567 HelpFormatter - [Thu Oct 19 16:35:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:35:27,567 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:35:27,570 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19559_S3_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19559_S3_L001_R2_001.bam.list \n",
      "INFO  16:35:27,573 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:35:27,574 HelpFormatter - Date/Time: 2017/10/19 16:35:27 \n",
      "INFO  16:35:27,574 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:27,574 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:27,579 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:35:27,725 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:35:27,731 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:35:27,772 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:35:27,937 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:35:27,964 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:35:27,964 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:35:27,965 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:35:27,965 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:35:32,850 ProgressMeter -            done     16100.0     4.0 s       5.1 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:35:32,850 ProgressMeter - Total runtime 4.89 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:35:32,851 MicroScheduler - 387 reads were filtered out during the traversal out of approximately 826851 total reads (0.05%) \n",
      "INFO  16:35:32,851 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:35:32,851 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:35:32,851 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:35:32,852 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:35:32,852 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:35:32,852 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:35:32,852 MicroScheduler -   -> 387 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:35:32,853 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:35:32,853 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:35:32,853 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R2_001.bam -targetIntervals 2010AC19559_S3_L001_R2_001.bam.list -o 2010AC19559_S3_L001_R2_001.realigned.bam\n",
      "INFO  16:35:34,606 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:34,608 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:35:34,609 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:35:34,609 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:35:34,609 HelpFormatter - [Thu Oct 19 16:35:34 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:35:34,609 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:35:34,612 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R2_001.bam -targetIntervals 2010AC19559_S3_L001_R2_001.bam.list -o 2010AC19559_S3_L001_R2_001.realigned.bam \n",
      "INFO  16:35:34,615 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:35:34,616 HelpFormatter - Date/Time: 2017/10/19 16:35:34 \n",
      "INFO  16:35:34,616 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:34,616 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:35:34,692 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:35:34,779 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:35:34,785 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:35:34,830 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:35:34,989 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:35:34,993 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:35:34,993 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:35:34,994 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:35:34,994 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:35:35,028 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:35:35,029 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:35:36,585 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-68 because there are too many reads. \n",
      "INFO  16:35:42,303 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  16:36:00,620 ProgressMeter -            done   1133993.0    25.0 s      22.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  16:36:00,621 ProgressMeter - Total runtime 25.63 secs, 0.43 min, 0.01 hours \n",
      "INFO  16:36:00,624 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1133993 total reads (0.00%) \n",
      "INFO  16:36:00,624 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:36:00,624 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19565_S4_L001_R1_001 --rg SM:2010AC19565_S4_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19565_S4_L001_R1_001.fastq -S 2010AC19565_S4_L001_R1_001.sam 2> 2010AC19565_S4_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19565_S4_L001_R1_001.sam OUTPUT=2010AC19565_S4_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:37:20 PDT 2017] picard.sam.SortSam INPUT=2010AC19565_S4_L001_R1_001.sam OUTPUT=2010AC19565_S4_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:37:20 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:37:30\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:37:40 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.32 minutes.\n",
      "Runtime.totalMemory()=1744306176\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19565_S4_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19565_S4_L001_R1_001.bam.list\n",
      "INFO  16:37:41,997 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:41,999 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:37:41,999 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:37:42,000 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:37:42,000 HelpFormatter - [Thu Oct 19 16:37:41 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:37:42,000 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:37:42,003 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19565_S4_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19565_S4_L001_R1_001.bam.list \n",
      "INFO  16:37:42,006 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:37:42,007 HelpFormatter - Date/Time: 2017/10/19 16:37:41 \n",
      "INFO  16:37:42,007 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:42,007 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:42,012 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:37:42,159 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:37:42,165 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:37:42,202 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:37:42,364 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:37:42,390 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:37:42,390 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:37:42,391 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:37:42,391 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:37:47,123 ProgressMeter -            done     16100.0     4.0 s       4.9 m       99.5%     4.0 s       0.0 s \n",
      "INFO  16:37:47,124 ProgressMeter - Total runtime 4.73 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:37:47,124 MicroScheduler - 104 reads were filtered out during the traversal out of approximately 676218 total reads (0.02%) \n",
      "INFO  16:37:47,125 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:37:47,125 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:37:47,126 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:37:47,126 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:37:47,126 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:37:47,126 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:37:47,127 MicroScheduler -   -> 104 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:37:47,127 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:37:47,127 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:37:47,127 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R1_001.bam -targetIntervals 2010AC19565_S4_L001_R1_001.bam.list -o 2010AC19565_S4_L001_R1_001.realigned.bam\n",
      "INFO  16:37:48,932 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:48,934 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:37:48,934 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:37:48,935 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:37:48,935 HelpFormatter - [Thu Oct 19 16:37:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:37:48,935 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:37:48,938 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R1_001.bam -targetIntervals 2010AC19565_S4_L001_R1_001.bam.list -o 2010AC19565_S4_L001_R1_001.realigned.bam \n",
      "INFO  16:37:48,941 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:37:48,941 HelpFormatter - Date/Time: 2017/10/19 16:37:48 \n",
      "INFO  16:37:48,942 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:48,942 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:37:49,023 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:37:49,118 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:37:49,124 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:37:49,168 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:37:49,323 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:37:49,327 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:37:49,328 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:37:49,328 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:37:49,328 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:37:49,364 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:37:49,366 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:37:51,145 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-81 because there are too many reads. \n",
      "INFO  16:37:57,785 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:6-68 because there are too many reads. \n",
      "INFO  16:38:08,569 ProgressMeter -            done    996772.0    19.0 s      19.0 s      100.0%    19.0 s       0.0 s \n",
      "INFO  16:38:08,569 ProgressMeter - Total runtime 19.24 secs, 0.32 min, 0.01 hours \n",
      "INFO  16:38:08,572 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 996772 total reads (0.00%) \n",
      "INFO  16:38:08,572 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:38:08,572 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19565_S4_L001_R2_001 --rg SM:2010AC19565_S4_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19565_S4_L001_R2_001.fastq -S 2010AC19565_S4_L001_R2_001.sam 2> 2010AC19565_S4_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19565_S4_L001_R2_001.sam OUTPUT=2010AC19565_S4_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:39:24 PDT 2017] picard.sam.SortSam INPUT=2010AC19565_S4_L001_R2_001.sam OUTPUT=2010AC19565_S4_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:39:24 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:39:35\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:39:45 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.36 minutes.\n",
      "Runtime.totalMemory()=1913126912\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19565_S4_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19565_S4_L001_R2_001.bam.list\n",
      "INFO  16:39:47,632 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:47,634 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:39:47,635 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:39:47,635 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:39:47,635 HelpFormatter - [Thu Oct 19 16:39:47 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:39:47,635 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:39:47,639 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19565_S4_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19565_S4_L001_R2_001.bam.list \n",
      "INFO  16:39:47,642 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:39:47,642 HelpFormatter - Date/Time: 2017/10/19 16:39:47 \n",
      "INFO  16:39:47,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:47,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:47,647 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:39:47,809 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:39:47,816 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:39:47,854 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:39:48,009 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:39:48,036 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:39:48,037 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:39:48,037 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:39:48,037 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:39:53,086 ProgressMeter -            done     16100.0     5.0 s       5.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:39:53,087 ProgressMeter - Total runtime 5.05 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:39:53,087 MicroScheduler - 247 reads were filtered out during the traversal out of approximately 692177 total reads (0.04%) \n",
      "INFO  16:39:53,087 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:39:53,088 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:39:53,088 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:39:53,088 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:39:53,088 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:39:53,088 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:39:53,089 MicroScheduler -   -> 247 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:39:53,089 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:39:53,089 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:39:53,089 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R2_001.bam -targetIntervals 2010AC19565_S4_L001_R2_001.bam.list -o 2010AC19565_S4_L001_R2_001.realigned.bam\n",
      "INFO  16:39:54,905 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:54,907 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:39:54,907 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:39:54,908 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:39:54,908 HelpFormatter - [Thu Oct 19 16:39:54 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:39:54,908 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:39:54,911 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R2_001.bam -targetIntervals 2010AC19565_S4_L001_R2_001.bam.list -o 2010AC19565_S4_L001_R2_001.realigned.bam \n",
      "INFO  16:39:54,914 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:39:54,915 HelpFormatter - Date/Time: 2017/10/19 16:39:54 \n",
      "INFO  16:39:54,915 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:54,915 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:39:54,995 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:39:55,081 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:39:55,087 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:39:55,132 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:39:55,283 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:39:55,287 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:39:55,288 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:39:55,288 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:39:55,288 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:39:55,322 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:39:55,323 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:39:56,956 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-59 because there are too many reads. \n",
      "INFO  16:40:03,528 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:18-78 because there are too many reads. \n",
      "INFO  16:40:16,614 ProgressMeter -            done    996772.0    21.0 s      21.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  16:40:16,615 ProgressMeter - Total runtime 21.33 secs, 0.36 min, 0.01 hours \n",
      "INFO  16:40:16,617 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 996772 total reads (0.00%) \n",
      "INFO  16:40:16,618 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:40:16,618 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19685_S5_L001_R1_001 --rg SM:2010AC19685_S5_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19685_S5_L001_R1_001.fastq -S 2010AC19685_S5_L001_R1_001.sam 2> 2010AC19685_S5_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19685_S5_L001_R1_001.sam OUTPUT=2010AC19685_S5_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:41:44 PDT 2017] picard.sam.SortSam INPUT=2010AC19685_S5_L001_R1_001.sam OUTPUT=2010AC19685_S5_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:41:44 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:41:57\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:42:05 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=2117599232\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19685_S5_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19685_S5_L001_R1_001.bam.list\n",
      "INFO  16:42:06,832 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:06,834 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:42:06,834 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:42:06,834 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:42:06,835 HelpFormatter - [Thu Oct 19 16:42:06 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:42:06,835 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:42:06,838 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19685_S5_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19685_S5_L001_R1_001.bam.list \n",
      "INFO  16:42:06,841 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:42:06,841 HelpFormatter - Date/Time: 2017/10/19 16:42:06 \n",
      "INFO  16:42:06,842 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:06,842 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:06,847 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:42:06,977 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:42:06,983 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:42:07,023 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:42:07,186 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:42:07,210 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:42:07,211 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:42:07,211 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:42:07,212 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:42:12,576 ProgressMeter -            done     16100.0     5.0 s       5.6 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:42:12,577 ProgressMeter - Total runtime 5.37 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:42:12,577 MicroScheduler - 176 reads were filtered out during the traversal out of approximately 768249 total reads (0.02%) \n",
      "INFO  16:42:12,578 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:42:12,578 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:42:12,578 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:42:12,578 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:42:12,579 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:42:12,579 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:42:12,579 MicroScheduler -   -> 176 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:42:12,579 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:42:12,580 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:42:12,580 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R1_001.bam -targetIntervals 2010AC19685_S5_L001_R1_001.bam.list -o 2010AC19685_S5_L001_R1_001.realigned.bam\n",
      "INFO  16:42:14,330 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:14,333 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:42:14,333 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:42:14,333 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:42:14,333 HelpFormatter - [Thu Oct 19 16:42:14 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:42:14,333 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:42:14,336 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R1_001.bam -targetIntervals 2010AC19685_S5_L001_R1_001.bam.list -o 2010AC19685_S5_L001_R1_001.realigned.bam \n",
      "INFO  16:42:14,340 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:42:14,340 HelpFormatter - Date/Time: 2017/10/19 16:42:14 \n",
      "INFO  16:42:14,340 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:14,341 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:42:14,430 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:42:14,523 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:42:14,529 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:42:14,576 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:42:14,730 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:42:14,735 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:42:14,735 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:42:14,736 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:42:14,736 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:42:14,783 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:42:14,784 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:42:16,516 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-56 because there are too many reads. \n",
      "INFO  16:42:23,353 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-78 because there are too many reads. \n",
      "INFO  16:42:35,800 ProgressMeter -            done   1080941.0    21.0 s      19.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  16:42:35,800 ProgressMeter - Total runtime 21.07 secs, 0.35 min, 0.01 hours \n",
      "INFO  16:42:35,803 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1080941 total reads (0.00%) \n",
      "INFO  16:42:35,804 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:42:35,804 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19685_S5_L001_R2_001 --rg SM:2010AC19685_S5_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19685_S5_L001_R2_001.fastq -S 2010AC19685_S5_L001_R2_001.sam 2> 2010AC19685_S5_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19685_S5_L001_R2_001.sam OUTPUT=2010AC19685_S5_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:43:55 PDT 2017] picard.sam.SortSam INPUT=2010AC19685_S5_L001_R2_001.sam OUTPUT=2010AC19685_S5_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:43:55 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:44:08\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:44:16 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=2030567424\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19685_S5_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19685_S5_L001_R2_001.bam.list\n",
      "INFO  16:44:18,596 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:18,599 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:44:18,599 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:44:18,599 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:44:18,599 HelpFormatter - [Thu Oct 19 16:44:18 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:44:18,600 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:44:18,603 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19685_S5_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19685_S5_L001_R2_001.bam.list \n",
      "INFO  16:44:18,606 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:44:18,606 HelpFormatter - Date/Time: 2017/10/19 16:44:18 \n",
      "INFO  16:44:18,607 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:18,607 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:18,612 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:44:18,774 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:44:18,780 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:44:18,820 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:44:18,982 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:44:19,009 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:44:19,010 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:44:19,010 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:44:19,010 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:44:24,414 ProgressMeter -            done     16100.0     5.0 s       5.6 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:44:24,414 ProgressMeter - Total runtime 5.40 secs, 0.09 min, 0.00 hours \n",
      "INFO  16:44:24,415 MicroScheduler - 368 reads were filtered out during the traversal out of approximately 779597 total reads (0.05%) \n",
      "INFO  16:44:24,415 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:44:24,415 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:44:24,416 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:44:24,416 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:44:24,416 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:44:24,416 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:44:24,417 MicroScheduler -   -> 368 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:44:24,417 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:44:24,417 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:44:24,417 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R2_001.bam -targetIntervals 2010AC19685_S5_L001_R2_001.bam.list -o 2010AC19685_S5_L001_R2_001.realigned.bam\n",
      "INFO  16:44:26,161 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:26,163 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:44:26,163 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:44:26,164 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:44:26,164 HelpFormatter - [Thu Oct 19 16:44:26 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:44:26,164 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:44:26,167 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R2_001.bam -targetIntervals 2010AC19685_S5_L001_R2_001.bam.list -o 2010AC19685_S5_L001_R2_001.realigned.bam \n",
      "INFO  16:44:26,170 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:44:26,170 HelpFormatter - Date/Time: 2017/10/19 16:44:26 \n",
      "INFO  16:44:26,171 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:26,171 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:44:26,254 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:44:26,339 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:44:26,345 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:44:26,391 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:44:26,547 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:44:26,551 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:44:26,552 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:44:26,552 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:44:26,552 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:44:26,579 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:44:26,580 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:44:28,394 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-68 because there are too many reads. \n",
      "INFO  16:44:35,058 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:10-68 because there are too many reads. \n",
      "INFO  16:44:50,766 ProgressMeter -            done   1080941.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  16:44:50,766 ProgressMeter - Total runtime 24.21 secs, 0.40 min, 0.01 hours \n",
      "INFO  16:44:50,769 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1080941 total reads (0.00%) \n",
      "INFO  16:44:50,769 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:44:50,769 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19689_S6_L001_R1_001 --rg SM:2010AC19689_S6_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19689_S6_L001_R1_001.fastq -S 2010AC19689_S6_L001_R1_001.sam 2> 2010AC19689_S6_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19689_S6_L001_R1_001.sam OUTPUT=2010AC19689_S6_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:45:16 PDT 2017] picard.sam.SortSam INPUT=2010AC19689_S6_L001_R1_001.sam OUTPUT=2010AC19689_S6_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:45:16 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:45:19\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:45:22 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1017643008\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19689_S6_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19689_S6_L001_R1_001.bam.list\n",
      "INFO  16:45:23,798 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:23,800 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:45:23,800 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:45:23,801 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:45:23,801 HelpFormatter - [Thu Oct 19 16:45:23 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:45:23,801 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:45:23,804 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19689_S6_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19689_S6_L001_R1_001.bam.list \n",
      "INFO  16:45:23,807 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:45:23,807 HelpFormatter - Date/Time: 2017/10/19 16:45:23 \n",
      "INFO  16:45:23,808 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:23,808 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:23,813 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:45:23,948 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:45:23,954 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:45:23,993 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:45:24,150 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:45:24,176 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:45:24,176 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:45:24,177 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:45:24,177 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:45:26,483 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  16:45:26,484 ProgressMeter - Total runtime 2.31 secs, 0.04 min, 0.00 hours \n",
      "INFO  16:45:26,484 MicroScheduler - 28 reads were filtered out during the traversal out of approximately 198116 total reads (0.01%) \n",
      "INFO  16:45:26,485 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:45:26,485 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:45:26,485 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:45:26,485 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:45:26,486 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:45:26,486 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:45:26,486 MicroScheduler -   -> 28 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:45:26,487 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:45:26,487 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:45:26,487 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R1_001.bam -targetIntervals 2010AC19689_S6_L001_R1_001.bam.list -o 2010AC19689_S6_L001_R1_001.realigned.bam\n",
      "INFO  16:45:28,240 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:28,243 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:45:28,243 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:45:28,243 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:45:28,243 HelpFormatter - [Thu Oct 19 16:45:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:45:28,244 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:45:28,247 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R1_001.bam -targetIntervals 2010AC19689_S6_L001_R1_001.bam.list -o 2010AC19689_S6_L001_R1_001.realigned.bam \n",
      "INFO  16:45:28,250 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:45:28,250 HelpFormatter - Date/Time: 2017/10/19 16:45:28 \n",
      "INFO  16:45:28,250 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:28,251 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:45:28,335 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:45:28,421 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:45:28,427 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:45:28,469 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:45:28,622 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:45:28,626 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:45:28,626 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:45:28,626 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:45:28,627 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:45:28,656 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:45:28,658 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:45:38,319 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:6-75 because there are too many reads. \n",
      "INFO  16:45:41,812 ProgressMeter -            done    314578.0    13.0 s      41.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  16:45:41,812 ProgressMeter - Total runtime 13.19 secs, 0.22 min, 0.00 hours \n",
      "INFO  16:45:41,815 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 314578 total reads (0.00%) \n",
      "INFO  16:45:41,815 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:45:41,815 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19689_S6_L001_R2_001 --rg SM:2010AC19689_S6_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19689_S6_L001_R2_001.fastq -S 2010AC19689_S6_L001_R2_001.sam 2> 2010AC19689_S6_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19689_S6_L001_R2_001.sam OUTPUT=2010AC19689_S6_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:46:05 PDT 2017] picard.sam.SortSam INPUT=2010AC19689_S6_L001_R2_001.sam OUTPUT=2010AC19689_S6_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:46:05 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:46:08\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:46:11 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.11 minutes.\n",
      "Runtime.totalMemory()=990380032\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19689_S6_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19689_S6_L001_R2_001.bam.list\n",
      "INFO  16:46:13,161 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:13,163 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:46:13,163 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:46:13,163 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:46:13,164 HelpFormatter - [Thu Oct 19 16:46:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:46:13,164 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:46:13,167 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19689_S6_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19689_S6_L001_R2_001.bam.list \n",
      "INFO  16:46:13,170 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:46:13,170 HelpFormatter - Date/Time: 2017/10/19 16:46:13 \n",
      "INFO  16:46:13,171 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:13,171 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:13,176 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:46:13,322 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:46:13,328 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:46:13,367 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:46:13,535 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:46:13,560 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:46:13,561 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:46:13,561 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:46:13,561 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:46:15,788 ProgressMeter -            done     16100.0     2.0 s       2.3 m       99.5%     2.0 s       0.0 s \n",
      "INFO  16:46:15,789 ProgressMeter - Total runtime 2.23 secs, 0.04 min, 0.00 hours \n",
      "INFO  16:46:15,790 MicroScheduler - 99 reads were filtered out during the traversal out of approximately 198329 total reads (0.05%) \n",
      "INFO  16:46:15,790 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:46:15,790 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:46:15,791 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:46:15,791 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:46:15,791 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:46:15,791 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:46:15,792 MicroScheduler -   -> 99 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:46:15,792 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:46:15,792 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:46:15,792 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R2_001.bam -targetIntervals 2010AC19689_S6_L001_R2_001.bam.list -o 2010AC19689_S6_L001_R2_001.realigned.bam\n",
      "INFO  16:46:17,545 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:17,547 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:46:17,548 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:46:17,548 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:46:17,548 HelpFormatter - [Thu Oct 19 16:46:17 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:46:17,548 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:46:17,552 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R2_001.bam -targetIntervals 2010AC19689_S6_L001_R2_001.bam.list -o 2010AC19689_S6_L001_R2_001.realigned.bam \n",
      "INFO  16:46:17,555 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:46:17,555 HelpFormatter - Date/Time: 2017/10/19 16:46:17 \n",
      "INFO  16:46:17,555 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:17,556 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:46:17,641 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:46:17,728 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:46:17,734 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:46:17,777 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:46:17,930 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:46:17,934 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:46:17,935 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:46:17,935 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:46:17,936 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:46:17,963 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:46:17,964 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:46:18,435 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-84 because there are too many reads. \n",
      "INFO  16:46:20,659 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  16:46:24,657 ProgressMeter -            done    314578.0     6.0 s      21.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  16:46:24,658 ProgressMeter - Total runtime 6.72 secs, 0.11 min, 0.00 hours \n",
      "INFO  16:46:24,660 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 314578 total reads (0.00%) \n",
      "INFO  16:46:24,661 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:46:24,661 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19909_S1_L001_R1_001 --rg SM:2010AC19909_S1_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19909_S1_L001_R1_001.fastq -S 2010AC19909_S1_L001_R1_001.sam 2> 2010AC19909_S1_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19909_S1_L001_R1_001.sam OUTPUT=2010AC19909_S1_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:48:33 PDT 2017] picard.sam.SortSam INPUT=2010AC19909_S1_L001_R1_001.sam OUTPUT=2010AC19909_S1_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:48:33 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:48:54\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:49:06 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.55 minutes.\n",
      "Runtime.totalMemory()=2181038080\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19909_S1_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19909_S1_L001_R1_001.bam.list\n",
      "INFO  16:49:08,157 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:08,159 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:49:08,160 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:49:08,160 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:49:08,160 HelpFormatter - [Thu Oct 19 16:49:08 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:49:08,160 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:49:08,164 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19909_S1_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19909_S1_L001_R1_001.bam.list \n",
      "INFO  16:49:08,167 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:49:08,167 HelpFormatter - Date/Time: 2017/10/19 16:49:08 \n",
      "INFO  16:49:08,167 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:08,168 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:08,173 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:49:08,318 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:49:08,325 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:49:08,365 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:49:08,527 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:49:08,554 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:49:08,554 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:49:08,555 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:49:08,555 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:49:15,113 ProgressMeter -            done     16100.0     6.0 s       6.8 m       99.5%     6.0 s       0.0 s \n",
      "INFO  16:49:15,114 ProgressMeter - Total runtime 6.56 secs, 0.11 min, 0.00 hours \n",
      "INFO  16:49:15,114 MicroScheduler - 161 reads were filtered out during the traversal out of approximately 1104767 total reads (0.01%) \n",
      "INFO  16:49:15,115 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:49:15,115 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:49:15,115 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:49:15,115 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:49:15,116 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:49:15,116 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:49:15,116 MicroScheduler -   -> 161 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:49:15,116 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:49:15,117 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:49:15,117 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R1_001.bam -targetIntervals 2010AC19909_S1_L001_R1_001.bam.list -o 2010AC19909_S1_L001_R1_001.realigned.bam\n",
      "INFO  16:49:16,928 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:16,930 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:49:16,930 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:49:16,930 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:49:16,931 HelpFormatter - [Thu Oct 19 16:49:16 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:49:16,931 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:49:16,934 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R1_001.bam -targetIntervals 2010AC19909_S1_L001_R1_001.bam.list -o 2010AC19909_S1_L001_R1_001.realigned.bam \n",
      "INFO  16:49:16,937 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:49:16,937 HelpFormatter - Date/Time: 2017/10/19 16:49:16 \n",
      "INFO  16:49:16,937 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:16,938 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:49:17,033 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:49:17,117 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:49:17,123 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:49:17,166 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:49:17,329 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:49:17,333 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:49:17,333 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:49:17,334 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:49:17,334 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:49:17,360 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:49:17,361 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:49:19,277 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-57 because there are too many reads. \n",
      "INFO  16:49:28,475 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  16:49:47,033 ProgressMeter -            done   1596639.0    29.0 s      18.0 s      100.0%    29.0 s       0.0 s \n",
      "INFO  16:49:47,033 ProgressMeter - Total runtime 29.70 secs, 0.50 min, 0.01 hours \n",
      "INFO  16:49:47,036 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1596639 total reads (0.00%) \n",
      "INFO  16:49:47,036 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:49:47,037 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19909_S1_L001_R2_001 --rg SM:2010AC19909_S1_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19909_S1_L001_R2_001.fastq -S 2010AC19909_S1_L001_R2_001.sam 2> 2010AC19909_S1_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19909_S1_L001_R2_001.sam OUTPUT=2010AC19909_S1_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:51:42 PDT 2017] picard.sam.SortSam INPUT=2010AC19909_S1_L001_R2_001.sam OUTPUT=2010AC19909_S1_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:51:42 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:52:03\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:52:15 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.56 minutes.\n",
      "Runtime.totalMemory()=2317877248\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19909_S1_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19909_S1_L001_R2_001.bam.list\n",
      "INFO  16:52:17,329 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:17,331 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:52:17,332 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:52:17,332 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:52:17,332 HelpFormatter - [Thu Oct 19 16:52:17 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:52:17,332 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:52:17,335 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19909_S1_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19909_S1_L001_R2_001.bam.list \n",
      "INFO  16:52:17,338 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:52:17,339 HelpFormatter - Date/Time: 2017/10/19 16:52:17 \n",
      "INFO  16:52:17,339 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:17,339 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:17,344 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:52:17,476 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:52:17,482 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:52:17,520 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:52:17,694 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:52:17,721 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:52:17,722 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:52:17,722 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:52:17,722 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:52:24,406 ProgressMeter -            done     16100.0     6.0 s       6.9 m       99.5%     6.0 s       0.0 s \n",
      "INFO  16:52:24,407 ProgressMeter - Total runtime 6.69 secs, 0.11 min, 0.00 hours \n",
      "INFO  16:52:24,407 MicroScheduler - 454 reads were filtered out during the traversal out of approximately 1127583 total reads (0.04%) \n",
      "INFO  16:52:24,407 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:52:24,408 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:52:24,408 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:52:24,408 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:52:24,408 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:52:24,409 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:52:24,409 MicroScheduler -   -> 454 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:52:24,409 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:52:24,409 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:52:24,409 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R2_001.bam -targetIntervals 2010AC19909_S1_L001_R2_001.bam.list -o 2010AC19909_S1_L001_R2_001.realigned.bam\n",
      "INFO  16:52:26,146 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:26,149 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:52:26,149 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:52:26,149 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:52:26,149 HelpFormatter - [Thu Oct 19 16:52:26 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:52:26,149 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:52:26,152 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R2_001.bam -targetIntervals 2010AC19909_S1_L001_R2_001.bam.list -o 2010AC19909_S1_L001_R2_001.realigned.bam \n",
      "INFO  16:52:26,155 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:52:26,156 HelpFormatter - Date/Time: 2017/10/19 16:52:26 \n",
      "INFO  16:52:26,156 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:26,156 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:52:26,239 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:52:26,324 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:52:26,330 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:52:26,374 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:52:26,526 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:52:26,530 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:52:26,530 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:52:26,531 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:52:26,531 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:52:26,556 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:52:26,557 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:52:28,706 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-84 because there are too many reads. \n",
      "INFO  16:52:36,627 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  16:52:44,993 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:14-57 because there are too many reads. \n",
      "INFO  16:52:52,728 ProgressMeter -            done   1596639.0    26.0 s      16.0 s      100.0%    26.0 s       0.0 s \n",
      "INFO  16:52:52,729 ProgressMeter - Total runtime 26.20 secs, 0.44 min, 0.01 hours \n",
      "INFO  16:52:52,732 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1596639 total reads (0.00%) \n",
      "INFO  16:52:52,732 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:52:52,732 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19938_S7_L001_R1_001 --rg SM:2010AC19938_S7_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19938_S7_L001_R1_001.fastq -S 2010AC19938_S7_L001_R1_001.sam 2> 2010AC19938_S7_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19938_S7_L001_R1_001.sam OUTPUT=2010AC19938_S7_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:54:31 PDT 2017] picard.sam.SortSam INPUT=2010AC19938_S7_L001_R1_001.sam OUTPUT=2010AC19938_S7_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:54:31 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:54:46\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:54:54 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.38 minutes.\n",
      "Runtime.totalMemory()=2348285952\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19938_S7_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19938_S7_L001_R1_001.bam.list\n",
      "INFO  16:54:55,988 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:54:55,990 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:54:55,990 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:54:55,991 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:54:55,991 HelpFormatter - [Thu Oct 19 16:54:55 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:54:55,991 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:54:55,994 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19938_S7_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19938_S7_L001_R1_001.bam.list \n",
      "INFO  16:54:55,997 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:54:55,998 HelpFormatter - Date/Time: 2017/10/19 16:54:55 \n",
      "INFO  16:54:55,998 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:54:55,998 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:54:56,004 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:54:56,155 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:54:56,161 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:54:56,199 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:54:56,352 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:54:56,379 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:54:56,380 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:54:56,380 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:54:56,381 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:55:01,462 ProgressMeter -            done     16100.0     5.0 s       5.3 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:55:01,463 ProgressMeter - Total runtime 5.08 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:55:01,463 MicroScheduler - 196 reads were filtered out during the traversal out of approximately 825591 total reads (0.02%) \n",
      "INFO  16:55:01,464 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:55:01,464 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:55:01,464 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:55:01,464 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:55:01,465 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:55:01,465 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:55:01,465 MicroScheduler -   -> 196 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:55:01,465 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:55:01,466 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:55:01,466 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R1_001.bam -targetIntervals 2010AC19938_S7_L001_R1_001.bam.list -o 2010AC19938_S7_L001_R1_001.realigned.bam\n",
      "INFO  16:55:03,353 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:55:03,355 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:55:03,355 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:55:03,356 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:55:03,356 HelpFormatter - [Thu Oct 19 16:55:03 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:55:03,356 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:55:03,359 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R1_001.bam -targetIntervals 2010AC19938_S7_L001_R1_001.bam.list -o 2010AC19938_S7_L001_R1_001.realigned.bam \n",
      "INFO  16:55:03,362 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:55:03,363 HelpFormatter - Date/Time: 2017/10/19 16:55:03 \n",
      "INFO  16:55:03,363 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:55:03,363 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:55:03,439 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:55:03,534 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:55:03,540 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:55:03,587 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:55:03,749 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:55:03,753 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:55:03,754 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:55:03,754 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:55:03,754 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:55:03,783 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:55:03,784 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:55:05,504 IndelRealigner - Not attempting realignment in interval One_RAD56448:8-82 because there are too many reads. \n",
      "INFO  16:55:12,568 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:7-78 because there are too many reads. \n",
      "INFO  16:55:24,669 ProgressMeter -            done   1052397.0    20.0 s      19.0 s      100.0%    20.0 s       0.0 s \n",
      "INFO  16:55:24,669 ProgressMeter - Total runtime 20.92 secs, 0.35 min, 0.01 hours \n",
      "INFO  16:55:24,672 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1052397 total reads (0.00%) \n",
      "INFO  16:55:24,672 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:55:24,672 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19938_S7_L001_R2_001 --rg SM:2010AC19938_S7_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19938_S7_L001_R2_001.fastq -S 2010AC19938_S7_L001_R2_001.sam 2> 2010AC19938_S7_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19938_S7_L001_R2_001.sam OUTPUT=2010AC19938_S7_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:56:48 PDT 2017] picard.sam.SortSam INPUT=2010AC19938_S7_L001_R2_001.sam OUTPUT=2010AC19938_S7_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:56:48 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:57:01\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:57:09 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.36 minutes.\n",
      "Runtime.totalMemory()=2012217344\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19938_S7_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19938_S7_L001_R2_001.bam.list\n",
      "INFO  16:57:11,744 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:11,746 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:57:11,746 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:57:11,746 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:57:11,747 HelpFormatter - [Thu Oct 19 16:57:11 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:57:11,747 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:57:11,750 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19938_S7_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19938_S7_L001_R2_001.bam.list \n",
      "INFO  16:57:11,753 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:57:11,753 HelpFormatter - Date/Time: 2017/10/19 16:57:11 \n",
      "INFO  16:57:11,753 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:11,754 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:11,758 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:57:11,890 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:57:11,897 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:57:11,943 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  16:57:12,116 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:57:12,141 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:57:12,142 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:57:12,142 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:57:12,143 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:57:17,144 ProgressMeter -            done     16100.0     5.0 s       5.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  16:57:17,144 ProgressMeter - Total runtime 5.00 secs, 0.08 min, 0.00 hours \n",
      "INFO  16:57:17,145 MicroScheduler - 335 reads were filtered out during the traversal out of approximately 833476 total reads (0.04%) \n",
      "INFO  16:57:17,145 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:57:17,145 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:57:17,145 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:57:17,145 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:57:17,146 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:57:17,146 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:57:17,147 MicroScheduler -   -> 335 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:57:17,147 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:57:17,147 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:57:17,147 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R2_001.bam -targetIntervals 2010AC19938_S7_L001_R2_001.bam.list -o 2010AC19938_S7_L001_R2_001.realigned.bam\n",
      "INFO  16:57:18,880 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:18,882 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:57:18,883 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:57:18,883 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:57:18,883 HelpFormatter - [Thu Oct 19 16:57:18 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:57:18,883 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:57:18,887 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R2_001.bam -targetIntervals 2010AC19938_S7_L001_R2_001.bam.list -o 2010AC19938_S7_L001_R2_001.realigned.bam \n",
      "INFO  16:57:18,890 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:57:18,891 HelpFormatter - Date/Time: 2017/10/19 16:57:18 \n",
      "INFO  16:57:18,891 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:18,891 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:57:18,967 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:57:19,053 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:57:19,059 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:57:19,104 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:57:19,260 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:57:19,264 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:57:19,265 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:57:19,265 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:57:19,265 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:57:19,291 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:57:19,292 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:57:21,055 IndelRealigner - Not attempting realignment in interval One_RAD56448:18-61 because there are too many reads. \n",
      "INFO  16:57:26,957 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  16:57:43,235 ProgressMeter -            done   1052397.0    23.0 s      22.0 s      100.0%    23.0 s       0.0 s \n",
      "INFO  16:57:43,235 ProgressMeter - Total runtime 23.97 secs, 0.40 min, 0.01 hours \n",
      "INFO  16:57:43,238 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1052397 total reads (0.00%) \n",
      "INFO  16:57:43,238 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:57:43,238 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19940_S8_L001_R1_001 --rg SM:2010AC19940_S8_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19940_S8_L001_R1_001.fastq -S 2010AC19940_S8_L001_R1_001.sam 2> 2010AC19940_S8_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19940_S8_L001_R1_001.sam OUTPUT=2010AC19940_S8_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 16:58:51 PDT 2017] picard.sam.SortSam INPUT=2010AC19940_S8_L001_R1_001.sam OUTPUT=2010AC19940_S8_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 16:58:51 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 16:58:58\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 16:59:04 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.22 minutes.\n",
      "Runtime.totalMemory()=1533018112\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19940_S8_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC19940_S8_L001_R1_001.bam.list\n",
      "INFO  16:59:06,675 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:06,678 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:59:06,678 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:59:06,678 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:59:06,678 HelpFormatter - [Thu Oct 19 16:59:06 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:59:06,679 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:59:06,682 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19940_S8_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC19940_S8_L001_R1_001.bam.list \n",
      "INFO  16:59:06,685 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:59:06,686 HelpFormatter - Date/Time: 2017/10/19 16:59:06 \n",
      "INFO  16:59:06,686 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:06,686 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:06,691 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:59:06,824 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:59:06,831 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:59:06,870 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:59:07,031 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:59:07,058 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:59:07,059 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:59:07,059 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:59:07,060 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:59:10,879 ProgressMeter -            done     16100.0     3.0 s       4.0 m       99.5%     3.0 s       0.0 s \n",
      "INFO  16:59:10,880 ProgressMeter - Total runtime 3.82 secs, 0.06 min, 0.00 hours \n",
      "INFO  16:59:10,880 MicroScheduler - 122 reads were filtered out during the traversal out of approximately 551051 total reads (0.02%) \n",
      "INFO  16:59:10,881 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:59:10,881 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  16:59:10,881 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  16:59:10,882 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  16:59:10,882 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  16:59:10,882 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  16:59:10,882 MicroScheduler -   -> 122 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  16:59:10,883 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  16:59:10,883 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  16:59:10,883 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R1_001.bam -targetIntervals 2010AC19940_S8_L001_R1_001.bam.list -o 2010AC19940_S8_L001_R1_001.realigned.bam\n",
      "INFO  16:59:12,662 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:12,664 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  16:59:12,664 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:59:12,664 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  16:59:12,665 HelpFormatter - [Thu Oct 19 16:59:12 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  16:59:12,665 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  16:59:12,668 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R1_001.bam -targetIntervals 2010AC19940_S8_L001_R1_001.bam.list -o 2010AC19940_S8_L001_R1_001.realigned.bam \n",
      "INFO  16:59:12,671 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  16:59:12,671 HelpFormatter - Date/Time: 2017/10/19 16:59:12 \n",
      "INFO  16:59:12,671 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:12,672 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  16:59:12,768 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:59:12,854 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  16:59:12,860 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  16:59:12,903 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  16:59:13,057 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  16:59:13,061 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:59:13,062 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:59:13,062 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:59:13,062 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  16:59:13,091 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  16:59:13,092 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  16:59:13,802 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-59 because there are too many reads. \n",
      "INFO  16:59:19,058 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-75 because there are too many reads. \n",
      "INFO  16:59:27,120 ProgressMeter -            done    719695.0    14.0 s      19.0 s      100.0%    14.0 s       0.0 s \n",
      "INFO  16:59:27,121 ProgressMeter - Total runtime 14.06 secs, 0.23 min, 0.00 hours \n",
      "INFO  16:59:27,123 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 719695 total reads (0.00%) \n",
      "INFO  16:59:27,124 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  16:59:27,124 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC19940_S8_L001_R2_001 --rg SM:2010AC19940_S8_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19940_S8_L001_R2_001.fastq -S 2010AC19940_S8_L001_R2_001.sam 2> 2010AC19940_S8_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19940_S8_L001_R2_001.sam OUTPUT=2010AC19940_S8_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 17:00:26 PDT 2017] picard.sam.SortSam INPUT=2010AC19940_S8_L001_R2_001.sam OUTPUT=2010AC19940_S8_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 17:00:26 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 17:00:33\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 17:00:40 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.24 minutes.\n",
      "Runtime.totalMemory()=1543503872\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19940_S8_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC19940_S8_L001_R2_001.bam.list\n",
      "INFO  17:00:41,987 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:41,989 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:00:41,989 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:00:41,989 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:00:41,990 HelpFormatter - [Thu Oct 19 17:00:41 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:00:41,990 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:00:41,993 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19940_S8_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC19940_S8_L001_R2_001.bam.list \n",
      "INFO  17:00:41,996 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:00:41,996 HelpFormatter - Date/Time: 2017/10/19 17:00:41 \n",
      "INFO  17:00:41,997 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:41,997 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:42,002 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:00:42,156 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  17:00:42,162 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:00:42,201 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  17:00:42,356 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:00:42,383 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:00:42,383 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:00:42,384 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:00:42,384 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  17:00:46,423 ProgressMeter -            done     16100.0     4.0 s       4.2 m       99.5%     4.0 s       0.0 s \n",
      "INFO  17:00:46,424 ProgressMeter - Total runtime 4.04 secs, 0.07 min, 0.00 hours \n",
      "INFO  17:00:46,424 MicroScheduler - 271 reads were filtered out during the traversal out of approximately 549465 total reads (0.05%) \n",
      "INFO  17:00:46,425 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:00:46,425 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  17:00:46,425 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  17:00:46,426 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  17:00:46,426 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  17:00:46,426 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  17:00:46,427 MicroScheduler -   -> 271 reads (0.05% of total) failing MappingQualityZeroFilter \n",
      "INFO  17:00:46,427 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  17:00:46,427 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  17:00:46,427 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R2_001.bam -targetIntervals 2010AC19940_S8_L001_R2_001.bam.list -o 2010AC19940_S8_L001_R2_001.realigned.bam\n",
      "INFO  17:00:48,181 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:48,183 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:00:48,183 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:00:48,183 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:00:48,184 HelpFormatter - [Thu Oct 19 17:00:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:00:48,184 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:00:48,187 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R2_001.bam -targetIntervals 2010AC19940_S8_L001_R2_001.bam.list -o 2010AC19940_S8_L001_R2_001.realigned.bam \n",
      "INFO  17:00:48,190 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:00:48,190 HelpFormatter - Date/Time: 2017/10/19 17:00:48 \n",
      "INFO  17:00:48,191 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:48,191 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:00:48,286 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:00:48,370 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  17:00:48,376 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:00:48,420 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  17:00:48,575 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:00:48,580 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:00:48,580 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:00:48,580 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:00:48,581 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  17:00:48,611 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  17:00:48,612 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  17:00:49,306 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-61 because there are too many reads. \n",
      "INFO  17:00:54,506 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:23-63 because there are too many reads. \n",
      "INFO  17:01:04,613 ProgressMeter -            done    719695.0    16.0 s      22.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  17:01:04,614 ProgressMeter - Total runtime 16.03 secs, 0.27 min, 0.00 hours \n",
      "INFO  17:01:04,617 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 719695 total reads (0.00%) \n",
      "INFO  17:01:04,617 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:01:04,617 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC20099_S2_L001_R1_001 --rg SM:2010AC20099_S2_L001_R1_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC20099_S2_L001_R1_001.fastq -S 2010AC20099_S2_L001_R1_001.sam 2> 2010AC20099_S2_L001_R1_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC20099_S2_L001_R1_001.sam OUTPUT=2010AC20099_S2_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 17:02:57 PDT 2017] picard.sam.SortSam INPUT=2010AC20099_S2_L001_R1_001.sam OUTPUT=2010AC20099_S2_L001_R1_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 17:02:57 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 17:03:12\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 17:03:25 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.47 minutes.\n",
      "Runtime.totalMemory()=2028470272\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC20099_S2_L001_R1_001.bam -R 233_Amplicons.fa -o  2010AC20099_S2_L001_R1_001.bam.list\n",
      "INFO  17:03:27,343 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:27,345 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:03:27,345 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:03:27,345 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:03:27,346 HelpFormatter - [Thu Oct 19 17:03:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:03:27,346 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:03:27,349 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC20099_S2_L001_R1_001.bam -R 233_Amplicons.fa -o 2010AC20099_S2_L001_R1_001.bam.list \n",
      "INFO  17:03:27,352 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:03:27,352 HelpFormatter - Date/Time: 2017/10/19 17:03:27 \n",
      "INFO  17:03:27,353 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:27,353 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:27,358 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:03:27,498 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  17:03:27,504 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:03:27,545 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  17:03:27,713 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:03:27,740 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:03:27,740 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:03:27,741 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:03:27,741 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  17:03:34,041 ProgressMeter -            done     16100.0     6.0 s       6.5 m       99.5%     6.0 s       0.0 s \n",
      "INFO  17:03:34,041 ProgressMeter - Total runtime 6.30 secs, 0.11 min, 0.00 hours \n",
      "INFO  17:03:34,042 MicroScheduler - 298 reads were filtered out during the traversal out of approximately 974660 total reads (0.03%) \n",
      "INFO  17:03:34,042 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:03:34,042 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  17:03:34,042 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  17:03:34,043 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  17:03:34,043 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  17:03:34,043 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  17:03:34,043 MicroScheduler -   -> 298 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  17:03:34,043 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  17:03:34,044 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  17:03:34,044 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R1_001.bam -targetIntervals 2010AC20099_S2_L001_R1_001.bam.list -o 2010AC20099_S2_L001_R1_001.realigned.bam\n",
      "INFO  17:03:35,851 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:35,853 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:03:35,853 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:03:35,854 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:03:35,854 HelpFormatter - [Thu Oct 19 17:03:35 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:03:35,854 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:03:35,857 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R1_001.bam -targetIntervals 2010AC20099_S2_L001_R1_001.bam.list -o 2010AC20099_S2_L001_R1_001.realigned.bam \n",
      "INFO  17:03:35,860 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:03:35,861 HelpFormatter - Date/Time: 2017/10/19 17:03:35 \n",
      "INFO  17:03:35,861 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:35,861 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:03:35,948 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:03:36,034 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  17:03:36,040 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:03:36,084 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  17:03:36,242 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:03:36,246 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:03:36,247 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:03:36,247 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:03:36,247 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  17:03:36,274 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  17:03:36,276 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  17:03:38,238 IndelRealigner - Not attempting realignment in interval One_RAD56448:10-60 because there are too many reads. \n",
      "INFO  17:03:47,074 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  17:04:03,112 ProgressMeter -            done   1468971.0    26.0 s      18.0 s      100.0%    26.0 s       0.0 s \n",
      "INFO  17:04:03,112 ProgressMeter - Total runtime 26.87 secs, 0.45 min, 0.01 hours \n",
      "INFO  17:04:03,115 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1468971 total reads (0.00%) \n",
      "INFO  17:04:03,115 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:04:03,115 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC20099_S2_L001_R2_001 --rg SM:2010AC20099_S2_L001_R2_001 --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC20099_S2_L001_R2_001.fastq -S 2010AC20099_S2_L001_R2_001.sam 2> 2010AC20099_S2_L001_R2_001.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC20099_S2_L001_R2_001.sam OUTPUT=2010AC20099_S2_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Thu Oct 19 17:05:45 PDT 2017] picard.sam.SortSam INPUT=2010AC20099_S2_L001_R2_001.sam OUTPUT=2010AC20099_S2_L001_R2_001.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Oct 19 17:05:45 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-19 17:05:59\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Thu Oct 19 17:06:12 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.45 minutes.\n",
      "Runtime.totalMemory()=1990721536\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC20099_S2_L001_R2_001.bam -R 233_Amplicons.fa -o  2010AC20099_S2_L001_R2_001.bam.list\n",
      "INFO  17:06:14,509 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:14,511 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:06:14,511 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:06:14,512 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:06:14,512 HelpFormatter - [Thu Oct 19 17:06:14 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:06:14,512 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:06:14,515 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC20099_S2_L001_R2_001.bam -R 233_Amplicons.fa -o 2010AC20099_S2_L001_R2_001.bam.list \n",
      "INFO  17:06:14,518 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:06:14,518 HelpFormatter - Date/Time: 2017/10/19 17:06:14 \n",
      "INFO  17:06:14,519 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:14,519 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:14,524 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:06:14,663 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  17:06:14,669 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:06:14,707 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  17:06:14,868 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:06:14,894 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:06:14,895 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:06:14,895 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:06:14,896 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  17:06:21,103 ProgressMeter -            done     16100.0     6.0 s       6.4 m       99.5%     6.0 s       0.0 s \n",
      "INFO  17:06:21,103 ProgressMeter - Total runtime 6.21 secs, 0.10 min, 0.00 hours \n",
      "INFO  17:06:21,104 MicroScheduler - 331 reads were filtered out during the traversal out of approximately 997849 total reads (0.03%) \n",
      "INFO  17:06:21,104 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:06:21,105 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  17:06:21,105 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  17:06:21,105 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  17:06:21,105 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  17:06:21,106 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  17:06:21,106 MicroScheduler -   -> 331 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  17:06:21,106 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  17:06:21,107 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  17:06:21,107 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R2_001.bam -targetIntervals 2010AC20099_S2_L001_R2_001.bam.list -o 2010AC20099_S2_L001_R2_001.realigned.bam\n",
      "INFO  17:06:22,883 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:22,885 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  17:06:22,885 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  17:06:22,886 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  17:06:22,886 HelpFormatter - [Thu Oct 19 17:06:22 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  17:06:22,886 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  17:06:22,889 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R2_001.bam -targetIntervals 2010AC20099_S2_L001_R2_001.bam.list -o 2010AC20099_S2_L001_R2_001.realigned.bam \n",
      "INFO  17:06:22,892 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  17:06:22,893 HelpFormatter - Date/Time: 2017/10/19 17:06:22 \n",
      "INFO  17:06:22,893 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:22,893 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  17:06:22,977 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  17:06:23,061 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  17:06:23,067 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  17:06:23,117 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  17:06:23,274 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  17:06:23,278 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  17:06:23,278 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  17:06:23,279 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  17:06:23,279 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  17:06:23,306 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  17:06:23,307 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  17:06:25,189 IndelRealigner - Not attempting realignment in interval One_RAD56448:18-77 because there are too many reads. \n",
      "INFO  17:06:33,367 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  17:06:40,185 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:6-57 because there are too many reads. \n",
      "INFO  17:06:48,528 ProgressMeter -            done   1468971.0    25.0 s      17.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  17:06:48,528 ProgressMeter - Total runtime 25.25 secs, 0.42 min, 0.01 hours \n",
      "INFO  17:06:48,531 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1468971 total reads (0.00%) \n",
      "INFO  17:06:48,531 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  17:06:48,531 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "!perl GATK_pipeline_SM.pl SampleFile.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl GATK_pipeline_SM_printCommands.pl SampleFile.txt >GATK_pipeline_commands.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out\n"
     ]
    }
   ],
   "source": [
    "cd /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC20099_S2_L001_R1_001.sam"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC19096_S21_L001_R1_001.sam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Counting reads per locus for an individual with a tiny amount of data:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true,
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "!samtools depth 2010AC19006_S10_L001_R1_001.bam > samtoolsdepth_2010AC19006.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Counting reads per locus for an individual with a large data file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!samtools depth 2010AC19105_S24_L001_R1_001.bam > samtoolsdepth_2010AC19105.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Apparently needed to run Trimmomatic at the beginning of this pipeline, so that's why I've been getting these weird fish-eye plots"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "![fish-eye-plot](/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Rplot.png)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
