{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Mistakenly ran through the pipeline without trimmomatic, this is a second try hopefully correctly:"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Trimmomatic:"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "downloaded binary trimmomatic and placed in directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data\n"
     ]
    }
   ],
   "source": [
    "cd /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Example of one line of code, however I am going to do with all individuals:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "java -jar ../../Programs/Trimmomatic-0.36/trimmomatic-0.32.jar PE -phred33 -trimlog S05.trimlog.txt \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/OtsTest2_KKOGR07_0011_R1.fastq \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/OtsTest2_KKOGR07_0011_R2.fastq \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/OtsTest2_KKOGR07_0011_R1paired.fq \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/unpaired/OtsTest2_KKOGR07_0011_R1unpaired.fq \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/OtsTest2_KKOGR07_0011_R2paired.fq \\\n",
    "~/Desktop/katmai/WORK/PASCAL/OtsTest2/allBarcodeSplit/unpaired/OtsTest2_KKOGR07_0011_R2unpaired.fq \\\n",
    "LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19003_S9_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19003_S9_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19003_S9_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19003_S9_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19003_S9_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19003_S9_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 931392 Both Surviving: 904433 (97.11%) Forward Only Surviving: 16214 (1.74%) Reverse Only Surviving: 8043 (0.86%) Dropped: 2702 (0.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19006_S10_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19006_S10_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19006_S10_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19006_S10_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19006_S10_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19006_S10_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 264331 Both Surviving: 258333 (97.73%) Forward Only Surviving: 4005 (1.52%) Reverse Only Surviving: 1425 (0.54%) Dropped: 568 (0.21%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19008_S11_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19008_S11_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19008_S11_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19008_S11_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19008_S11_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19008_S11_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1301783 Both Surviving: 1270789 (97.62%) Forward Only Surviving: 19310 (1.48%) Reverse Only Surviving: 8623 (0.66%) Dropped: 3061 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19015_S12_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19015_S12_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19015_S12_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19015_S12_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19015_S12_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19015_S12_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 252023 Both Surviving: 244888 (97.17%) Forward Only Surviving: 5060 (2.01%) Reverse Only Surviving: 1416 (0.56%) Dropped: 659 (0.26%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19041_S17_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19041_S17_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19041_S17_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19041_S17_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19041_S17_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19041_S17_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 288358 Both Surviving: 281245 (97.53%) Forward Only Surviving: 4581 (1.59%) Reverse Only Surviving: 1901 (0.66%) Dropped: 631 (0.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19047_S13_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19047_S13_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19047_S13_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19047_S13_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19047_S13_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19047_S13_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1465140 Both Surviving: 1433096 (97.81%) Forward Only Surviving: 20607 (1.41%) Reverse Only Surviving: 8135 (0.56%) Dropped: 3302 (0.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19049_S14_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19049_S14_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19049_S14_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19049_S14_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19049_S14_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19049_S14_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 387460 Both Surviving: 378287 (97.63%) Forward Only Surviving: 5958 (1.54%) Reverse Only Surviving: 2308 (0.60%) Dropped: 907 (0.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19050_S15_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19050_S15_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19050_S15_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19050_S15_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19050_S15_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19050_S15_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 689484 Both Surviving: 672628 (97.56%) Forward Only Surviving: 10617 (1.54%) Reverse Only Surviving: 4666 (0.68%) Dropped: 1573 (0.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19057_S16_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19057_S16_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19057_S16_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19057_S16_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 712444 Both Surviving: 694333 (97.46%) Forward Only Surviving: 11165 (1.57%) Reverse Only Surviving: 5206 (0.73%) Dropped: 1740 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19091_S18_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19091_S18_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19091_S18_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19091_S18_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19091_S18_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19091_S18_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 989427 Both Surviving: 966717 (97.70%) Forward Only Surviving: 13923 (1.41%) Reverse Only Surviving: 6653 (0.67%) Dropped: 2134 (0.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19096_S21_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19096_S21_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19096_S21_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19096_S21_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19096_S21_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19096_S21_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 735422 Both Surviving: 716968 (97.49%) Forward Only Surviving: 10829 (1.47%) Reverse Only Surviving: 5755 (0.78%) Dropped: 1870 (0.25%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19098_S22_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19098_S22_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19098_S22_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19098_S22_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19098_S22_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19098_S22_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 821460 Both Surviving: 803691 (97.84%) Forward Only Surviving: 10978 (1.34%) Reverse Only Surviving: 4974 (0.61%) Dropped: 1817 (0.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19099_S19_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19099_S19_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19099_S19_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19099_S19_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19099_S19_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19099_S19_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1106797 Both Surviving: 1078742 (97.47%) Forward Only Surviving: 17202 (1.55%) Reverse Only Surviving: 8251 (0.75%) Dropped: 2602 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19101_S23_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19101_S23_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19101_S23_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19101_S23_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19101_S23_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19101_S23_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1239009 Both Surviving: 1209501 (97.62%) Forward Only Surviving: 17532 (1.42%) Reverse Only Surviving: 8926 (0.72%) Dropped: 3050 (0.25%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19102_S20_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19102_S20_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19102_S20_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19102_S20_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19102_S20_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19102_S20_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1126561 Both Surviving: 1100757 (97.71%) Forward Only Surviving: 15096 (1.34%) Reverse Only Surviving: 8120 (0.72%) Dropped: 2588 (0.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19105_S24_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19105_S24_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19105_S24_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19105_S24_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 2115329 Both Surviving: 2065514 (97.65%) Forward Only Surviving: 30756 (1.45%) Reverse Only Surviving: 12765 (0.60%) Dropped: 6294 (0.30%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19559_S3_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19559_S3_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19559_S3_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19559_S3_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19559_S3_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19559_S3_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1133993 Both Surviving: 1106227 (97.55%) Forward Only Surviving: 16675 (1.47%) Reverse Only Surviving: 8406 (0.74%) Dropped: 2685 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19565_S4_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19565_S4_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19565_S4_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19565_S4_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19565_S4_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19565_S4_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 996772 Both Surviving: 973818 (97.70%) Forward Only Surviving: 15320 (1.54%) Reverse Only Surviving: 5414 (0.54%) Dropped: 2220 (0.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19685_S5_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19685_S5_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19685_S5_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19685_S5_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19685_S5_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19685_S5_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1080941 Both Surviving: 1053330 (97.45%) Forward Only Surviving: 17418 (1.61%) Reverse Only Surviving: 7348 (0.68%) Dropped: 2845 (0.26%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19689_S6_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19689_S6_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19689_S6_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19689_S6_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 314578 Both Surviving: 307045 (97.61%) Forward Only Surviving: 4674 (1.49%) Reverse Only Surviving: 2112 (0.67%) Dropped: 747 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19909_S1_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19909_S1_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19909_S1_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19909_S1_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19909_S1_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19909_S1_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1596639 Both Surviving: 1560253 (97.72%) Forward Only Surviving: 23212 (1.45%) Reverse Only Surviving: 9389 (0.59%) Dropped: 3785 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19938_S7_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19938_S7_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19938_S7_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19938_S7_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19938_S7_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19938_S7_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1052397 Both Surviving: 1025650 (97.46%) Forward Only Surviving: 16299 (1.55%) Reverse Only Surviving: 7821 (0.74%) Dropped: 2627 (0.25%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19940_S8_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19940_S8_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19940_S8_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19940_S8_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19940_S8_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19940_S8_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 719695 Both Surviving: 701343 (97.45%) Forward Only Surviving: 11343 (1.58%) Reverse Only Surviving: 5248 (0.73%) Dropped: 1761 (0.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "TrimmomaticPE: Started with arguments:\n",
      " -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC20099_S2_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC20099_S2_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC20099_S2_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC20099_S2_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36\n",
      "Multiple cores found: Using 4 threads\n",
      "java.io.FileNotFoundException: /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Notebooks/TruSeq3-PE.fa (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:110)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:71)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:32)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.createTrimmers(Trimmomatic.java:59)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:536)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:80)\n",
      "Input Read Pairs: 1468971 Both Surviving: 1434928 (97.68%) Forward Only Surviving: 20948 (1.43%) Reverse Only Surviving: 9495 (0.65%) Dropped: 3600 (0.25%)\n",
      "TrimmomaticPE: Completed successfully\n"
     ]
    }
   ],
   "source": [
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19003_S9_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19003_S9_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19003_S9_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19003_S9_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19003_S9_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19003_S9_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19006_S10_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19006_S10_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19006_S10_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19006_S10_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19006_S10_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19006_S10_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19008_S11_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19008_S11_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19008_S11_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19008_S11_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19008_S11_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19008_S11_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19015_S12_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19015_S12_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19015_S12_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19015_S12_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19015_S12_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19015_S12_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19041_S17_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19041_S17_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19041_S17_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19041_S17_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19041_S17_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19041_S17_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19047_S13_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19047_S13_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19047_S13_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19047_S13_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19047_S13_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19047_S13_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19049_S14_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19049_S14_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19049_S14_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19049_S14_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19049_S14_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19049_S14_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19050_S15_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19050_S15_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19050_S15_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19050_S15_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19050_S15_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19050_S15_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19057_S16_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19057_S16_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19057_S16_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19057_S16_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19091_S18_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19091_S18_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19091_S18_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19091_S18_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19091_S18_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19091_S18_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19096_S21_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19096_S21_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19096_S21_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19096_S21_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19096_S21_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19096_S21_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19098_S22_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19098_S22_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19098_S22_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19098_S22_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19098_S22_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19098_S22_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19099_S19_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19099_S19_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19099_S19_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19099_S19_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19099_S19_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19099_S19_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19101_S23_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19101_S23_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19101_S23_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19101_S23_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19101_S23_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19101_S23_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19102_S20_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19102_S20_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19102_S20_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19102_S20_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19102_S20_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19102_S20_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19105_S24_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19105_S24_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19105_S24_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19105_S24_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19559_S3_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19559_S3_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19559_S3_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19559_S3_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19559_S3_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19559_S3_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19565_S4_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19565_S4_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19565_S4_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19565_S4_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19565_S4_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19565_S4_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19685_S5_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19685_S5_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19685_S5_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19685_S5_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19685_S5_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19685_S5_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19689_S6_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19689_S6_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19689_S6_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19689_S6_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19909_S1_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19909_S1_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19909_S1_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19909_S1_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19909_S1_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19909_S1_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19938_S7_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19938_S7_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19938_S7_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19938_S7_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19938_S7_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19938_S7_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19940_S8_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC19940_S8_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19940_S8_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19940_S8_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19940_S8_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC19940_S8_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt\n",
    "!java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC20099_S2_L001_R1_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Unzipped_Data/2010AC20099_S2_L001_R2_001.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC20099_S2_L001_R1_001_unpaired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/unpaired/2010AC20099_S2_L001_R1_001_unpaired.fastq ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36>>/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/TrimmomaticStdOut.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Running GATK Pipeline out of Trimmomatic_Out directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out\n"
     ]
    }
   ],
   "source": [
    "cd ../Data/Trimmomatic_Out/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bowtie2-build 233_Amplicons.fa GATKRefTrim\n",
      "Building a SMALL index\n",
      "bwa index -p GATKRefTrim 233_Amplicons.fa\n",
      "[bwa_index] Pack FASTA... 0.00 sec\n",
      "[bwa_index] Construct BWT for the packed sequence...\n",
      "[bwa_index] 0.00 seconds elapse.\n",
      "[bwa_index] Update BWT... 0.00 sec\n",
      "[bwa_index] Pack forward-only FASTA... 0.00 sec\n",
      "[bwa_index] Construct SA from BWT and Occ... 0.00 sec\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa index -p GATKRefTrim 233_Amplicons.fa\n",
      "[main] Real time: 0.033 sec; CPU: 0.008 sec\n",
      "samtools faidx 233_Amplicons.fa\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar CreateSequenceDictionary REFERENCE=233_Amplicons.fa OUTPUT=233_Amplicons.dict\n",
      "[Tue Oct 24 12:27:41 PDT 2017] picard.sam.CreateSequenceDictionary REFERENCE=233_Amplicons.fa OUTPUT=233_Amplicons.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:27:41 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "[Tue Oct 24 12:27:41 PDT 2017] picard.sam.CreateSequenceDictionary done. Elapsed time: 0.01 minutes.\n",
      "Runtime.totalMemory()=331874304\n",
      " -S 2010AC19003_S9_L001_R1_001_paired.sam 2> 2010AC19003_S9_L001_R1_001_paired.txt:2010AC19003_S9_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19003_S9_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19003_S9_L001_R1_001_paired.sam OUTPUT=2010AC19003_S9_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:29:01 PDT 2017] picard.sam.SortSam INPUT=2010AC19003_S9_L001_R1_001_paired.sam OUTPUT=2010AC19003_S9_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:29:01 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:29:10\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:29:19 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.30 minutes.\n",
      "Runtime.totalMemory()=1740111872\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19003_S9_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19003_S9_L001_R1_001_paired.bam.list\n",
      "INFO  12:29:21,364 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:21,367 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:29:21,367 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:29:21,368 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:29:21,368 HelpFormatter - [Tue Oct 24 12:29:21 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:29:21,368 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:29:21,372 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19003_S9_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19003_S9_L001_R1_001_paired.bam.list \n",
      "INFO  12:29:21,375 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:29:21,375 HelpFormatter - Date/Time: 2017/10/24 12:29:21 \n",
      "INFO  12:29:21,376 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:21,376 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:21,381 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:29:21,528 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:29:21,534 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:29:21,582 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:29:21,799 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:29:21,829 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:29:21,829 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:29:21,829 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:29:21,830 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:29:26,868 ProgressMeter -            done     16100.0     5.0 s       5.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  12:29:26,868 ProgressMeter - Total runtime 5.04 secs, 0.08 min, 0.00 hours \n",
      "INFO  12:29:26,869 MicroScheduler - 288 reads were filtered out during the traversal out of approximately 680180 total reads (0.04%) \n",
      "INFO  12:29:26,869 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:29:26,870 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:29:26,870 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:29:26,870 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:29:26,871 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:29:26,871 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:29:26,871 MicroScheduler -   -> 288 reads (0.04% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:29:26,871 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:29:26,872 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:29:26,872 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R1_001_paired.bam -targetIntervals 2010AC19003_S9_L001_R1_001_paired.bam.list -o 2010AC19003_S9_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:29:28,765 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:28,767 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:29:28,768 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:29:28,768 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:29:28,768 HelpFormatter - [Tue Oct 24 12:29:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:29:28,768 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:29:28,771 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R1_001_paired.bam -targetIntervals 2010AC19003_S9_L001_R1_001_paired.bam.list -o 2010AC19003_S9_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:29:28,775 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:29:28,775 HelpFormatter - Date/Time: 2017/10/24 12:29:28 \n",
      "INFO  12:29:28,775 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:28,775 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:29:28,880 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:29:28,973 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:29:28,979 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:29:29,026 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:29:29,192 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:29:29,196 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:29:29,197 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:29:29,197 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:29:29,197 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:29:29,225 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:29:29,227 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:29:30,922 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-57 because there are too many reads. \n",
      "INFO  12:29:37,562 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  12:29:47,316 ProgressMeter -            done    904433.0    18.0 s      20.0 s      100.0%    18.0 s       0.0 s \n",
      "INFO  12:29:47,316 ProgressMeter - Total runtime 18.12 secs, 0.30 min, 0.01 hours \n",
      "INFO  12:29:47,319 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 904433 total reads (0.00%) \n",
      "INFO  12:29:47,319 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:29:47,320 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19006_S10_L001_R1_001_paired.sam 2> 2010AC19006_S10_L001_R1_001_paired.txt2010AC19006_S10_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19006_S10_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19006_S10_L001_R1_001_paired.sam OUTPUT=2010AC19006_S10_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:30:06 PDT 2017] picard.sam.SortSam INPUT=2010AC19006_S10_L001_R1_001_paired.sam OUTPUT=2010AC19006_S10_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:30:06 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:30:09\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:30:11 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.08 minutes.\n",
      "Runtime.totalMemory()=967311360\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19006_S10_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19006_S10_L001_R1_001_paired.bam.list\n",
      "INFO  12:30:13,434 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:13,436 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:30:13,436 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:30:13,436 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:30:13,437 HelpFormatter - [Tue Oct 24 12:30:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:30:13,437 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:30:13,440 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19006_S10_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19006_S10_L001_R1_001_paired.bam.list \n",
      "INFO  12:30:13,443 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:30:13,443 HelpFormatter - Date/Time: 2017/10/24 12:30:13 \n",
      "INFO  12:30:13,444 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:13,444 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:13,449 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:30:13,598 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:30:13,604 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:30:13,642 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:30:13,816 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:30:13,842 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:30:13,842 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:30:13,843 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:30:13,843 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:30:16,207 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  12:30:16,208 ProgressMeter - Total runtime 2.37 secs, 0.04 min, 0.00 hours \n",
      "INFO  12:30:16,208 MicroScheduler - 7 reads were filtered out during the traversal out of approximately 174067 total reads (0.00%) \n",
      "INFO  12:30:16,209 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:30:16,209 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:30:16,209 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:30:16,209 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:30:16,210 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:30:16,210 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:30:16,210 MicroScheduler -   -> 7 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:30:16,210 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:30:16,211 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:30:16,211 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R1_001_paired.bam -targetIntervals 2010AC19006_S10_L001_R1_001_paired.bam.list -o 2010AC19006_S10_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:30:17,930 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:17,932 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:30:17,933 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:30:17,933 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:30:17,933 HelpFormatter - [Tue Oct 24 12:30:17 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:30:17,933 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:30:17,937 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R1_001_paired.bam -targetIntervals 2010AC19006_S10_L001_R1_001_paired.bam.list -o 2010AC19006_S10_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:30:17,940 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:30:17,940 HelpFormatter - Date/Time: 2017/10/24 12:30:17 \n",
      "INFO  12:30:17,940 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:17,941 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:30:18,025 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:30:18,113 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:30:18,120 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:30:18,162 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:30:18,315 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:30:18,319 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:30:18,320 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:30:18,320 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:30:18,320 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:30:18,346 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:30:18,347 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:30:18,869 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-77 because there are too many reads. \n",
      "INFO  12:30:20,474 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:7-68 because there are too many reads. \n",
      "INFO  12:30:23,323 ProgressMeter -            done    258333.0     5.0 s      19.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  12:30:23,324 ProgressMeter - Total runtime 5.00 secs, 0.08 min, 0.00 hours \n",
      "INFO  12:30:23,326 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 258333 total reads (0.00%) \n",
      "INFO  12:30:23,327 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:30:23,327 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19008_S11_L001_R1_001_paired.sam 2> 2010AC19008_S11_L001_R1_001_paired.txt2010AC19008_S11_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19008_S11_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19008_S11_L001_R1_001_paired.sam OUTPUT=2010AC19008_S11_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:32:22 PDT 2017] picard.sam.SortSam INPUT=2010AC19008_S11_L001_R1_001_paired.sam OUTPUT=2010AC19008_S11_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:32:22 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:32:37\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:32:48 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.44 minutes.\n",
      "Runtime.totalMemory()=2188902400\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19008_S11_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19008_S11_L001_R1_001_paired.bam.list\n",
      "INFO  12:32:50,078 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:50,081 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:32:50,081 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:32:50,081 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:32:50,081 HelpFormatter - [Tue Oct 24 12:32:50 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:32:50,082 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:32:50,086 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19008_S11_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19008_S11_L001_R1_001_paired.bam.list \n",
      "INFO  12:32:50,089 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:32:50,089 HelpFormatter - Date/Time: 2017/10/24 12:32:50 \n",
      "INFO  12:32:50,089 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:50,090 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:50,095 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:32:50,233 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:32:50,239 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:32:50,275 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:32:50,440 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:32:50,466 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:32:50,466 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:32:50,467 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:32:50,467 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:32:56,432 ProgressMeter -            done     16100.0     5.0 s       6.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  12:32:56,433 ProgressMeter - Total runtime 5.97 secs, 0.10 min, 0.00 hours \n",
      "INFO  12:32:56,433 MicroScheduler - 109 reads were filtered out during the traversal out of approximately 976249 total reads (0.01%) \n",
      "INFO  12:32:56,433 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:32:56,433 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:32:56,434 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:32:56,434 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:32:56,434 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:32:56,434 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:32:56,434 MicroScheduler -   -> 109 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:32:56,435 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:32:56,435 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:32:56,435 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R1_001_paired.bam -targetIntervals 2010AC19008_S11_L001_R1_001_paired.bam.list -o 2010AC19008_S11_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:32:58,190 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:58,192 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:32:58,192 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:32:58,192 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:32:58,192 HelpFormatter - [Tue Oct 24 12:32:58 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:32:58,193 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:32:58,196 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R1_001_paired.bam -targetIntervals 2010AC19008_S11_L001_R1_001_paired.bam.list -o 2010AC19008_S11_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:32:58,199 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:32:58,199 HelpFormatter - Date/Time: 2017/10/24 12:32:58 \n",
      "INFO  12:32:58,199 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:58,199 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:32:58,277 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:32:58,366 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:32:58,372 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:32:58,419 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:32:58,579 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:32:58,583 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:32:58,583 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:32:58,584 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:32:58,584 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:32:58,613 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:32:58,614 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:33:00,285 IndelRealigner - Not attempting realignment in interval One_RAD56448:4-82 because there are too many reads. \n",
      "INFO  12:33:07,436 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:7-78 because there are too many reads. \n",
      "INFO  12:33:26,105 ProgressMeter -            done   1270789.0    27.0 s      21.0 s      100.0%    27.0 s       0.0 s \n",
      "INFO  12:33:26,105 ProgressMeter - Total runtime 27.01 secs, 0.45 min, 0.01 hours \n",
      "INFO  12:33:26,108 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1270789 total reads (0.00%) \n",
      "INFO  12:33:26,109 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:33:26,109 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19015_S12_L001_R1_001_paired.sam 2> 2010AC19015_S12_L001_R1_001_paired.txt2010AC19015_S12_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19015_S12_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19015_S12_L001_R1_001_paired.sam OUTPUT=2010AC19015_S12_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:33:45 PDT 2017] picard.sam.SortSam INPUT=2010AC19015_S12_L001_R1_001_paired.sam OUTPUT=2010AC19015_S12_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:33:45 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:33:48\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:33:51 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=938475520\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19015_S12_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19015_S12_L001_R1_001_paired.bam.list\n",
      "INFO  12:33:52,791 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:52,793 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:33:52,794 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:33:52,794 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:33:52,794 HelpFormatter - [Tue Oct 24 12:33:52 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:33:52,794 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:33:52,797 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19015_S12_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19015_S12_L001_R1_001_paired.bam.list \n",
      "INFO  12:33:52,800 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:33:52,801 HelpFormatter - Date/Time: 2017/10/24 12:33:52 \n",
      "INFO  12:33:52,801 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:52,801 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:52,807 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:33:52,946 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:33:52,953 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:33:52,991 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:33:53,168 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:33:53,193 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:33:53,194 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:33:53,194 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:33:53,194 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:33:55,809 ProgressMeter -            done     16100.0     2.0 s       2.7 m       99.5%     2.0 s       0.0 s \n",
      "INFO  12:33:55,810 ProgressMeter - Total runtime 2.62 secs, 0.04 min, 0.00 hours \n",
      "INFO  12:33:55,810 MicroScheduler - 23 reads were filtered out during the traversal out of approximately 171280 total reads (0.01%) \n",
      "INFO  12:33:55,810 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:33:55,811 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:33:55,811 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:33:55,811 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:33:55,811 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:33:55,812 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:33:55,812 MicroScheduler -   -> 23 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:33:55,812 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:33:55,812 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:33:55,813 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R1_001_paired.bam -targetIntervals 2010AC19015_S12_L001_R1_001_paired.bam.list -o 2010AC19015_S12_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:33:57,541 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:57,543 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:33:57,544 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:33:57,544 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:33:57,544 HelpFormatter - [Tue Oct 24 12:33:57 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:33:57,544 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:33:57,548 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R1_001_paired.bam -targetIntervals 2010AC19015_S12_L001_R1_001_paired.bam.list -o 2010AC19015_S12_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:33:57,551 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:33:57,551 HelpFormatter - Date/Time: 2017/10/24 12:33:57 \n",
      "INFO  12:33:57,551 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:57,551 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:33:57,637 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:33:57,732 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:33:57,738 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:33:57,781 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:33:57,940 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:33:57,944 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:33:57,945 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:33:57,945 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:33:57,945 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:33:57,974 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:33:57,975 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:33:58,592 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-57 because there are too many reads. \n",
      "INFO  12:34:00,298 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:6-68 because there are too many reads. \n",
      "INFO  12:34:03,126 ProgressMeter -            done    244888.0     5.0 s      21.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  12:34:03,126 ProgressMeter - Total runtime 5.18 secs, 0.09 min, 0.00 hours \n",
      "INFO  12:34:03,129 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 244888 total reads (0.00%) \n",
      "INFO  12:34:03,130 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:34:03,130 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19041_S17_L001_R1_001_paired.sam 2> 2010AC19041_S17_L001_R1_001_paired.txt2010AC19041_S17_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19041_S17_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19041_S17_L001_R1_001_paired.sam OUTPUT=2010AC19041_S17_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:34:31 PDT 2017] picard.sam.SortSam INPUT=2010AC19041_S17_L001_R1_001_paired.sam OUTPUT=2010AC19041_S17_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:34:31 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:34:34\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:34:37 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=978845696\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19041_S17_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19041_S17_L001_R1_001_paired.bam.list\n",
      "INFO  12:34:39,180 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:39,183 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:34:39,183 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:34:39,183 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:34:39,183 HelpFormatter - [Tue Oct 24 12:34:39 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:34:39,183 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:34:39,186 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19041_S17_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19041_S17_L001_R1_001_paired.bam.list \n",
      "INFO  12:34:39,189 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:34:39,190 HelpFormatter - Date/Time: 2017/10/24 12:34:39 \n",
      "INFO  12:34:39,190 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:39,190 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:39,195 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:34:39,336 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:34:39,342 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:34:39,380 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:34:39,560 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:34:39,586 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:34:39,586 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:34:39,587 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:34:39,587 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:34:41,921 ProgressMeter -            done     16100.0     2.0 s       2.4 m       99.5%     2.0 s       0.0 s \n",
      "INFO  12:34:41,921 ProgressMeter - Total runtime 2.34 secs, 0.04 min, 0.00 hours \n",
      "INFO  12:34:41,922 MicroScheduler - 10 reads were filtered out during the traversal out of approximately 215947 total reads (0.00%) \n",
      "INFO  12:34:41,922 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:34:41,923 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:34:41,923 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:34:41,923 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:34:41,923 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:34:41,923 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:34:41,924 MicroScheduler -   -> 10 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:34:41,924 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:34:41,924 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:34:41,924 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R1_001_paired.bam -targetIntervals 2010AC19041_S17_L001_R1_001_paired.bam.list -o 2010AC19041_S17_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:34:43,736 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:43,739 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:34:43,739 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:34:43,739 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:34:43,739 HelpFormatter - [Tue Oct 24 12:34:43 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:34:43,739 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:34:43,743 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R1_001_paired.bam -targetIntervals 2010AC19041_S17_L001_R1_001_paired.bam.list -o 2010AC19041_S17_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:34:43,746 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:34:43,746 HelpFormatter - Date/Time: 2017/10/24 12:34:43 \n",
      "INFO  12:34:43,746 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:43,746 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:34:43,828 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:34:43,918 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:34:43,925 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:34:43,966 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:34:44,127 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:34:44,131 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:34:44,132 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:34:44,132 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:34:44,132 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:34:44,157 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:34:44,158 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:34:49,775 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  12:34:53,313 ProgressMeter -            done    281245.0     9.0 s      32.0 s      100.0%     9.0 s       0.0 s \n",
      "INFO  12:34:53,314 ProgressMeter - Total runtime 9.18 secs, 0.15 min, 0.00 hours \n",
      "INFO  12:34:53,316 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 281245 total reads (0.00%) \n",
      "INFO  12:34:53,317 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:34:53,317 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19047_S13_L001_R1_001_paired.sam 2> 2010AC19047_S13_L001_R1_001_paired.txt2010AC19047_S13_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19047_S13_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19047_S13_L001_R1_001_paired.sam OUTPUT=2010AC19047_S13_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:36:53 PDT 2017] picard.sam.SortSam INPUT=2010AC19047_S13_L001_R1_001_paired.sam OUTPUT=2010AC19047_S13_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:36:53 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:37:09\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:37:21 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.46 minutes.\n",
      "Runtime.totalMemory()=2152202240\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19047_S13_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19047_S13_L001_R1_001_paired.bam.list\n",
      "INFO  12:37:23,166 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:23,169 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:37:23,169 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:37:23,169 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:37:23,169 HelpFormatter - [Tue Oct 24 12:37:23 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:37:23,169 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:37:23,173 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19047_S13_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19047_S13_L001_R1_001_paired.bam.list \n",
      "INFO  12:37:23,177 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:37:23,177 HelpFormatter - Date/Time: 2017/10/24 12:37:23 \n",
      "INFO  12:37:23,177 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:23,177 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:23,182 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:37:23,334 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:37:23,343 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:37:23,386 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:37:23,579 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:37:23,606 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:37:23,607 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:37:23,607 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:37:23,608 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:37:30,479 ProgressMeter -            done     16100.0     6.0 s       7.1 m       99.5%     6.0 s       0.0 s \n",
      "INFO  12:37:30,480 ProgressMeter - Total runtime 6.87 secs, 0.11 min, 0.00 hours \n",
      "INFO  12:37:30,480 MicroScheduler - 30 reads were filtered out during the traversal out of approximately 1025813 total reads (0.00%) \n",
      "INFO  12:37:30,481 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:37:30,481 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:37:30,481 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:37:30,481 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:37:30,482 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:37:30,482 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:37:30,482 MicroScheduler -   -> 30 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:37:30,482 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:37:30,483 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:37:30,483 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R1_001_paired.bam -targetIntervals 2010AC19047_S13_L001_R1_001_paired.bam.list -o 2010AC19047_S13_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:37:32,379 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:32,381 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:37:32,381 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:37:32,382 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:37:32,382 HelpFormatter - [Tue Oct 24 12:37:32 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:37:32,382 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:37:32,385 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R1_001_paired.bam -targetIntervals 2010AC19047_S13_L001_R1_001_paired.bam.list -o 2010AC19047_S13_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:37:32,389 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:37:32,389 HelpFormatter - Date/Time: 2017/10/24 12:37:32 \n",
      "INFO  12:37:32,389 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:32,389 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:37:32,499 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:37:32,606 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:37:32,613 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:37:32,684 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.07 \n",
      "INFO  12:37:32,863 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:37:32,867 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:37:32,867 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:37:32,868 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:37:32,868 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:37:32,895 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:37:32,897 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:37:34,729 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-56 because there are too many reads. \n",
      "INFO  12:37:43,582 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-75 because there are too many reads. \n",
      " -S 2010AC19049_S14_L001_R1_001_paired.sam 2> 2010AC19049_S14_L001_R1_001_paired.txt2010AC19049_S14_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19049_S14_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19049_S14_L001_R1_001_paired.sam OUTPUT=2010AC19049_S14_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:41:38 PDT 2017] picard.sam.SortSam INPUT=2010AC19049_S14_L001_R1_001_paired.sam OUTPUT=2010AC19049_S14_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:41:38 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:41:41\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:41:45 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.12 minutes.\n",
      "Runtime.totalMemory()=1191706624\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19049_S14_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19049_S14_L001_R1_001_paired.bam.list\n",
      "INFO  12:41:47,341 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:47,343 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:41:47,343 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:41:47,344 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:41:47,344 HelpFormatter - [Tue Oct 24 12:41:47 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:41:47,344 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:41:47,347 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19049_S14_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19049_S14_L001_R1_001_paired.bam.list \n",
      "INFO  12:41:47,350 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:41:47,350 HelpFormatter - Date/Time: 2017/10/24 12:41:47 \n",
      "INFO  12:41:47,351 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:47,351 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:47,356 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:41:47,507 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:41:47,513 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:41:47,557 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:41:47,743 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:41:47,782 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:41:47,786 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:41:47,786 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:41:47,787 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:41:50,938 ProgressMeter -            done     16100.0     3.0 s       3.3 m       99.5%     3.0 s       0.0 s \n",
      "INFO  12:41:50,939 ProgressMeter - Total runtime 3.15 secs, 0.05 min, 0.00 hours \n",
      "INFO  12:41:50,940 MicroScheduler - 13 reads were filtered out during the traversal out of approximately 283254 total reads (0.00%) \n",
      "INFO  12:41:50,940 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:41:50,940 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:41:50,941 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:41:50,941 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:41:50,941 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:41:50,941 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:41:50,942 MicroScheduler -   -> 13 reads (0.00% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:41:50,942 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:41:50,942 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:41:50,942 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R1_001_paired.bam -targetIntervals 2010AC19049_S14_L001_R1_001_paired.bam.list -o 2010AC19049_S14_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:41:52,707 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:52,709 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:41:52,710 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:41:52,710 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:41:52,710 HelpFormatter - [Tue Oct 24 12:41:52 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:41:52,710 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:41:52,713 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R1_001_paired.bam -targetIntervals 2010AC19049_S14_L001_R1_001_paired.bam.list -o 2010AC19049_S14_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:41:52,716 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:41:52,717 HelpFormatter - Date/Time: 2017/10/24 12:41:52 \n",
      "INFO  12:41:52,717 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:52,717 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:41:52,810 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:41:52,905 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:41:52,911 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:41:52,957 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:41:53,115 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:41:53,119 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:41:53,120 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:41:53,120 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:41:53,120 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:41:53,146 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:41:53,147 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:42:02,269 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:15-75 because there are too many reads. \n",
      "INFO  12:42:06,583 ProgressMeter -            done    378287.0    13.0 s      35.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  12:42:06,583 ProgressMeter - Total runtime 13.46 secs, 0.22 min, 0.00 hours \n",
      "INFO  12:42:06,586 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 378287 total reads (0.00%) \n",
      "INFO  12:42:06,586 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:42:06,587 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19050_S15_L001_R1_001_paired.sam 2> 2010AC19050_S15_L001_R1_001_paired.txt2010AC19050_S15_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19050_S15_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19050_S15_L001_R1_001_paired.sam OUTPUT=2010AC19050_S15_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:43:08 PDT 2017] picard.sam.SortSam INPUT=2010AC19050_S15_L001_R1_001_paired.sam OUTPUT=2010AC19050_S15_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:43:08 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:43:17\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:43:23 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.24 minutes.\n",
      "Runtime.totalMemory()=1661992960\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19050_S15_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19050_S15_L001_R1_001_paired.bam.list\n",
      "INFO  12:43:24,841 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:24,843 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:43:24,843 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:43:24,844 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:43:24,844 HelpFormatter - [Tue Oct 24 12:43:24 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:43:24,844 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:43:24,847 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19050_S15_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19050_S15_L001_R1_001_paired.bam.list \n",
      "INFO  12:43:24,850 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:43:24,850 HelpFormatter - Date/Time: 2017/10/24 12:43:24 \n",
      "INFO  12:43:24,851 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:24,851 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:24,856 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:43:24,984 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:43:24,990 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:43:25,032 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:43:25,224 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:43:25,251 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:43:25,252 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:43:25,253 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:43:25,253 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:43:29,735 ProgressMeter -            done     16100.0     4.0 s       4.6 m       99.5%     4.0 s       0.0 s \n",
      "INFO  12:43:29,736 ProgressMeter - Total runtime 4.48 secs, 0.07 min, 0.00 hours \n",
      "INFO  12:43:29,736 MicroScheduler - 97 reads were filtered out during the traversal out of approximately 503983 total reads (0.02%) \n",
      "INFO  12:43:29,737 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:43:29,737 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:43:29,737 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:43:29,737 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:43:29,738 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:43:29,738 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:43:29,738 MicroScheduler -   -> 97 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:43:29,738 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:43:29,738 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:43:29,739 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R1_001_paired.bam -targetIntervals 2010AC19050_S15_L001_R1_001_paired.bam.list -o 2010AC19050_S15_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:43:31,471 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:31,474 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:43:31,474 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:43:31,474 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:43:31,474 HelpFormatter - [Tue Oct 24 12:43:31 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:43:31,474 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:43:31,477 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R1_001_paired.bam -targetIntervals 2010AC19050_S15_L001_R1_001_paired.bam.list -o 2010AC19050_S15_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:43:31,480 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:43:31,481 HelpFormatter - Date/Time: 2017/10/24 12:43:31 \n",
      "INFO  12:43:31,481 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:31,481 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:43:31,575 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:43:31,676 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:43:31,683 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:43:31,740 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.06 \n",
      "INFO  12:43:31,918 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:43:31,923 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:43:31,924 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:43:31,924 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:43:31,924 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:43:31,949 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:43:31,951 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:43:32,779 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-61 because there are too many reads. \n",
      "INFO  12:43:38,073 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  12:43:45,500 ProgressMeter -            done    672628.0    13.0 s      20.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  12:43:45,500 ProgressMeter - Total runtime 13.58 secs, 0.23 min, 0.00 hours \n",
      "INFO  12:43:45,503 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 672628 total reads (0.00%) \n",
      "INFO  12:43:45,504 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:43:45,504 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19057_S16_L001_R1_001_paired.sam 2> 2010AC19057_S16_L001_R1_001_paired.txt2010AC19057_S16_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19057_S16_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19057_S16_L001_R1_001_paired.sam OUTPUT=2010AC19057_S16_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:44:56 PDT 2017] picard.sam.SortSam INPUT=2010AC19057_S16_L001_R1_001_paired.sam OUTPUT=2010AC19057_S16_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:44:56 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:45:06\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:45:12 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.27 minutes.\n",
      "Runtime.totalMemory()=1739587584\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19057_S16_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19057_S16_L001_R1_001_paired.bam.list\n",
      "INFO  12:45:14,964 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:14,966 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:45:14,966 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:45:14,966 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:45:14,967 HelpFormatter - [Tue Oct 24 12:45:14 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:45:14,967 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:45:14,970 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19057_S16_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19057_S16_L001_R1_001_paired.bam.list \n",
      "INFO  12:45:14,974 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:45:14,974 HelpFormatter - Date/Time: 2017/10/24 12:45:14 \n",
      "INFO  12:45:14,974 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:14,975 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:14,980 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:45:15,135 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:45:15,141 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:45:15,187 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:45:15,361 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:45:15,388 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:45:15,388 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:45:15,389 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:45:15,389 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:45:19,514 ProgressMeter -            done     16100.0     4.0 s       4.3 m       99.5%     4.0 s       0.0 s \n",
      "INFO  12:45:19,515 ProgressMeter - Total runtime 4.13 secs, 0.07 min, 0.00 hours \n",
      "INFO  12:45:19,515 MicroScheduler - 69 reads were filtered out during the traversal out of approximately 565119 total reads (0.01%) \n",
      "INFO  12:45:19,515 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:45:19,516 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:45:19,516 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:45:19,516 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:45:19,516 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:45:19,517 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:45:19,517 MicroScheduler -   -> 69 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:45:19,517 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:45:19,517 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:45:19,518 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R1_001_paired.bam -targetIntervals 2010AC19057_S16_L001_R1_001_paired.bam.list -o 2010AC19057_S16_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:45:21,355 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:21,358 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:45:21,358 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:45:21,358 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:45:21,358 HelpFormatter - [Tue Oct 24 12:45:21 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:45:21,359 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:45:21,362 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R1_001_paired.bam -targetIntervals 2010AC19057_S16_L001_R1_001_paired.bam.list -o 2010AC19057_S16_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:45:21,365 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:45:21,365 HelpFormatter - Date/Time: 2017/10/24 12:45:21 \n",
      "INFO  12:45:21,365 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:21,366 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:45:21,445 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:45:21,532 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:45:21,539 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:45:21,585 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:45:21,751 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:45:21,756 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:45:21,756 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:45:21,756 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:45:21,757 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:45:21,784 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:45:21,785 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:45:22,378 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-65 because there are too many reads. \n",
      "INFO  12:45:28,153 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  12:45:36,050 ProgressMeter -            done    694333.0    14.0 s      20.0 s      100.0%    14.0 s       0.0 s \n",
      "INFO  12:45:36,050 ProgressMeter - Total runtime 14.29 secs, 0.24 min, 0.00 hours \n",
      "INFO  12:45:36,053 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 694333 total reads (0.00%) \n",
      "INFO  12:45:36,053 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:45:36,054 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19091_S18_L001_R1_001_paired.sam 2> 2010AC19091_S18_L001_R1_001_paired.txt2010AC19091_S18_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19091_S18_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19091_S18_L001_R1_001_paired.sam OUTPUT=2010AC19091_S18_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:47:12 PDT 2017] picard.sam.SortSam INPUT=2010AC19091_S18_L001_R1_001_paired.sam OUTPUT=2010AC19091_S18_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:47:12 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:47:24\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:47:33 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.36 minutes.\n",
      "Runtime.totalMemory()=2022703104\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19091_S18_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19091_S18_L001_R1_001_paired.bam.list\n",
      "INFO  12:47:35,468 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:35,470 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:47:35,470 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:47:35,470 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:47:35,471 HelpFormatter - [Tue Oct 24 12:47:35 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:47:35,471 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:47:35,474 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19091_S18_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19091_S18_L001_R1_001_paired.bam.list \n",
      "INFO  12:47:35,477 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:47:35,477 HelpFormatter - Date/Time: 2017/10/24 12:47:35 \n",
      "INFO  12:47:35,478 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:35,478 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:35,483 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:47:35,628 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:47:35,634 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:47:35,677 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:47:35,845 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:47:35,870 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:47:35,871 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:47:35,871 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:47:35,871 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:47:41,234 ProgressMeter -            done     16100.0     5.0 s       5.6 m       99.5%     5.0 s       0.0 s \n",
      "INFO  12:47:41,235 ProgressMeter - Total runtime 5.36 secs, 0.09 min, 0.00 hours \n",
      "INFO  12:47:41,235 MicroScheduler - 198 reads were filtered out during the traversal out of approximately 786184 total reads (0.03%) \n",
      "INFO  12:47:41,236 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:47:41,236 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:47:41,236 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:47:41,236 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:47:41,237 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:47:41,237 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:47:41,237 MicroScheduler -   -> 198 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:47:41,238 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:47:41,238 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:47:41,238 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R1_001_paired.bam -targetIntervals 2010AC19091_S18_L001_R1_001_paired.bam.list -o 2010AC19091_S18_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:47:43,010 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:43,013 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:47:43,013 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:47:43,013 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:47:43,013 HelpFormatter - [Tue Oct 24 12:47:42 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:47:43,014 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:47:43,017 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R1_001_paired.bam -targetIntervals 2010AC19091_S18_L001_R1_001_paired.bam.list -o 2010AC19091_S18_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:47:43,020 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:47:43,021 HelpFormatter - Date/Time: 2017/10/24 12:47:43 \n",
      "INFO  12:47:43,021 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:43,021 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:47:43,114 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:47:43,249 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:47:43,257 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:47:43,336 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.08 \n",
      "INFO  12:47:43,514 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:47:43,518 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:47:43,518 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:47:43,519 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:47:43,519 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:47:43,547 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:47:43,549 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:47:44,395 IndelRealigner - Not attempting realignment in interval One_RAD56448:20-57 because there are too many reads. \n",
      "INFO  12:47:53,783 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  12:48:05,733 ProgressMeter -            done    966717.0    22.0 s      22.0 s      100.0%    22.0 s       0.0 s \n",
      "INFO  12:48:05,733 ProgressMeter - Total runtime 22.21 secs, 0.37 min, 0.01 hours \n",
      "INFO  12:48:05,736 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 966717 total reads (0.00%) \n",
      "INFO  12:48:05,736 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:48:05,737 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19096_S21_L001_R1_001_paired.sam 2> 2010AC19096_S21_L001_R1_001_paired.txt2010AC19096_S21_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19096_S21_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19096_S21_L001_R1_001_paired.sam OUTPUT=2010AC19096_S21_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:49:13 PDT 2017] picard.sam.SortSam INPUT=2010AC19096_S21_L001_R1_001_paired.sam OUTPUT=2010AC19096_S21_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:49:13 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:49:23\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:49:29 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.28 minutes.\n",
      "Runtime.totalMemory()=1728577536\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19096_S21_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19096_S21_L001_R1_001_paired.bam.list\n",
      "INFO  12:49:31,751 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:31,753 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:49:31,753 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:49:31,754 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:49:31,754 HelpFormatter - [Tue Oct 24 12:49:31 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:49:31,754 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:49:31,757 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19096_S21_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19096_S21_L001_R1_001_paired.bam.list \n",
      "INFO  12:49:31,761 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:49:31,761 HelpFormatter - Date/Time: 2017/10/24 12:49:31 \n",
      "INFO  12:49:31,761 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:31,761 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:31,766 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:49:31,919 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:49:31,925 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:49:31,967 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:49:32,142 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:49:32,169 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:49:32,169 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:49:32,170 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:49:32,170 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:49:36,141 ProgressMeter -            done     16100.0     3.0 s       4.1 m       99.5%     3.0 s       0.0 s \n",
      "INFO  12:49:36,142 ProgressMeter - Total runtime 3.97 secs, 0.07 min, 0.00 hours \n",
      "INFO  12:49:36,142 MicroScheduler - 100 reads were filtered out during the traversal out of approximately 562157 total reads (0.02%) \n",
      "INFO  12:49:36,143 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:49:36,143 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:49:36,143 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:49:36,144 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:49:36,144 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:49:36,144 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:49:36,144 MicroScheduler -   -> 100 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:49:36,145 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:49:36,145 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:49:36,145 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R1_001_paired.bam -targetIntervals 2010AC19096_S21_L001_R1_001_paired.bam.list -o 2010AC19096_S21_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:49:37,828 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:37,830 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:49:37,830 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:49:37,831 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:49:37,831 HelpFormatter - [Tue Oct 24 12:49:37 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:49:37,831 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:49:37,834 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R1_001_paired.bam -targetIntervals 2010AC19096_S21_L001_R1_001_paired.bam.list -o 2010AC19096_S21_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:49:37,837 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:49:37,837 HelpFormatter - Date/Time: 2017/10/24 12:49:37 \n",
      "INFO  12:49:37,838 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:37,838 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:49:37,911 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:49:37,998 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:49:38,004 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:49:38,051 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:49:38,214 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:49:38,217 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:49:38,218 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:49:38,218 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:49:38,218 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:49:38,244 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:49:38,246 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:49:38,944 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-57 because there are too many reads. \n",
      "INFO  12:49:43,779 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  12:49:51,277 ProgressMeter -            done    716968.0    13.0 s      18.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  12:49:51,277 ProgressMeter - Total runtime 13.06 secs, 0.22 min, 0.00 hours \n",
      "INFO  12:49:51,280 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 716968 total reads (0.00%) \n",
      "INFO  12:49:51,280 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:49:51,280 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19098_S22_L001_R1_001_paired.sam 2> 2010AC19098_S22_L001_R1_001_paired.txt2010AC19098_S22_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19098_S22_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19098_S22_L001_R1_001_paired.sam OUTPUT=2010AC19098_S22_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:50:58 PDT 2017] picard.sam.SortSam INPUT=2010AC19098_S22_L001_R1_001_paired.sam OUTPUT=2010AC19098_S22_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:50:58 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:51:09\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:51:17 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.30 minutes.\n",
      "Runtime.totalMemory()=1773666304\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19098_S22_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19098_S22_L001_R1_001_paired.bam.list\n",
      "INFO  12:51:19,125 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:19,127 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:51:19,128 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:51:19,128 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:51:19,128 HelpFormatter - [Tue Oct 24 12:51:19 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:51:19,128 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:51:19,131 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19098_S22_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19098_S22_L001_R1_001_paired.bam.list \n",
      "INFO  12:51:19,134 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:51:19,135 HelpFormatter - Date/Time: 2017/10/24 12:51:19 \n",
      "INFO  12:51:19,135 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:19,135 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:19,140 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:51:19,295 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:51:19,301 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:51:19,340 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:51:19,495 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:51:19,521 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:51:19,521 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:51:19,522 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:51:19,522 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:51:24,458 ProgressMeter -            done     16100.0     4.0 s       5.1 m       99.5%     4.0 s       0.0 s \n",
      "INFO  12:51:24,458 ProgressMeter - Total runtime 4.94 secs, 0.08 min, 0.00 hours \n",
      "INFO  12:51:24,459 MicroScheduler - 88 reads were filtered out during the traversal out of approximately 630281 total reads (0.01%) \n",
      "INFO  12:51:24,459 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:51:24,460 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:51:24,460 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:51:24,460 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:51:24,460 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:51:24,461 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:51:24,461 MicroScheduler -   -> 88 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:51:24,461 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:51:24,462 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:51:24,462 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R1_001_paired.bam -targetIntervals 2010AC19098_S22_L001_R1_001_paired.bam.list -o 2010AC19098_S22_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:51:26,198 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:26,201 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:51:26,201 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:51:26,201 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:51:26,201 HelpFormatter - [Tue Oct 24 12:51:26 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:51:26,201 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:51:26,205 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R1_001_paired.bam -targetIntervals 2010AC19098_S22_L001_R1_001_paired.bam.list -o 2010AC19098_S22_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:51:26,207 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:51:26,208 HelpFormatter - Date/Time: 2017/10/24 12:51:26 \n",
      "INFO  12:51:26,208 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:26,208 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:51:26,293 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:51:26,386 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:51:26,392 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:51:26,446 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:51:26,604 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:51:26,608 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:51:26,608 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:51:26,608 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:51:26,609 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:51:26,635 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:51:26,637 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:51:27,560 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-57 because there are too many reads. \n",
      "INFO  12:51:34,177 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:5-78 because there are too many reads. \n",
      "INFO  12:51:43,138 ProgressMeter -            done    803691.0    16.0 s      20.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  12:51:43,139 ProgressMeter - Total runtime 16.53 secs, 0.28 min, 0.00 hours \n",
      "INFO  12:51:43,141 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 803691 total reads (0.00%) \n",
      "INFO  12:51:43,142 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:51:43,142 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19099_S19_L001_R1_001_paired.sam 2> 2010AC19099_S19_L001_R1_001_paired.txt2010AC19099_S19_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19099_S19_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19099_S19_L001_R1_001_paired.sam OUTPUT=2010AC19099_S19_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:53:26 PDT 2017] picard.sam.SortSam INPUT=2010AC19099_S19_L001_R1_001_paired.sam OUTPUT=2010AC19099_S19_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:53:26 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:53:41\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:53:49 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.39 minutes.\n",
      "Runtime.totalMemory()=2045247488\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19099_S19_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19099_S19_L001_R1_001_paired.bam.list\n",
      "INFO  12:53:51,639 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:51,641 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:53:51,642 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:53:51,642 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:53:51,642 HelpFormatter - [Tue Oct 24 12:53:51 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:53:51,642 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:53:51,646 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19099_S19_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19099_S19_L001_R1_001_paired.bam.list \n",
      "INFO  12:53:51,649 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:53:51,649 HelpFormatter - Date/Time: 2017/10/24 12:53:51 \n",
      "INFO  12:53:51,650 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:51,650 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:51,655 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:53:51,817 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:53:51,823 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:53:51,863 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:53:52,022 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:53:52,048 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:53:52,049 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:53:52,049 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:53:52,050 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:53:57,950 ProgressMeter -            done     16100.0     5.0 s       6.1 m       99.5%     5.0 s       0.0 s \n",
      "INFO  12:53:57,950 ProgressMeter - Total runtime 5.90 secs, 0.10 min, 0.00 hours \n",
      "INFO  12:53:57,951 MicroScheduler - 284 reads were filtered out during the traversal out of approximately 864498 total reads (0.03%) \n",
      "INFO  12:53:57,951 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:53:57,951 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:53:57,952 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:53:57,952 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:53:57,952 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:53:57,953 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:53:57,953 MicroScheduler -   -> 284 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:53:57,953 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:53:57,954 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:53:57,954 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R1_001_paired.bam -targetIntervals 2010AC19099_S19_L001_R1_001_paired.bam.list -o 2010AC19099_S19_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:53:59,853 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:59,855 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:53:59,855 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:53:59,856 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:53:59,856 HelpFormatter - [Tue Oct 24 12:53:59 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:53:59,856 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:53:59,859 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R1_001_paired.bam -targetIntervals 2010AC19099_S19_L001_R1_001_paired.bam.list -o 2010AC19099_S19_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:53:59,862 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:53:59,863 HelpFormatter - Date/Time: 2017/10/24 12:53:59 \n",
      "INFO  12:53:59,863 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:59,863 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:53:59,943 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:54:00,029 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:54:00,035 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:54:00,087 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:54:00,252 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:54:00,256 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:54:00,257 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:54:00,257 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:54:00,257 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:54:00,284 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:54:00,286 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:54:01,223 IndelRealigner - Not attempting realignment in interval One_RAD56448:12-69 because there are too many reads. \n",
      "INFO  12:54:08,739 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-77 because there are too many reads. \n",
      "INFO  12:54:23,391 ProgressMeter -            done   1078742.0    23.0 s      21.0 s      100.0%    23.0 s       0.0 s \n",
      "INFO  12:54:23,391 ProgressMeter - Total runtime 23.13 secs, 0.39 min, 0.01 hours \n",
      "INFO  12:54:23,394 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1078742 total reads (0.00%) \n",
      "INFO  12:54:23,394 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:54:23,395 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19101_S23_L001_R1_001_paired.sam 2> 2010AC19101_S23_L001_R1_001_paired.txt2010AC19101_S23_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19101_S23_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19101_S23_L001_R1_001_paired.sam OUTPUT=2010AC19101_S23_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:56:17 PDT 2017] picard.sam.SortSam INPUT=2010AC19101_S23_L001_R1_001_paired.sam OUTPUT=2010AC19101_S23_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:56:17 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:56:33\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:56:42 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.43 minutes.\n",
      "Runtime.totalMemory()=2127560704\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19101_S23_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19101_S23_L001_R1_001_paired.bam.list\n",
      "INFO  12:56:44,584 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:44,586 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:56:44,586 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:56:44,587 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:56:44,587 HelpFormatter - [Tue Oct 24 12:56:44 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:56:44,587 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:56:44,590 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19101_S23_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19101_S23_L001_R1_001_paired.bam.list \n",
      "INFO  12:56:44,594 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:56:44,594 HelpFormatter - Date/Time: 2017/10/24 12:56:44 \n",
      "INFO  12:56:44,594 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:44,594 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:44,599 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:56:44,736 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:56:44,743 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:56:44,804 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.06 \n",
      "INFO  12:56:44,964 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:56:44,990 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:56:44,991 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:56:44,991 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:56:44,991 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:56:51,229 ProgressMeter -            done     16100.0     6.0 s       6.5 m       99.5%     6.0 s       0.0 s \n",
      "INFO  12:56:51,230 ProgressMeter - Total runtime 6.24 secs, 0.10 min, 0.00 hours \n",
      "INFO  12:56:51,230 MicroScheduler - 156 reads were filtered out during the traversal out of approximately 1012410 total reads (0.02%) \n",
      "INFO  12:56:51,231 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:56:51,231 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:56:51,231 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:56:51,231 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:56:51,232 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:56:51,232 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:56:51,232 MicroScheduler -   -> 156 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:56:51,232 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:56:51,233 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:56:51,233 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R1_001_paired.bam -targetIntervals 2010AC19101_S23_L001_R1_001_paired.bam.list -o 2010AC19101_S23_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:56:53,036 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:53,039 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:56:53,039 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:56:53,039 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:56:53,039 HelpFormatter - [Tue Oct 24 12:56:53 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:56:53,039 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:56:53,043 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R1_001_paired.bam -targetIntervals 2010AC19101_S23_L001_R1_001_paired.bam.list -o 2010AC19101_S23_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:56:53,046 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:56:53,047 HelpFormatter - Date/Time: 2017/10/24 12:56:53 \n",
      "INFO  12:56:53,047 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:53,047 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:56:53,150 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:56:53,241 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:56:53,247 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:56:53,303 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:56:53,468 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:56:53,472 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:56:53,473 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:56:53,473 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:56:53,474 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:56:53,499 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:56:53,501 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:56:55,066 IndelRealigner - Not attempting realignment in interval One_RAD56448:8-53 because there are too many reads. \n",
      "INFO  12:57:05,198 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-68 because there are too many reads. \n",
      "INFO  12:57:19,592 ProgressMeter -            done   1209501.0    26.0 s      21.0 s      100.0%    26.0 s       0.0 s \n",
      "INFO  12:57:19,593 ProgressMeter - Total runtime 26.12 secs, 0.44 min, 0.01 hours \n",
      "INFO  12:57:19,596 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1209501 total reads (0.00%) \n",
      "INFO  12:57:19,596 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:57:19,596 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19102_S20_L001_R1_001_paired.sam 2> 2010AC19102_S20_L001_R1_001_paired.txt2010AC19102_S20_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19102_S20_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19102_S20_L001_R1_001_paired.sam OUTPUT=2010AC19102_S20_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 12:59:09 PDT 2017] picard.sam.SortSam INPUT=2010AC19102_S20_L001_R1_001_paired.sam OUTPUT=2010AC19102_S20_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 12:59:09 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 12:59:24\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 12:59:32 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.39 minutes.\n",
      "Runtime.totalMemory()=1978138624\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19102_S20_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19102_S20_L001_R1_001_paired.bam.list\n",
      "INFO  12:59:34,352 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:34,354 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:59:34,355 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:59:34,355 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:59:34,355 HelpFormatter - [Tue Oct 24 12:59:34 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:59:34,356 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:59:34,359 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19102_S20_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19102_S20_L001_R1_001_paired.bam.list \n",
      "INFO  12:59:34,362 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:59:34,363 HelpFormatter - Date/Time: 2017/10/24 12:59:34 \n",
      "INFO  12:59:34,363 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:34,363 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:34,369 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:59:34,509 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  12:59:34,515 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:59:34,555 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  12:59:34,751 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:59:34,779 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:59:34,780 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:59:34,780 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:59:34,780 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  12:59:40,528 ProgressMeter -            done     16100.0     5.0 s       6.0 m       99.5%     5.0 s       0.0 s \n",
      "INFO  12:59:40,529 ProgressMeter - Total runtime 5.75 secs, 0.10 min, 0.00 hours \n",
      "INFO  12:59:40,529 MicroScheduler - 56 reads were filtered out during the traversal out of approximately 904522 total reads (0.01%) \n",
      "INFO  12:59:40,530 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  12:59:40,530 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  12:59:40,530 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  12:59:40,531 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  12:59:40,531 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  12:59:40,531 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  12:59:40,531 MicroScheduler -   -> 56 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  12:59:40,532 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  12:59:40,532 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  12:59:40,532 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R1_001_paired.bam -targetIntervals 2010AC19102_S20_L001_R1_001_paired.bam.list -o 2010AC19102_S20_L001_R1_001_paired.realigned.bam\n",
      "INFO  12:59:42,294 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:42,296 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  12:59:42,296 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  12:59:42,296 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  12:59:42,297 HelpFormatter - [Tue Oct 24 12:59:42 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  12:59:42,297 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  12:59:42,300 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R1_001_paired.bam -targetIntervals 2010AC19102_S20_L001_R1_001_paired.bam.list -o 2010AC19102_S20_L001_R1_001_paired.realigned.bam \n",
      "INFO  12:59:42,303 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  12:59:42,304 HelpFormatter - Date/Time: 2017/10/24 12:59:42 \n",
      "INFO  12:59:42,304 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:42,304 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  12:59:42,384 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  12:59:42,469 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  12:59:42,475 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  12:59:42,525 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  12:59:42,684 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  12:59:42,688 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  12:59:42,688 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  12:59:42,689 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  12:59:42,689 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  12:59:42,718 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  12:59:42,719 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  12:59:43,525 IndelRealigner - Not attempting realignment in interval One_RAD56448:20-57 because there are too many reads. \n",
      "INFO  12:59:52,346 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-77 because there are too many reads. \n",
      "INFO  13:00:07,021 ProgressMeter -            done   1100757.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  13:00:07,022 ProgressMeter - Total runtime 24.33 secs, 0.41 min, 0.01 hours \n",
      "INFO  13:00:07,024 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1100757 total reads (0.00%) \n",
      "INFO  13:00:07,025 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:00:07,025 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19105_S24_L001_R1_001_paired.sam 2> 2010AC19105_S24_L001_R1_001_paired.txt2010AC19105_S24_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19105_S24_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19105_S24_L001_R1_001_paired.sam OUTPUT=2010AC19105_S24_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:02:50 PDT 2017] picard.sam.SortSam INPUT=2010AC19105_S24_L001_R1_001_paired.sam OUTPUT=2010AC19105_S24_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:02:50 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:03:19\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:03:34 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.74 minutes.\n",
      "Runtime.totalMemory()=2102394880\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19105_S24_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19105_S24_L001_R1_001_paired.bam.list\n",
      "INFO  13:03:36,214 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:36,216 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:03:36,216 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:03:36,216 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:03:36,217 HelpFormatter - [Tue Oct 24 13:03:36 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:03:36,217 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:03:36,220 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19105_S24_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19105_S24_L001_R1_001_paired.bam.list \n",
      "INFO  13:03:36,223 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:03:36,223 HelpFormatter - Date/Time: 2017/10/24 13:03:36 \n",
      "INFO  13:03:36,224 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:36,224 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:36,229 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:03:36,358 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:03:36,364 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:03:36,403 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:03:36,569 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:03:36,594 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:03:36,595 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:03:36,595 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:03:36,595 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:03:45,355 ProgressMeter -            done     16100.0     8.0 s       9.1 m       99.5%     8.0 s       0.0 s \n",
      "INFO  13:03:45,356 ProgressMeter - Total runtime 8.76 secs, 0.15 min, 0.00 hours \n",
      "INFO  13:03:45,356 MicroScheduler - 313 reads were filtered out during the traversal out of approximately 1608527 total reads (0.02%) \n",
      "INFO  13:03:45,357 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:03:45,357 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:03:45,357 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:03:45,357 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:03:45,358 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:03:45,358 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:03:45,358 MicroScheduler -   -> 313 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:03:45,359 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:03:45,359 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:03:45,359 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R1_001_paired.bam -targetIntervals 2010AC19105_S24_L001_R1_001_paired.bam.list -o 2010AC19105_S24_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:03:47,174 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:47,176 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:03:47,176 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:03:47,176 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:03:47,176 HelpFormatter - [Tue Oct 24 13:03:47 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:03:47,177 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:03:47,180 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R1_001_paired.bam -targetIntervals 2010AC19105_S24_L001_R1_001_paired.bam.list -o 2010AC19105_S24_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:03:47,183 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:03:47,184 HelpFormatter - Date/Time: 2017/10/24 13:03:47 \n",
      "INFO  13:03:47,184 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:47,184 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:03:47,269 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:03:47,363 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:03:47,370 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:03:47,420 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:03:47,582 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:03:47,586 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:03:47,587 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:03:47,587 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:03:47,587 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:03:47,614 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:03:47,615 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:03:49,990 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-54 because there are too many reads. \n",
      "INFO  13:03:55,462 IndelRealigner - Not attempting realignment in interval CM007957.1_1049506:8-60 because there are too many reads. \n",
      "INFO  13:04:06,096 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  13:04:15,975 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:6-72 because there are too many reads. \n",
      "INFO  13:04:18,015 ProgressMeter - 69876_x1_FemaleLG_8_FemalePos_9.73_MaleLG_NA_MalePos_NA_duplicated_no:74   1469065.0    30.0 s      20.0 s       78.6%    38.0 s       8.0 s \n",
      "INFO  13:04:25,801 ProgressMeter -            done   2065514.0    38.0 s      18.0 s      100.0%    38.0 s       0.0 s \n",
      "INFO  13:04:25,801 ProgressMeter - Total runtime 38.21 secs, 0.64 min, 0.01 hours \n",
      "INFO  13:04:25,804 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 2065514 total reads (0.00%) \n",
      "INFO  13:04:25,804 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:04:25,805 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19559_S3_L001_R1_001_paired.sam 2> 2010AC19559_S3_L001_R1_001_paired.txt:2010AC19559_S3_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19559_S3_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19559_S3_L001_R1_001_paired.sam OUTPUT=2010AC19559_S3_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:06:04 PDT 2017] picard.sam.SortSam INPUT=2010AC19559_S3_L001_R1_001_paired.sam OUTPUT=2010AC19559_S3_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:06:04 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:06:17\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:06:25 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.34 minutes.\n",
      "Runtime.totalMemory()=1951399936\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19559_S3_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19559_S3_L001_R1_001_paired.bam.list\n",
      "INFO  13:06:27,138 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:27,141 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:06:27,141 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:06:27,141 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:06:27,141 HelpFormatter - [Tue Oct 24 13:06:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:06:27,142 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:06:27,145 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19559_S3_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19559_S3_L001_R1_001_paired.bam.list \n",
      "INFO  13:06:27,148 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:06:27,148 HelpFormatter - Date/Time: 2017/10/24 13:06:27 \n",
      "INFO  13:06:27,148 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:27,149 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:27,154 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:06:27,303 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:06:27,310 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:06:27,356 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:06:27,526 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:06:27,554 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:06:27,554 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:06:27,555 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:06:27,555 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:06:33,027 ProgressMeter -            done     16100.0     5.0 s       5.7 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:06:33,028 ProgressMeter - Total runtime 5.47 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:06:33,028 MicroScheduler - 206 reads were filtered out during the traversal out of approximately 801646 total reads (0.03%) \n",
      "INFO  13:06:33,029 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:06:33,029 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:06:33,029 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:06:33,029 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:06:33,030 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:06:33,030 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:06:33,030 MicroScheduler -   -> 206 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:06:33,031 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:06:33,031 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:06:33,031 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R1_001_paired.bam -targetIntervals 2010AC19559_S3_L001_R1_001_paired.bam.list -o 2010AC19559_S3_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:06:35,263 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:35,265 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:06:35,265 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:06:35,266 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:06:35,266 HelpFormatter - [Tue Oct 24 13:06:35 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:06:35,266 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:06:35,269 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R1_001_paired.bam -targetIntervals 2010AC19559_S3_L001_R1_001_paired.bam.list -o 2010AC19559_S3_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:06:35,272 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:06:35,272 HelpFormatter - Date/Time: 2017/10/24 13:06:35 \n",
      "INFO  13:06:35,273 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:35,273 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:06:35,367 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:06:35,454 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:06:35,460 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:06:35,511 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:06:35,673 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:06:35,677 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:06:35,678 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:06:35,678 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:06:35,678 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:06:35,705 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:06:35,706 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:06:37,276 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-56 because there are too many reads. \n",
      "INFO  13:06:43,986 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-68 because there are too many reads. \n",
      "INFO  13:06:56,880 ProgressMeter -            done   1106227.0    21.0 s      19.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  13:06:56,881 ProgressMeter - Total runtime 21.20 secs, 0.35 min, 0.01 hours \n",
      "INFO  13:06:56,884 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1106227 total reads (0.00%) \n",
      "INFO  13:06:56,884 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:06:56,884 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19565_S4_L001_R1_001_paired.sam 2> 2010AC19565_S4_L001_R1_001_paired.txt:2010AC19565_S4_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19565_S4_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19565_S4_L001_R1_001_paired.sam OUTPUT=2010AC19565_S4_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:08:13 PDT 2017] picard.sam.SortSam INPUT=2010AC19565_S4_L001_R1_001_paired.sam OUTPUT=2010AC19565_S4_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:08:13 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:08:22\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:08:31 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.30 minutes.\n",
      "Runtime.totalMemory()=1844969472\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19565_S4_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19565_S4_L001_R1_001_paired.bam.list\n",
      "INFO  13:08:33,035 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:33,037 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:08:33,038 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:08:33,038 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:08:33,038 HelpFormatter - [Tue Oct 24 13:08:33 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:08:33,038 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:08:33,041 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19565_S4_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19565_S4_L001_R1_001_paired.bam.list \n",
      "INFO  13:08:33,044 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:08:33,045 HelpFormatter - Date/Time: 2017/10/24 13:08:33 \n",
      "INFO  13:08:33,045 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:33,045 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:33,050 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:08:33,182 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:08:33,188 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:08:33,229 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:08:33,401 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:08:33,427 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:08:33,428 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:08:33,428 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:08:33,429 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:08:38,416 ProgressMeter -            done     16100.0     4.0 s       5.2 m       99.5%     4.0 s       0.0 s \n",
      "INFO  13:08:38,416 ProgressMeter - Total runtime 4.99 secs, 0.08 min, 0.00 hours \n",
      "INFO  13:08:38,417 MicroScheduler - 97 reads were filtered out during the traversal out of approximately 661014 total reads (0.01%) \n",
      "INFO  13:08:38,417 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:08:38,417 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:08:38,417 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:08:38,418 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:08:38,418 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:08:38,418 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:08:38,418 MicroScheduler -   -> 97 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:08:38,419 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:08:38,419 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:08:38,419 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R1_001_paired.bam -targetIntervals 2010AC19565_S4_L001_R1_001_paired.bam.list -o 2010AC19565_S4_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:08:40,123 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:40,125 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:08:40,125 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:08:40,125 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:08:40,126 HelpFormatter - [Tue Oct 24 13:08:40 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:08:40,126 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:08:40,129 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R1_001_paired.bam -targetIntervals 2010AC19565_S4_L001_R1_001_paired.bam.list -o 2010AC19565_S4_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:08:40,132 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:08:40,132 HelpFormatter - Date/Time: 2017/10/24 13:08:40 \n",
      "INFO  13:08:40,132 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:40,133 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:08:40,214 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:08:40,301 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:08:40,307 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:08:40,351 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:08:40,517 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:08:40,522 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:08:40,522 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:08:40,522 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:08:40,522 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:08:40,548 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:08:40,550 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:08:42,146 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-71 because there are too many reads. \n",
      "INFO  13:08:48,333 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:14-78 because there are too many reads. \n",
      "INFO  13:08:59,347 ProgressMeter -            done    973818.0    18.0 s      19.0 s      100.0%    18.0 s       0.0 s \n",
      "INFO  13:08:59,348 ProgressMeter - Total runtime 18.83 secs, 0.31 min, 0.01 hours \n",
      "INFO  13:08:59,350 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 973818 total reads (0.00%) \n",
      "INFO  13:08:59,351 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:08:59,351 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19685_S5_L001_R1_001_paired.sam 2> 2010AC19685_S5_L001_R1_001_paired.txt:2010AC19685_S5_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19685_S5_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19685_S5_L001_R1_001_paired.sam OUTPUT=2010AC19685_S5_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:10:25 PDT 2017] picard.sam.SortSam INPUT=2010AC19685_S5_L001_R1_001_paired.sam OUTPUT=2010AC19685_S5_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:10:25 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:10:38\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:10:46 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=1986527232\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19685_S5_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19685_S5_L001_R1_001_paired.bam.list\n",
      "INFO  13:10:48,280 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:48,282 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:10:48,282 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:10:48,283 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:10:48,283 HelpFormatter - [Tue Oct 24 13:10:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:10:48,283 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:10:48,286 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19685_S5_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19685_S5_L001_R1_001_paired.bam.list \n",
      "INFO  13:10:48,289 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:10:48,290 HelpFormatter - Date/Time: 2017/10/24 13:10:48 \n",
      "INFO  13:10:48,290 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:48,290 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:48,295 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:10:48,435 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:10:48,441 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:10:48,481 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:10:48,655 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:10:48,680 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:10:48,681 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:10:48,681 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:10:48,682 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:10:53,965 ProgressMeter -            done     16100.0     5.0 s       5.5 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:10:53,966 ProgressMeter - Total runtime 5.28 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:10:53,966 MicroScheduler - 167 reads were filtered out during the traversal out of approximately 749275 total reads (0.02%) \n",
      "INFO  13:10:53,966 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:10:53,967 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:10:53,967 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:10:53,967 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:10:53,968 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:10:53,968 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:10:53,968 MicroScheduler -   -> 167 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:10:53,968 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:10:53,969 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:10:53,969 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R1_001_paired.bam -targetIntervals 2010AC19685_S5_L001_R1_001_paired.bam.list -o 2010AC19685_S5_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:10:55,766 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:55,768 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:10:55,769 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:10:55,769 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:10:55,769 HelpFormatter - [Tue Oct 24 13:10:55 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:10:55,769 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:10:55,773 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R1_001_paired.bam -targetIntervals 2010AC19685_S5_L001_R1_001_paired.bam.list -o 2010AC19685_S5_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:10:55,776 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:10:55,776 HelpFormatter - Date/Time: 2017/10/24 13:10:55 \n",
      "INFO  13:10:55,777 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:55,777 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:10:55,870 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:10:55,958 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:10:55,964 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:10:56,011 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:10:56,180 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:10:56,184 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:10:56,184 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:10:56,185 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:10:56,185 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:10:56,210 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:10:56,211 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:10:57,809 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-71 because there are too many reads. \n",
      "INFO  13:11:04,249 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  13:11:15,979 ProgressMeter -            done   1053330.0    19.0 s      18.0 s      100.0%    19.0 s       0.0 s \n",
      "INFO  13:11:15,979 ProgressMeter - Total runtime 19.80 secs, 0.33 min, 0.01 hours \n",
      "INFO  13:11:15,982 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1053330 total reads (0.00%) \n",
      "INFO  13:11:15,982 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:11:15,983 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19689_S6_L001_R1_001_paired.sam 2> 2010AC19689_S6_L001_R1_001_paired.txt:2010AC19689_S6_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19689_S6_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19689_S6_L001_R1_001_paired.sam OUTPUT=2010AC19689_S6_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:11:41 PDT 2017] picard.sam.SortSam INPUT=2010AC19689_S6_L001_R1_001_paired.sam OUTPUT=2010AC19689_S6_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:11:41 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:11:43\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:11:46 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1007157248\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19689_S6_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19689_S6_L001_R1_001_paired.bam.list\n",
      "INFO  13:11:48,570 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:48,572 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:11:48,573 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:11:48,573 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:11:48,573 HelpFormatter - [Tue Oct 24 13:11:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:11:48,573 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:11:48,576 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19689_S6_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19689_S6_L001_R1_001_paired.bam.list \n",
      "INFO  13:11:48,579 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:11:48,580 HelpFormatter - Date/Time: 2017/10/24 13:11:48 \n",
      "INFO  13:11:48,580 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:48,580 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:48,585 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:11:48,718 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:11:48,724 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:11:48,762 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:11:48,927 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:11:48,952 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:11:48,953 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:11:48,953 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:11:48,953 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:11:51,463 ProgressMeter -            done     16100.0     2.0 s       2.6 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:11:51,464 ProgressMeter - Total runtime 2.51 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:11:51,464 MicroScheduler - 28 reads were filtered out during the traversal out of approximately 193630 total reads (0.01%) \n",
      "INFO  13:11:51,464 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:11:51,465 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:11:51,465 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:11:51,465 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:11:51,465 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:11:51,465 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:11:51,466 MicroScheduler -   -> 28 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:11:51,466 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:11:51,466 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:11:51,471 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R1_001_paired.bam -targetIntervals 2010AC19689_S6_L001_R1_001_paired.bam.list -o 2010AC19689_S6_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:11:53,196 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:53,198 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:11:53,199 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:11:53,199 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:11:53,199 HelpFormatter - [Tue Oct 24 13:11:53 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:11:53,199 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:11:53,202 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R1_001_paired.bam -targetIntervals 2010AC19689_S6_L001_R1_001_paired.bam.list -o 2010AC19689_S6_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:11:53,206 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:11:53,206 HelpFormatter - Date/Time: 2017/10/24 13:11:53 \n",
      "INFO  13:11:53,206 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:53,206 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:11:53,285 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:11:53,369 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:11:53,375 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:11:53,413 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:11:53,578 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:11:53,582 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:11:53,582 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:11:53,582 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:11:53,583 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:11:53,607 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:11:53,608 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:12:04,162 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:7-75 because there are too many reads. \n",
      "INFO  13:12:07,809 ProgressMeter -            done    307045.0    13.0 s      45.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  13:12:07,809 ProgressMeter - Total runtime 13.83 secs, 0.23 min, 0.00 hours \n",
      "INFO  13:12:07,812 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 307045 total reads (0.00%) \n",
      "INFO  13:12:07,812 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:12:07,813 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19909_S1_L001_R1_001_paired.sam 2> 2010AC19909_S1_L001_R1_001_paired.txt:2010AC19909_S1_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19909_S1_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19909_S1_L001_R1_001_paired.sam OUTPUT=2010AC19909_S1_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:14:18 PDT 2017] picard.sam.SortSam INPUT=2010AC19909_S1_L001_R1_001_paired.sam OUTPUT=2010AC19909_S1_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:14:18 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:14:40\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:14:52 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.56 minutes.\n",
      "Runtime.totalMemory()=2059403264\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19909_S1_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19909_S1_L001_R1_001_paired.bam.list\n",
      "INFO  13:14:54,745 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:14:54,747 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:14:54,747 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:14:54,747 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:14:54,748 HelpFormatter - [Tue Oct 24 13:14:54 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:14:54,748 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:14:54,751 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19909_S1_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19909_S1_L001_R1_001_paired.bam.list \n",
      "INFO  13:14:54,754 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:14:54,755 HelpFormatter - Date/Time: 2017/10/24 13:14:54 \n",
      "INFO  13:14:54,755 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:14:54,755 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:14:54,762 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:14:54,956 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:14:54,964 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:14:55,013 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:14:55,266 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:14:55,294 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:14:55,295 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:14:55,295 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:14:55,295 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:15:02,946 ProgressMeter -            done     16100.0     7.0 s       7.9 m       99.5%     7.0 s       0.0 s \n",
      "INFO  13:15:02,946 ProgressMeter - Total runtime 7.65 secs, 0.13 min, 0.00 hours \n",
      "INFO  13:15:02,947 MicroScheduler - 157 reads were filtered out during the traversal out of approximately 1080629 total reads (0.01%) \n",
      "INFO  13:15:02,947 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:15:02,947 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:15:02,948 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:15:02,948 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:15:02,948 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:15:02,948 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:15:02,949 MicroScheduler -   -> 157 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:15:02,949 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:15:02,949 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:15:02,949 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R1_001_paired.bam -targetIntervals 2010AC19909_S1_L001_R1_001_paired.bam.list -o 2010AC19909_S1_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:15:05,117 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:15:05,120 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:15:05,120 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:15:05,120 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:15:05,120 HelpFormatter - [Tue Oct 24 13:15:05 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:15:05,120 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:15:05,125 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R1_001_paired.bam -targetIntervals 2010AC19909_S1_L001_R1_001_paired.bam.list -o 2010AC19909_S1_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:15:05,128 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:15:05,129 HelpFormatter - Date/Time: 2017/10/24 13:15:05 \n",
      "INFO  13:15:05,129 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:15:05,129 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:15:05,246 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:15:05,408 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:15:05,415 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:15:05,522 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.11 \n",
      "INFO  13:15:05,723 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:15:05,730 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:15:05,731 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:15:05,731 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:15:05,731 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:15:05,760 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:15:05,761 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:15:08,867 IndelRealigner - Not attempting realignment in interval One_RAD56448:19-53 because there are too many reads. \n",
      "INFO  13:15:18,068 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-78 because there are too many reads. \n",
      "INFO  13:15:36,501 ProgressMeter - 9801_x1_FemaleLG_28_FemalePos_166.1_MaleLG_NA_MalePos_NA_duplicated_no:76   1078908.0    30.0 s      28.0 s      100.0%    30.0 s       0.0 s \n",
      "INFO  13:15:40,656 ProgressMeter -            done   1560253.0    34.0 s      22.0 s      100.0%    34.0 s       0.0 s \n",
      "INFO  13:15:40,657 ProgressMeter - Total runtime 34.93 secs, 0.58 min, 0.01 hours \n",
      "INFO  13:15:40,660 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1560253 total reads (0.00%) \n",
      "INFO  13:15:40,660 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:15:40,661 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19938_S7_L001_R1_001_paired.sam 2> 2010AC19938_S7_L001_R1_001_paired.txt:2010AC19938_S7_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19938_S7_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19938_S7_L001_R1_001_paired.sam OUTPUT=2010AC19938_S7_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:17:15 PDT 2017] picard.sam.SortSam INPUT=2010AC19938_S7_L001_R1_001_paired.sam OUTPUT=2010AC19938_S7_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:17:15 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:17:27\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:17:35 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.33 minutes.\n",
      "Runtime.totalMemory()=2031091712\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19938_S7_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19938_S7_L001_R1_001_paired.bam.list\n",
      "INFO  13:17:36,974 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:36,976 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:17:36,976 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:17:36,976 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:17:36,977 HelpFormatter - [Tue Oct 24 13:17:36 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:17:36,977 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:17:36,980 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19938_S7_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19938_S7_L001_R1_001_paired.bam.list \n",
      "INFO  13:17:36,983 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:17:36,983 HelpFormatter - Date/Time: 2017/10/24 13:17:36 \n",
      "INFO  13:17:36,984 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:36,984 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:36,989 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:17:37,131 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:17:37,137 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:17:37,177 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:17:37,332 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:17:37,357 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:17:37,358 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:17:37,358 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:17:37,359 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:17:42,787 ProgressMeter -            done     16100.0     5.0 s       5.6 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:17:42,788 ProgressMeter - Total runtime 5.43 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:17:42,788 MicroScheduler - 186 reads were filtered out during the traversal out of approximately 805464 total reads (0.02%) \n",
      "INFO  13:17:42,789 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:17:42,789 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:17:42,789 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:17:42,789 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:17:42,790 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:17:42,790 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:17:42,790 MicroScheduler -   -> 186 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:17:42,791 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:17:42,791 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:17:42,791 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R1_001_paired.bam -targetIntervals 2010AC19938_S7_L001_R1_001_paired.bam.list -o 2010AC19938_S7_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:17:44,533 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:44,535 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:17:44,535 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:17:44,536 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:17:44,536 HelpFormatter - [Tue Oct 24 13:17:44 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:17:44,536 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:17:44,539 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R1_001_paired.bam -targetIntervals 2010AC19938_S7_L001_R1_001_paired.bam.list -o 2010AC19938_S7_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:17:44,542 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:17:44,543 HelpFormatter - Date/Time: 2017/10/24 13:17:44 \n",
      "INFO  13:17:44,543 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:44,543 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:17:44,636 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:17:44,731 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:17:44,737 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:17:44,781 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:17:44,939 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:17:44,943 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:17:44,943 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:17:44,944 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:17:44,944 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:17:44,970 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:17:44,971 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:17:46,660 IndelRealigner - Not attempting realignment in interval One_RAD56448:11-71 because there are too many reads. \n",
      "INFO  13:17:53,327 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:10-68 because there are too many reads. \n",
      "INFO  13:18:04,992 ProgressMeter -            done   1025650.0    20.0 s      19.0 s      100.0%    20.0 s       0.0 s \n",
      "INFO  13:18:04,993 ProgressMeter - Total runtime 20.05 secs, 0.33 min, 0.01 hours \n",
      "INFO  13:18:04,996 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1025650 total reads (0.00%) \n",
      "INFO  13:18:04,996 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:18:04,996 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19940_S8_L001_R1_001_paired.sam 2> 2010AC19940_S8_L001_R1_001_paired.txt:2010AC19940_S8_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19940_S8_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19940_S8_L001_R1_001_paired.sam OUTPUT=2010AC19940_S8_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:19:11 PDT 2017] picard.sam.SortSam INPUT=2010AC19940_S8_L001_R1_001_paired.sam OUTPUT=2010AC19940_S8_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:19:11 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:19:20\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:19:26 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.25 minutes.\n",
      "Runtime.totalMemory()=1717567488\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19940_S8_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC19940_S8_L001_R1_001_paired.bam.list\n",
      "INFO  13:19:27,854 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:27,856 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:19:27,857 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:19:27,857 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:19:27,857 HelpFormatter - [Tue Oct 24 13:19:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:19:27,857 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:19:27,861 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19940_S8_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC19940_S8_L001_R1_001_paired.bam.list \n",
      "INFO  13:19:27,864 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:19:27,864 HelpFormatter - Date/Time: 2017/10/24 13:19:27 \n",
      "INFO  13:19:27,864 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:27,864 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:27,869 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:19:28,018 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:19:28,024 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:19:28,070 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:19:28,230 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:19:28,258 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:19:28,258 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:19:28,259 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:19:28,259 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:19:31,978 ProgressMeter -            done     16100.0     3.0 s       3.8 m       99.5%     3.0 s       0.0 s \n",
      "INFO  13:19:31,978 ProgressMeter - Total runtime 3.72 secs, 0.06 min, 0.00 hours \n",
      "INFO  13:19:31,979 MicroScheduler - 111 reads were filtered out during the traversal out of approximately 537614 total reads (0.02%) \n",
      "INFO  13:19:31,979 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:19:31,980 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:19:31,980 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:19:31,980 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:19:31,980 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:19:31,981 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:19:31,981 MicroScheduler -   -> 111 reads (0.02% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:19:31,981 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:19:31,981 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:19:31,982 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R1_001_paired.bam -targetIntervals 2010AC19940_S8_L001_R1_001_paired.bam.list -o 2010AC19940_S8_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:19:33,812 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:33,815 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:19:33,815 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:19:33,815 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:19:33,815 HelpFormatter - [Tue Oct 24 13:19:33 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:19:33,816 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:19:33,819 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R1_001_paired.bam -targetIntervals 2010AC19940_S8_L001_R1_001_paired.bam.list -o 2010AC19940_S8_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:19:33,822 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:19:33,822 HelpFormatter - Date/Time: 2017/10/24 13:19:33 \n",
      "INFO  13:19:33,822 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:33,823 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:19:33,900 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:19:33,987 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:19:33,993 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:19:34,043 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:19:34,202 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:19:34,206 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:19:34,206 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:19:34,207 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:19:34,207 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:19:34,232 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:19:34,233 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:19:34,897 IndelRealigner - Not attempting realignment in interval One_RAD56448:4-59 because there are too many reads. \n",
      "INFO  13:19:40,228 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:9-68 because there are too many reads. \n",
      "INFO  13:19:47,680 ProgressMeter -            done    701343.0    13.0 s      19.0 s      100.0%    13.0 s       0.0 s \n",
      "INFO  13:19:47,680 ProgressMeter - Total runtime 13.47 secs, 0.22 min, 0.00 hours \n",
      "INFO  13:19:47,683 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 701343 total reads (0.00%) \n",
      "INFO  13:19:47,683 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:19:47,683 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC20099_S2_L001_R1_001_paired.sam 2> 2010AC20099_S2_L001_R1_001_paired.txt:2010AC20099_S2_L001_R1_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC20099_S2_L001_R1_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC20099_S2_L001_R1_001_paired.sam OUTPUT=2010AC20099_S2_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:21:35 PDT 2017] picard.sam.SortSam INPUT=2010AC20099_S2_L001_R1_001_paired.sam OUTPUT=2010AC20099_S2_L001_R1_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:21:35 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:21:50\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:22:03 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.46 minutes.\n",
      "Runtime.totalMemory()=2138570752\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC20099_S2_L001_R1_001_paired.bam -R 233_Amplicons.fa -o  2010AC20099_S2_L001_R1_001_paired.bam.list\n",
      "INFO  13:22:05,007 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:05,009 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:22:05,009 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:22:05,009 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:22:05,009 HelpFormatter - [Tue Oct 24 13:22:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:22:05,010 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:22:05,013 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC20099_S2_L001_R1_001_paired.bam -R 233_Amplicons.fa -o 2010AC20099_S2_L001_R1_001_paired.bam.list \n",
      "INFO  13:22:05,016 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:22:05,016 HelpFormatter - Date/Time: 2017/10/24 13:22:05 \n",
      "INFO  13:22:05,016 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:05,016 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:05,021 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:22:05,167 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:22:05,173 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:22:05,212 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:22:05,367 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:22:05,392 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:22:05,392 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:22:05,393 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:22:05,393 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:22:11,659 ProgressMeter -            done     16100.0     6.0 s       6.5 m       99.5%     6.0 s       0.0 s \n",
      "INFO  13:22:11,659 ProgressMeter - Total runtime 6.27 secs, 0.10 min, 0.00 hours \n",
      "INFO  13:22:11,660 MicroScheduler - 282 reads were filtered out during the traversal out of approximately 952444 total reads (0.03%) \n",
      "INFO  13:22:11,660 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:22:11,660 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:22:11,661 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:22:11,661 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:22:11,661 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:22:11,661 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:22:11,662 MicroScheduler -   -> 282 reads (0.03% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:22:11,662 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:22:11,662 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:22:11,662 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R1_001_paired.bam -targetIntervals 2010AC20099_S2_L001_R1_001_paired.bam.list -o 2010AC20099_S2_L001_R1_001_paired.realigned.bam\n",
      "INFO  13:22:13,935 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:13,937 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:22:13,938 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:22:13,938 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:22:13,938 HelpFormatter - [Tue Oct 24 13:22:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:22:13,938 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:22:13,941 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R1_001_paired.bam -targetIntervals 2010AC20099_S2_L001_R1_001_paired.bam.list -o 2010AC20099_S2_L001_R1_001_paired.realigned.bam \n",
      "INFO  13:22:13,944 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:22:13,944 HelpFormatter - Date/Time: 2017/10/24 13:22:13 \n",
      "INFO  13:22:13,945 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:13,945 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:22:14,035 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:22:14,118 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:22:14,124 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:22:14,169 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:22:14,326 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:22:14,330 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:22:14,330 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:22:14,330 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:22:14,330 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:22:14,355 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:22:14,356 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:22:16,367 IndelRealigner - Not attempting realignment in interval One_RAD56448:10-53 because there are too many reads. \n",
      "INFO  13:22:24,587 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:8-77 because there are too many reads. \n",
      "INFO  13:22:39,609 ProgressMeter -            done   1434928.0    25.0 s      17.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  13:22:39,609 ProgressMeter - Total runtime 25.28 secs, 0.42 min, 0.01 hours \n",
      "INFO  13:22:39,612 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1434928 total reads (0.00%) \n",
      "INFO  13:22:39,612 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:22:39,613 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19003_S9_L001_R2_001_paired.sam 2> 2010AC19003_S9_L001_R2_001_paired.txt:2010AC19003_S9_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19003_S9_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19003_S9_L001_R2_001_paired.sam OUTPUT=2010AC19003_S9_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:23:49 PDT 2017] picard.sam.SortSam INPUT=2010AC19003_S9_L001_R2_001_paired.sam OUTPUT=2010AC19003_S9_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:23:49 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:23:58\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:24:07 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.29 minutes.\n",
      "Runtime.totalMemory()=1752170496\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19003_S9_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19003_S9_L001_R2_001_paired.bam.list\n",
      "INFO  13:24:08,866 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:08,868 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:24:08,868 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:24:08,869 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:24:08,869 HelpFormatter - [Tue Oct 24 13:24:08 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:24:08,869 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:24:08,872 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19003_S9_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19003_S9_L001_R2_001_paired.bam.list \n",
      "INFO  13:24:08,875 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:24:08,876 HelpFormatter - Date/Time: 2017/10/24 13:24:08 \n",
      "INFO  13:24:08,876 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:08,876 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:08,881 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:24:09,008 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:24:09,014 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:24:09,055 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:24:09,227 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:24:09,252 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:24:09,252 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:24:09,253 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:24:09,253 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:24:13,897 ProgressMeter -            done     16100.0     4.0 s       4.8 m       99.5%     4.0 s       0.0 s \n",
      "INFO  13:24:13,897 ProgressMeter - Total runtime 4.65 secs, 0.08 min, 0.00 hours \n",
      "INFO  13:24:13,898 MicroScheduler - 82 reads were filtered out during the traversal out of approximately 691727 total reads (0.01%) \n",
      "INFO  13:24:13,898 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:24:13,899 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:24:13,899 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:24:13,899 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:24:13,900 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:24:13,900 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:24:13,900 MicroScheduler -   -> 82 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:24:13,900 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:24:13,901 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:24:13,901 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R2_001_paired.bam -targetIntervals 2010AC19003_S9_L001_R2_001_paired.bam.list -o 2010AC19003_S9_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:24:15,632 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:15,634 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:24:15,635 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:24:15,635 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:24:15,635 HelpFormatter - [Tue Oct 24 13:24:15 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:24:15,635 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:24:15,638 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19003_S9_L001_R2_001_paired.bam -targetIntervals 2010AC19003_S9_L001_R2_001_paired.bam.list -o 2010AC19003_S9_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:24:15,641 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:24:15,642 HelpFormatter - Date/Time: 2017/10/24 13:24:15 \n",
      "INFO  13:24:15,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:15,642 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:24:15,723 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:24:15,806 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:24:15,812 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:24:15,854 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:24:16,010 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:24:16,015 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:24:16,015 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:24:16,016 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:24:16,016 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:24:16,041 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:24:16,043 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:24:17,709 IndelRealigner - Not attempting realignment in interval One_RAD56448:18-57 because there are too many reads. \n",
      "INFO  13:24:21,940 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-77 because there are too many reads. \n",
      "INFO  13:24:35,707 ProgressMeter -            done    904433.0    19.0 s      21.0 s      100.0%    19.0 s       0.0 s \n",
      "INFO  13:24:35,708 ProgressMeter - Total runtime 19.69 secs, 0.33 min, 0.01 hours \n",
      "INFO  13:24:35,710 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 904433 total reads (0.00%) \n",
      "INFO  13:24:35,711 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:24:35,711 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19006_S10_L001_R2_001_paired.sam 2> 2010AC19006_S10_L001_R2_001_paired.txt2010AC19006_S10_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19006_S10_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19006_S10_L001_R2_001_paired.sam OUTPUT=2010AC19006_S10_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:24:53 PDT 2017] picard.sam.SortSam INPUT=2010AC19006_S10_L001_R2_001_paired.sam OUTPUT=2010AC19006_S10_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:24:53 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:24:56\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:24:58 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=972554240\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19006_S10_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19006_S10_L001_R2_001_paired.bam.list\n",
      "INFO  13:25:00,724 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:00,726 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:25:00,726 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:25:00,727 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:25:00,727 HelpFormatter - [Tue Oct 24 13:25:00 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:25:00,727 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:25:00,730 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19006_S10_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19006_S10_L001_R2_001_paired.bam.list \n",
      "INFO  13:25:00,734 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:25:00,734 HelpFormatter - Date/Time: 2017/10/24 13:25:00 \n",
      "INFO  13:25:00,734 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:00,734 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:00,739 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:25:00,873 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:25:00,879 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:25:00,918 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:25:01,086 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:25:01,112 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:25:01,112 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:25:01,112 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:25:01,113 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:25:03,532 ProgressMeter -            done     16100.0     2.0 s       2.5 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:25:03,532 ProgressMeter - Total runtime 2.42 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:25:03,533 MicroScheduler - 13 reads were filtered out during the traversal out of approximately 179253 total reads (0.01%) \n",
      "INFO  13:25:03,533 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:25:03,533 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:25:03,534 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:25:03,534 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:25:03,534 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:25:03,535 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:25:03,535 MicroScheduler -   -> 13 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:25:03,535 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:25:03,535 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:25:03,536 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R2_001_paired.bam -targetIntervals 2010AC19006_S10_L001_R2_001_paired.bam.list -o 2010AC19006_S10_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:25:05,257 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:05,259 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:25:05,259 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:25:05,260 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:25:05,260 HelpFormatter - [Tue Oct 24 13:25:05 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:25:05,260 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:25:05,263 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19006_S10_L001_R2_001_paired.bam -targetIntervals 2010AC19006_S10_L001_R2_001_paired.bam.list -o 2010AC19006_S10_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:25:05,266 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:25:05,266 HelpFormatter - Date/Time: 2017/10/24 13:25:05 \n",
      "INFO  13:25:05,267 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:05,267 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:25:05,341 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:25:05,427 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:25:05,433 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:25:05,483 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:25:05,639 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:25:05,643 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:25:05,644 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:25:05,644 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:25:05,644 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:25:05,670 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:25:05,671 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:25:06,220 IndelRealigner - Not attempting realignment in interval One_RAD56448:26-73 because there are too many reads. \n",
      "INFO  13:25:07,831 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-75 because there are too many reads. \n",
      "INFO  13:25:11,313 ProgressMeter -            done    258333.0     5.0 s      21.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  13:25:11,314 ProgressMeter - Total runtime 5.67 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:25:11,317 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 258333 total reads (0.00%) \n",
      "INFO  13:25:11,317 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:25:11,318 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19008_S11_L001_R2_001_paired.sam 2> 2010AC19008_S11_L001_R2_001_paired.txt2010AC19008_S11_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19008_S11_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19008_S11_L001_R2_001_paired.sam OUTPUT=2010AC19008_S11_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:26:52 PDT 2017] picard.sam.SortSam INPUT=2010AC19008_S11_L001_R2_001_paired.sam OUTPUT=2010AC19008_S11_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:26:52 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:27:07\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:27:17 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.43 minutes.\n",
      "Runtime.totalMemory()=1970798592\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19008_S11_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19008_S11_L001_R2_001_paired.bam.list\n",
      "INFO  13:27:19,884 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:19,886 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:27:19,887 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:27:19,887 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:27:19,887 HelpFormatter - [Tue Oct 24 13:27:19 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:27:19,887 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:27:19,891 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19008_S11_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19008_S11_L001_R2_001_paired.bam.list \n",
      "INFO  13:27:19,893 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:27:19,894 HelpFormatter - Date/Time: 2017/10/24 13:27:19 \n",
      "INFO  13:27:19,894 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:19,894 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:19,899 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:27:20,043 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:27:20,049 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:27:20,088 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:27:20,257 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:27:20,283 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:27:20,283 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:27:20,284 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:27:20,284 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:27:26,055 ProgressMeter -            done     16100.0     5.0 s       6.0 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:27:26,055 ProgressMeter - Total runtime 5.77 secs, 0.10 min, 0.00 hours \n",
      "INFO  13:27:26,056 MicroScheduler - 93 reads were filtered out during the traversal out of approximately 987374 total reads (0.01%) \n",
      "INFO  13:27:26,056 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:27:26,056 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:27:26,057 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:27:26,057 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:27:26,057 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:27:26,057 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:27:26,058 MicroScheduler -   -> 93 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:27:26,058 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:27:26,058 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:27:26,058 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R2_001_paired.bam -targetIntervals 2010AC19008_S11_L001_R2_001_paired.bam.list -o 2010AC19008_S11_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:27:27,815 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:27,818 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:27:27,818 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:27:27,818 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:27:27,818 HelpFormatter - [Tue Oct 24 13:27:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:27:27,818 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:27:27,822 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19008_S11_L001_R2_001_paired.bam -targetIntervals 2010AC19008_S11_L001_R2_001_paired.bam.list -o 2010AC19008_S11_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:27:27,825 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:27:27,825 HelpFormatter - Date/Time: 2017/10/24 13:27:27 \n",
      "INFO  13:27:27,825 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:27,826 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:27:27,902 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:27:27,991 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:27:27,997 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:27:28,043 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:27:28,207 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:27:28,211 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:27:28,212 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:27:28,212 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:27:28,212 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:27:28,237 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:27:28,238 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:27:29,889 IndelRealigner - Not attempting realignment in interval One_RAD56448:8-56 because there are too many reads. \n",
      "INFO  13:27:35,743 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  13:27:56,476 ProgressMeter -            done   1270789.0    28.0 s      22.0 s      100.0%    28.0 s       0.0 s \n",
      "INFO  13:27:56,476 ProgressMeter - Total runtime 28.26 secs, 0.47 min, 0.01 hours \n",
      "INFO  13:27:56,479 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1270789 total reads (0.00%) \n",
      "INFO  13:27:56,479 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:27:56,480 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19015_S12_L001_R2_001_paired.sam 2> 2010AC19015_S12_L001_R2_001_paired.txt2010AC19015_S12_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19015_S12_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19015_S12_L001_R2_001_paired.sam OUTPUT=2010AC19015_S12_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:28:13 PDT 2017] picard.sam.SortSam INPUT=2010AC19015_S12_L001_R2_001_paired.sam OUTPUT=2010AC19015_S12_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:28:13 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:28:16\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:28:19 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.08 minutes.\n",
      "Runtime.totalMemory()=952107008\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19015_S12_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19015_S12_L001_R2_001_paired.bam.list\n",
      "INFO  13:28:20,758 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:20,760 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:28:20,761 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:28:20,761 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:28:20,761 HelpFormatter - [Tue Oct 24 13:28:20 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:28:20,761 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:28:20,765 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19015_S12_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19015_S12_L001_R2_001_paired.bam.list \n",
      "INFO  13:28:20,768 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:28:20,769 HelpFormatter - Date/Time: 2017/10/24 13:28:20 \n",
      "INFO  13:28:20,769 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:20,769 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:20,774 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:28:20,919 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:28:20,925 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:28:20,963 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:28:21,122 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:28:21,148 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:28:21,148 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:28:21,149 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:28:21,149 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:28:23,561 ProgressMeter -            done     16100.0     2.0 s       2.5 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:28:23,562 ProgressMeter - Total runtime 2.41 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:28:23,562 MicroScheduler - 14 reads were filtered out during the traversal out of approximately 177231 total reads (0.01%) \n",
      "INFO  13:28:23,563 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:28:23,563 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:28:23,563 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:28:23,564 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:28:23,564 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:28:23,564 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:28:23,564 MicroScheduler -   -> 14 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:28:23,565 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:28:23,565 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:28:23,565 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R2_001_paired.bam -targetIntervals 2010AC19015_S12_L001_R2_001_paired.bam.list -o 2010AC19015_S12_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:28:25,304 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:25,306 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:28:25,306 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:28:25,307 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:28:25,307 HelpFormatter - [Tue Oct 24 13:28:25 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:28:25,307 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:28:25,310 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19015_S12_L001_R2_001_paired.bam -targetIntervals 2010AC19015_S12_L001_R2_001_paired.bam.list -o 2010AC19015_S12_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:28:25,313 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:28:25,314 HelpFormatter - Date/Time: 2017/10/24 13:28:25 \n",
      "INFO  13:28:25,314 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:25,314 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:28:25,401 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:28:25,487 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:28:25,493 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:28:25,534 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:28:25,690 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:28:25,694 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:28:25,695 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:28:25,695 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:28:25,695 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:28:25,725 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:28:25,727 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:28:26,343 IndelRealigner - Not attempting realignment in interval One_RAD56448:5-56 because there are too many reads. \n",
      "INFO  13:28:27,944 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  13:28:31,095 ProgressMeter -            done    244888.0     5.0 s      22.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  13:28:31,096 ProgressMeter - Total runtime 5.40 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:28:31,099 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 244888 total reads (0.00%) \n",
      "INFO  13:28:31,099 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:28:31,100 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19041_S17_L001_R2_001_paired.sam 2> 2010AC19041_S17_L001_R2_001_paired.txt2010AC19041_S17_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19041_S17_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19041_S17_L001_R2_001_paired.sam OUTPUT=2010AC19041_S17_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:28:54 PDT 2017] picard.sam.SortSam INPUT=2010AC19041_S17_L001_R2_001_paired.sam OUTPUT=2010AC19041_S17_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:28:54 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:28:57\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:29:00 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1000341504\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19041_S17_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19041_S17_L001_R2_001_paired.bam.list\n",
      "INFO  13:29:01,752 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:01,754 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:29:01,755 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:29:01,755 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:29:01,755 HelpFormatter - [Tue Oct 24 13:29:01 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:29:01,756 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:29:01,759 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19041_S17_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19041_S17_L001_R2_001_paired.bam.list \n",
      "INFO  13:29:01,762 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:29:01,762 HelpFormatter - Date/Time: 2017/10/24 13:29:01 \n",
      "INFO  13:29:01,762 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:01,763 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:01,768 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:29:01,900 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:29:01,906 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:29:01,944 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:29:02,096 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:29:02,126 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:29:02,127 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:29:02,127 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:29:02,128 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:29:04,695 ProgressMeter -            done     16100.0     2.0 s       2.7 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:29:04,695 ProgressMeter - Total runtime 2.57 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:29:04,696 MicroScheduler - 28 reads were filtered out during the traversal out of approximately 215499 total reads (0.01%) \n",
      "INFO  13:29:04,696 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:29:04,696 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:29:04,697 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:29:04,697 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:29:04,697 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:29:04,697 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:29:04,698 MicroScheduler -   -> 28 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:29:04,698 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:29:04,698 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:29:04,698 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R2_001_paired.bam -targetIntervals 2010AC19041_S17_L001_R2_001_paired.bam.list -o 2010AC19041_S17_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:29:06,421 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:06,423 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:29:06,424 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:29:06,424 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:29:06,424 HelpFormatter - [Tue Oct 24 13:29:06 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:29:06,424 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:29:06,427 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19041_S17_L001_R2_001_paired.bam -targetIntervals 2010AC19041_S17_L001_R2_001_paired.bam.list -o 2010AC19041_S17_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:29:06,431 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:29:06,431 HelpFormatter - Date/Time: 2017/10/24 13:29:06 \n",
      "INFO  13:29:06,431 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:06,431 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:29:06,521 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:29:06,608 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:29:06,614 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:29:06,657 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:29:06,816 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:29:06,820 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:29:06,821 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:29:06,821 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:29:06,821 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:29:06,846 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:29:06,847 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:29:13,975 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:11-75 because there are too many reads. \n",
      "INFO  13:29:17,754 ProgressMeter -            done    281245.0    10.0 s      38.0 s      100.0%    10.0 s       0.0 s \n",
      "INFO  13:29:17,755 ProgressMeter - Total runtime 10.93 secs, 0.18 min, 0.00 hours \n",
      "INFO  13:29:17,757 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 281245 total reads (0.00%) \n",
      "INFO  13:29:17,758 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:29:17,758 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19047_S13_L001_R2_001_paired.sam 2> 2010AC19047_S13_L001_R2_001_paired.txt2010AC19047_S13_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19047_S13_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19047_S13_L001_R2_001_paired.sam OUTPUT=2010AC19047_S13_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:31:02 PDT 2017] picard.sam.SortSam INPUT=2010AC19047_S13_L001_R2_001_paired.sam OUTPUT=2010AC19047_S13_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:31:02 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:31:17\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:31:31 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.48 minutes.\n",
      "Runtime.totalMemory()=2125463552\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19047_S13_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19047_S13_L001_R2_001_paired.bam.list\n",
      "INFO  13:31:32,923 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:32,926 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:31:32,926 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:31:32,926 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:31:32,927 HelpFormatter - [Tue Oct 24 13:31:32 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:31:32,928 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:31:32,931 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19047_S13_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19047_S13_L001_R2_001_paired.bam.list \n",
      "INFO  13:31:32,934 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:31:32,935 HelpFormatter - Date/Time: 2017/10/24 13:31:32 \n",
      "INFO  13:31:32,935 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:32,935 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:32,940 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:31:33,125 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:31:33,132 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:31:33,171 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:31:33,347 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:31:33,374 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:31:33,374 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:31:33,375 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:31:33,375 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:31:40,312 ProgressMeter -            done     16100.0     6.0 s       7.2 m       99.5%     6.0 s       0.0 s \n",
      "INFO  13:31:40,313 ProgressMeter - Total runtime 6.94 secs, 0.12 min, 0.00 hours \n",
      "INFO  13:31:40,313 MicroScheduler - 79 reads were filtered out during the traversal out of approximately 1035718 total reads (0.01%) \n",
      "INFO  13:31:40,314 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:31:40,314 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:31:40,314 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:31:40,314 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:31:40,315 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:31:40,315 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:31:40,315 MicroScheduler -   -> 79 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:31:40,315 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:31:40,316 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:31:40,316 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R2_001_paired.bam -targetIntervals 2010AC19047_S13_L001_R2_001_paired.bam.list -o 2010AC19047_S13_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:31:42,104 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:42,106 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:31:42,106 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:31:42,106 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:31:42,106 HelpFormatter - [Tue Oct 24 13:31:42 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:31:42,106 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:31:42,109 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19047_S13_L001_R2_001_paired.bam -targetIntervals 2010AC19047_S13_L001_R2_001_paired.bam.list -o 2010AC19047_S13_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:31:42,112 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:31:42,113 HelpFormatter - Date/Time: 2017/10/24 13:31:42 \n",
      "INFO  13:31:42,113 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:42,113 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:31:42,189 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:31:42,276 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:31:42,282 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:31:42,328 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:31:42,490 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:31:42,494 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:31:42,494 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:31:42,495 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:31:42,495 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:31:42,523 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:31:42,524 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:31:44,286 IndelRealigner - Not attempting realignment in interval One_RAD56448:25-82 because there are too many reads. \n",
      "INFO  13:31:51,957 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:15-75 because there are too many reads. \n",
      "INFO  13:32:00,225 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:20-57 because there are too many reads. \n",
      "INFO  13:32:08,003 ProgressMeter -            done   1433096.0    25.0 s      17.0 s      100.0%    25.0 s       0.0 s \n",
      "INFO  13:32:08,003 ProgressMeter - Total runtime 25.51 secs, 0.43 min, 0.01 hours \n",
      "INFO  13:32:08,006 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1433096 total reads (0.00%) \n",
      "INFO  13:32:08,006 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:32:08,006 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19049_S14_L001_R2_001_paired.sam 2> 2010AC19049_S14_L001_R2_001_paired.txt2010AC19049_S14_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19049_S14_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19049_S14_L001_R2_001_paired.sam OUTPUT=2010AC19049_S14_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:32:38 PDT 2017] picard.sam.SortSam INPUT=2010AC19049_S14_L001_R2_001_paired.sam OUTPUT=2010AC19049_S14_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:32:38 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:32:41\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:32:45 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.12 minutes.\n",
      "Runtime.totalMemory()=1189609472\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19049_S14_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19049_S14_L001_R2_001_paired.bam.list\n",
      "INFO  13:32:46,971 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:46,973 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:32:46,973 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:32:46,973 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:32:46,973 HelpFormatter - [Tue Oct 24 13:32:46 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:32:46,973 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:32:46,977 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19049_S14_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19049_S14_L001_R2_001_paired.bam.list \n",
      "INFO  13:32:46,980 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:32:46,980 HelpFormatter - Date/Time: 2017/10/24 13:32:46 \n",
      "INFO  13:32:46,980 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:46,980 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:46,985 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:32:47,124 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:32:47,130 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:32:47,169 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:32:47,335 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:32:47,362 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:32:47,363 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:32:47,363 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:32:47,363 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:32:49,971 ProgressMeter -            done     16100.0     2.0 s       2.7 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:32:49,972 ProgressMeter - Total runtime 2.61 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:32:49,972 MicroScheduler - 22 reads were filtered out during the traversal out of approximately 284883 total reads (0.01%) \n",
      "INFO  13:32:49,972 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:32:49,973 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:32:49,973 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:32:49,973 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:32:49,974 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:32:49,974 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:32:49,974 MicroScheduler -   -> 22 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:32:49,975 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:32:49,975 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:32:49,975 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R2_001_paired.bam -targetIntervals 2010AC19049_S14_L001_R2_001_paired.bam.list -o 2010AC19049_S14_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:32:51,693 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:51,695 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:32:51,695 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:32:51,695 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:32:51,696 HelpFormatter - [Tue Oct 24 13:32:51 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:32:51,696 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:32:51,699 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19049_S14_L001_R2_001_paired.bam -targetIntervals 2010AC19049_S14_L001_R2_001_paired.bam.list -o 2010AC19049_S14_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:32:51,702 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:32:51,702 HelpFormatter - Date/Time: 2017/10/24 13:32:51 \n",
      "INFO  13:32:51,702 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:51,703 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:32:51,805 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:32:51,895 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:32:51,901 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:32:51,944 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:32:52,099 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:32:52,104 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:32:52,104 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:32:52,105 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:32:52,105 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:32:52,132 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:32:52,133 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:32:52,658 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-82 because there are too many reads. \n",
      "INFO  13:32:55,015 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-63 because there are too many reads. \n",
      "INFO  13:33:00,360 ProgressMeter -            done    378287.0     7.0 s      20.0 s      100.0%     7.0 s       0.0 s \n",
      "INFO  13:33:00,361 ProgressMeter - Total runtime 7.85 secs, 0.13 min, 0.00 hours \n",
      "INFO  13:33:00,364 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 378287 total reads (0.00%) \n",
      "INFO  13:33:00,364 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:33:00,364 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19050_S15_L001_R2_001_paired.sam 2> 2010AC19050_S15_L001_R2_001_paired.txt2010AC19050_S15_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19050_S15_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19050_S15_L001_R2_001_paired.sam OUTPUT=2010AC19050_S15_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:33:52 PDT 2017] picard.sam.SortSam INPUT=2010AC19050_S15_L001_R2_001_paired.sam OUTPUT=2010AC19050_S15_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:33:52 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:33:59\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:34:05 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.22 minutes.\n",
      "Runtime.totalMemory()=1446510592\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19050_S15_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19050_S15_L001_R2_001_paired.bam.list\n",
      "INFO  13:34:07,770 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:07,772 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:34:07,772 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:34:07,773 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:34:07,773 HelpFormatter - [Tue Oct 24 13:34:07 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:34:07,773 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:34:07,776 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19050_S15_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19050_S15_L001_R2_001_paired.bam.list \n",
      "INFO  13:34:07,779 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:34:07,779 HelpFormatter - Date/Time: 2017/10/24 13:34:07 \n",
      "INFO  13:34:07,780 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:07,780 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:07,785 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:34:07,920 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:34:07,926 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:34:07,967 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:34:08,144 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:34:08,171 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:34:08,172 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:34:08,172 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:34:08,172 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:34:12,027 ProgressMeter -            done     16100.0     3.0 s       4.0 m       99.5%     3.0 s       0.0 s \n",
      "INFO  13:34:12,028 ProgressMeter - Total runtime 3.86 secs, 0.06 min, 0.00 hours \n",
      "INFO  13:34:12,028 MicroScheduler - 60 reads were filtered out during the traversal out of approximately 507684 total reads (0.01%) \n",
      "INFO  13:34:12,028 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:34:12,029 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:34:12,029 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:34:12,029 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:34:12,029 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:34:12,030 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:34:12,030 MicroScheduler -   -> 60 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:34:12,030 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:34:12,031 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:34:12,031 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R2_001_paired.bam -targetIntervals 2010AC19050_S15_L001_R2_001_paired.bam.list -o 2010AC19050_S15_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:34:13,723 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:13,725 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:34:13,725 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:34:13,725 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:34:13,726 HelpFormatter - [Tue Oct 24 13:34:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:34:13,726 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:34:13,729 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19050_S15_L001_R2_001_paired.bam -targetIntervals 2010AC19050_S15_L001_R2_001_paired.bam.list -o 2010AC19050_S15_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:34:13,732 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:34:13,733 HelpFormatter - Date/Time: 2017/10/24 13:34:13 \n",
      "INFO  13:34:13,733 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:13,733 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:34:13,814 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:34:13,903 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:34:13,909 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:34:13,951 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:34:14,102 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:34:14,106 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:34:14,107 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:34:14,107 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:34:14,107 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:34:14,132 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:34:14,134 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:34:14,857 IndelRealigner - Not attempting realignment in interval One_RAD56448:24-59 because there are too many reads. \n",
      "INFO  13:34:19,314 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  13:34:28,617 ProgressMeter -            done    672628.0    14.0 s      21.0 s      100.0%    14.0 s       0.0 s \n",
      "INFO  13:34:28,618 ProgressMeter - Total runtime 14.51 secs, 0.24 min, 0.00 hours \n",
      "INFO  13:34:28,620 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 672628 total reads (0.00%) \n",
      "INFO  13:34:28,621 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:34:28,621 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19057_S16_L001_R2_001_paired.sam 2> 2010AC19057_S16_L001_R2_001_paired.txt2010AC19057_S16_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19057_S16_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19057_S16_L001_R2_001_paired.sam OUTPUT=2010AC19057_S16_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:35:26 PDT 2017] picard.sam.SortSam INPUT=2010AC19057_S16_L001_R2_001_paired.sam OUTPUT=2010AC19057_S16_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:35:26 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:35:37\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:35:44 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.29 minutes.\n",
      "Runtime.totalMemory()=1645740032\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19057_S16_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19057_S16_L001_R2_001_paired.bam.list\n",
      "INFO  13:35:46,484 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:46,486 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:35:46,486 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:35:46,487 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:35:46,487 HelpFormatter - [Tue Oct 24 13:35:46 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:35:46,487 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:35:46,490 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19057_S16_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19057_S16_L001_R2_001_paired.bam.list \n",
      "INFO  13:35:46,493 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:35:46,494 HelpFormatter - Date/Time: 2017/10/24 13:35:46 \n",
      "INFO  13:35:46,494 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:46,494 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:46,500 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:35:46,635 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:35:46,641 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:35:46,680 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:35:46,846 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:35:46,873 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:35:46,873 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:35:46,874 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:35:46,874 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:35:50,853 ProgressMeter -            done     16100.0     3.0 s       4.1 m       99.5%     3.0 s       0.0 s \n",
      "INFO  13:35:50,854 ProgressMeter - Total runtime 3.98 secs, 0.07 min, 0.00 hours \n",
      "INFO  13:35:50,855 MicroScheduler - 77 reads were filtered out during the traversal out of approximately 563283 total reads (0.01%) \n",
      "INFO  13:35:50,855 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:35:50,855 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:35:50,856 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:35:50,856 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:35:50,856 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:35:50,856 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:35:50,857 MicroScheduler -   -> 77 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:35:50,857 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:35:50,857 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:35:50,857 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R2_001_paired.bam -targetIntervals 2010AC19057_S16_L001_R2_001_paired.bam.list -o 2010AC19057_S16_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:35:52,628 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:52,630 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:35:52,631 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:35:52,631 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:35:52,631 HelpFormatter - [Tue Oct 24 13:35:52 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:35:52,631 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:35:52,634 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19057_S16_L001_R2_001_paired.bam -targetIntervals 2010AC19057_S16_L001_R2_001_paired.bam.list -o 2010AC19057_S16_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:35:52,638 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:35:52,638 HelpFormatter - Date/Time: 2017/10/24 13:35:52 \n",
      "INFO  13:35:52,638 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:52,638 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:35:52,723 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:35:52,812 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:35:52,818 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:35:52,863 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:35:53,029 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:35:53,033 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:35:53,034 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:35:53,034 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:35:53,034 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:35:53,093 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:35:53,097 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:35:53,744 IndelRealigner - Not attempting realignment in interval One_RAD56448:27-56 because there are too many reads. \n",
      "INFO  13:35:59,757 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-67 because there are too many reads. \n",
      "INFO  13:36:09,554 ProgressMeter -            done    694333.0    16.0 s      23.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  13:36:09,555 ProgressMeter - Total runtime 16.52 secs, 0.28 min, 0.00 hours \n",
      "INFO  13:36:09,557 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 694333 total reads (0.00%) \n",
      "INFO  13:36:09,557 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:36:09,558 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19091_S18_L001_R2_001_paired.sam 2> 2010AC19091_S18_L001_R2_001_paired.txt2010AC19091_S18_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19091_S18_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19091_S18_L001_R2_001_paired.sam OUTPUT=2010AC19091_S18_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:37:30 PDT 2017] picard.sam.SortSam INPUT=2010AC19091_S18_L001_R2_001_paired.sam OUTPUT=2010AC19091_S18_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:37:30 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:37:43\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:37:53 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.38 minutes.\n",
      "Runtime.totalMemory()=2064121856\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19091_S18_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19091_S18_L001_R2_001_paired.bam.list\n",
      "INFO  13:37:55,256 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:37:55,259 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:37:55,259 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:37:55,259 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:37:55,259 HelpFormatter - [Tue Oct 24 13:37:55 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:37:55,260 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:37:55,263 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19091_S18_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19091_S18_L001_R2_001_paired.bam.list \n",
      "INFO  13:37:55,266 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:37:55,266 HelpFormatter - Date/Time: 2017/10/24 13:37:55 \n",
      "INFO  13:37:55,266 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:37:55,267 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:37:55,272 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:37:55,426 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:37:55,432 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:37:55,471 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:37:55,625 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:37:55,652 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:37:55,652 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:37:55,653 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:37:55,653 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:38:01,117 ProgressMeter -            done     16100.0     5.0 s       5.7 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:38:01,118 ProgressMeter - Total runtime 5.47 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:38:01,118 MicroScheduler - 102 reads were filtered out during the traversal out of approximately 786060 total reads (0.01%) \n",
      "INFO  13:38:01,119 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:38:01,119 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:38:01,119 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:38:01,119 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:38:01,120 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:38:01,120 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:38:01,120 MicroScheduler -   -> 102 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:38:01,121 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:38:01,121 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:38:01,121 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R2_001_paired.bam -targetIntervals 2010AC19091_S18_L001_R2_001_paired.bam.list -o 2010AC19091_S18_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:38:02,903 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:38:02,905 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:38:02,905 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:38:02,906 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:38:02,906 HelpFormatter - [Tue Oct 24 13:38:02 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:38:02,906 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:38:02,909 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19091_S18_L001_R2_001_paired.bam -targetIntervals 2010AC19091_S18_L001_R2_001_paired.bam.list -o 2010AC19091_S18_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:38:02,912 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:38:02,913 HelpFormatter - Date/Time: 2017/10/24 13:38:02 \n",
      "INFO  13:38:02,913 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:38:02,913 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:38:02,990 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:38:03,121 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:38:03,128 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:38:03,192 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.06 \n",
      "INFO  13:38:03,391 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:38:03,395 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:38:03,395 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:38:03,396 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:38:03,396 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:38:03,459 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:38:03,460 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:38:04,690 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-81 because there are too many reads. \n",
      "INFO  13:38:10,121 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-77 because there are too many reads. \n",
      "INFO  13:38:24,847 ProgressMeter -            done    966717.0    21.0 s      22.0 s      100.0%    21.0 s       0.0 s \n",
      "INFO  13:38:24,848 ProgressMeter - Total runtime 21.45 secs, 0.36 min, 0.01 hours \n",
      "INFO  13:38:24,851 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 966717 total reads (0.00%) \n",
      "INFO  13:38:24,851 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:38:24,852 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19096_S21_L001_R2_001_paired.sam 2> 2010AC19096_S21_L001_R2_001_paired.txt2010AC19096_S21_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19096_S21_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19096_S21_L001_R2_001_paired.sam OUTPUT=2010AC19096_S21_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:39:23 PDT 2017] picard.sam.SortSam INPUT=2010AC19096_S21_L001_R2_001_paired.sam OUTPUT=2010AC19096_S21_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:39:23 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:39:32\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:39:39 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.26 minutes.\n",
      "Runtime.totalMemory()=1736441856\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19096_S21_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19096_S21_L001_R2_001_paired.bam.list\n",
      "INFO  13:39:40,981 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:40,983 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:39:40,984 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:39:40,984 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:39:40,984 HelpFormatter - [Tue Oct 24 13:39:40 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:39:40,984 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:39:40,988 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19096_S21_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19096_S21_L001_R2_001_paired.bam.list \n",
      "INFO  13:39:40,991 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:39:40,991 HelpFormatter - Date/Time: 2017/10/24 13:39:40 \n",
      "INFO  13:39:40,991 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:40,991 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:40,996 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:39:41,137 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:39:41,143 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:39:41,189 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:39:41,392 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:39:41,419 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:39:41,419 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:39:41,420 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:39:41,420 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:39:45,252 ProgressMeter -            done     16100.0     3.0 s       4.0 m       99.5%     3.0 s       0.0 s \n",
      "INFO  13:39:45,253 ProgressMeter - Total runtime 3.83 secs, 0.06 min, 0.00 hours \n",
      "INFO  13:39:45,253 MicroScheduler - 66 reads were filtered out during the traversal out of approximately 566938 total reads (0.01%) \n",
      "INFO  13:39:45,254 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:39:45,254 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:39:45,254 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:39:45,254 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:39:45,255 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:39:45,255 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:39:45,255 MicroScheduler -   -> 66 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:39:45,256 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:39:45,256 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:39:45,256 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R2_001_paired.bam -targetIntervals 2010AC19096_S21_L001_R2_001_paired.bam.list -o 2010AC19096_S21_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:39:47,150 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:47,153 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:39:47,153 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:39:47,153 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:39:47,153 HelpFormatter - [Tue Oct 24 13:39:47 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:39:47,154 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:39:47,157 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19096_S21_L001_R2_001_paired.bam -targetIntervals 2010AC19096_S21_L001_R2_001_paired.bam.list -o 2010AC19096_S21_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:39:47,160 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:39:47,160 HelpFormatter - Date/Time: 2017/10/24 13:39:47 \n",
      "INFO  13:39:47,160 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:47,161 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:39:47,246 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:39:47,342 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:39:47,348 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:39:47,398 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:39:47,563 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:39:47,567 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:39:47,568 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:39:47,568 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:39:47,568 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:39:47,595 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:39:47,596 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:39:48,457 IndelRealigner - Not attempting realignment in interval One_RAD56448:25-66 because there are too many reads. \n",
      "INFO  13:39:53,440 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  13:40:04,431 ProgressMeter -            done    716968.0    16.0 s      22.0 s      100.0%    16.0 s       0.0 s \n",
      "INFO  13:40:04,431 ProgressMeter - Total runtime 16.46 secs, 0.27 min, 0.00 hours \n",
      "INFO  13:40:04,434 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 716968 total reads (0.00%) \n",
      "INFO  13:40:04,434 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:40:04,434 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19098_S22_L001_R2_001_paired.sam 2> 2010AC19098_S22_L001_R2_001_paired.txt2010AC19098_S22_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19098_S22_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19098_S22_L001_R2_001_paired.sam OUTPUT=2010AC19098_S22_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:41:07 PDT 2017] picard.sam.SortSam INPUT=2010AC19098_S22_L001_R2_001_paired.sam OUTPUT=2010AC19098_S22_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:41:07 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:41:18\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:41:25 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.30 minutes.\n",
      "Runtime.totalMemory()=1821900800\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19098_S22_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19098_S22_L001_R2_001_paired.bam.list\n",
      "INFO  13:41:27,350 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:27,353 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:41:27,353 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:41:27,353 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:41:27,353 HelpFormatter - [Tue Oct 24 13:41:27 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:41:27,353 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:41:27,357 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19098_S22_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19098_S22_L001_R2_001_paired.bam.list \n",
      "INFO  13:41:27,360 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:41:27,360 HelpFormatter - Date/Time: 2017/10/24 13:41:27 \n",
      "INFO  13:41:27,360 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:27,361 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:27,367 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:41:27,516 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:41:27,522 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:41:27,559 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:41:27,715 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:41:27,740 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:41:27,741 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:41:27,742 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:41:27,742 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R2_001_paired.bam -targetIntervals 2010AC19098_S22_L001_R2_001_paired.bam.list -o 2010AC19098_S22_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:41:33,587 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:33,589 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:41:33,589 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:41:33,590 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:41:33,590 HelpFormatter - [Tue Oct 24 13:41:33 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:41:33,590 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:41:33,593 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19098_S22_L001_R2_001_paired.bam -targetIntervals 2010AC19098_S22_L001_R2_001_paired.bam.list -o 2010AC19098_S22_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:41:33,596 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:41:33,596 HelpFormatter - Date/Time: 2017/10/24 13:41:33 \n",
      "INFO  13:41:33,597 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:33,597 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:41:33,671 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:41:33,755 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:41:33,762 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:41:33,801 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:41:33,954 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:41:33,957 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:41:33,958 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:41:33,958 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:41:33,958 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: Could not read file /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19098_S22_L001_R2_001_paired.bam.list because The interval file does not exist.\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19099_S19_L001_R2_001_paired.sam 2> 2010AC19099_S19_L001_R2_001_paired.txt2010AC19099_S19_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19099_S19_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19099_S19_L001_R2_001_paired.sam OUTPUT=2010AC19099_S19_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:43:04 PDT 2017] picard.sam.SortSam INPUT=2010AC19099_S19_L001_R2_001_paired.sam OUTPUT=2010AC19099_S19_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:43:04 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:43:20\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:43:28 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.41 minutes.\n",
      "Runtime.totalMemory()=2108162048\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19099_S19_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19099_S19_L001_R2_001_paired.bam.list\n",
      "INFO  13:43:30,550 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:30,553 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:43:30,553 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:43:30,553 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:43:30,554 HelpFormatter - [Tue Oct 24 13:43:30 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:43:30,554 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:43:30,557 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19099_S19_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19099_S19_L001_R2_001_paired.bam.list \n",
      "INFO  13:43:30,560 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:43:30,560 HelpFormatter - Date/Time: 2017/10/24 13:43:30 \n",
      "INFO  13:43:30,561 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:30,561 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:30,566 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:43:30,700 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:43:30,706 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:43:30,744 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:43:30,896 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:43:30,921 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:43:30,921 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:43:30,922 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:43:30,922 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:43:36,943 ProgressMeter -            done     16100.0     6.0 s       6.2 m       99.5%     6.0 s       0.0 s \n",
      "INFO  13:43:36,944 ProgressMeter - Total runtime 6.02 secs, 0.10 min, 0.00 hours \n",
      "INFO  13:43:36,945 MicroScheduler - 127 reads were filtered out during the traversal out of approximately 867011 total reads (0.01%) \n",
      "INFO  13:43:36,945 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:43:36,945 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:43:36,945 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:43:36,946 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:43:36,946 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:43:36,946 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:43:36,947 MicroScheduler -   -> 127 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:43:36,947 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:43:36,947 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:43:36,948 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R2_001_paired.bam -targetIntervals 2010AC19099_S19_L001_R2_001_paired.bam.list -o 2010AC19099_S19_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:43:38,728 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:38,730 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:43:38,731 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:43:38,731 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:43:38,731 HelpFormatter - [Tue Oct 24 13:43:38 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:43:38,731 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:43:38,734 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19099_S19_L001_R2_001_paired.bam -targetIntervals 2010AC19099_S19_L001_R2_001_paired.bam.list -o 2010AC19099_S19_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:43:38,737 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:43:38,738 HelpFormatter - Date/Time: 2017/10/24 13:43:38 \n",
      "INFO  13:43:38,738 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:38,738 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:43:38,818 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:43:38,907 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:43:38,913 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:43:38,957 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:43:39,108 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:43:39,112 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:43:39,112 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:43:39,113 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:43:39,113 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:43:39,141 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:43:39,142 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:43:40,752 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-84 because there are too many reads. \n",
      "INFO  13:43:47,045 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  13:44:04,259 ProgressMeter -            done   1078742.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  13:44:04,259 ProgressMeter - Total runtime 24.69 secs, 0.41 min, 0.01 hours \n",
      "INFO  13:44:04,262 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1078742 total reads (0.00%) \n",
      "INFO  13:44:04,262 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:44:04,262 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19101_S23_L001_R2_001_paired.sam 2> 2010AC19101_S23_L001_R2_001_paired.txt2010AC19101_S23_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19101_S23_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19101_S23_L001_R2_001_paired.sam OUTPUT=2010AC19101_S23_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:45:44 PDT 2017] picard.sam.SortSam INPUT=2010AC19101_S23_L001_R2_001_paired.sam OUTPUT=2010AC19101_S23_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:45:44 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:46:02\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:46:12 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.46 minutes.\n",
      "Runtime.totalMemory()=2078277632\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19101_S23_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19101_S23_L001_R2_001_paired.bam.list\n",
      "INFO  13:46:14,022 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:14,024 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:46:14,024 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:46:14,024 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:46:14,025 HelpFormatter - [Tue Oct 24 13:46:13 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:46:14,025 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:46:14,028 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19101_S23_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19101_S23_L001_R2_001_paired.bam.list \n",
      "INFO  13:46:14,031 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:46:14,031 HelpFormatter - Date/Time: 2017/10/24 13:46:14 \n",
      "INFO  13:46:14,032 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:14,032 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:14,037 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:46:14,174 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:46:14,180 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:46:14,219 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:46:14,423 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:46:14,452 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:46:14,453 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:46:14,453 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:46:14,454 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:46:20,947 ProgressMeter -            done     16100.0     6.0 s       6.7 m       99.5%     6.0 s       0.0 s \n",
      "INFO  13:46:20,948 ProgressMeter - Total runtime 6.50 secs, 0.11 min, 0.00 hours \n",
      "INFO  13:46:20,948 MicroScheduler - 121 reads were filtered out during the traversal out of approximately 1026320 total reads (0.01%) \n",
      "INFO  13:46:20,948 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:46:20,949 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:46:20,949 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:46:20,949 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:46:20,950 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:46:20,950 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:46:20,950 MicroScheduler -   -> 121 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:46:20,950 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:46:20,951 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:46:20,951 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R2_001_paired.bam -targetIntervals 2010AC19101_S23_L001_R2_001_paired.bam.list -o 2010AC19101_S23_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:46:22,720 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:22,722 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:46:22,722 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:46:22,722 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:46:22,723 HelpFormatter - [Tue Oct 24 13:46:22 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:46:22,723 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:46:22,726 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19101_S23_L001_R2_001_paired.bam -targetIntervals 2010AC19101_S23_L001_R2_001_paired.bam.list -o 2010AC19101_S23_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:46:22,729 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:46:22,729 HelpFormatter - Date/Time: 2017/10/24 13:46:22 \n",
      "INFO  13:46:22,729 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:22,730 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:46:22,806 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:46:22,897 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:46:22,902 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:46:22,947 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:46:23,101 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:46:23,105 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:46:23,106 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:46:23,106 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:46:23,106 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:46:23,133 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:46:23,134 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:46:24,711 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-82 because there are too many reads. \n",
      "INFO  13:46:32,498 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  13:46:43,518 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:14-64 because there are too many reads. \n",
      "INFO  13:46:47,967 ProgressMeter -            done   1209501.0    24.0 s      20.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  13:46:47,968 ProgressMeter - Total runtime 24.86 secs, 0.41 min, 0.01 hours \n",
      "INFO  13:46:47,971 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1209501 total reads (0.00%) \n",
      "INFO  13:46:47,972 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:46:47,972 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19102_S20_L001_R2_001_paired.sam 2> 2010AC19102_S20_L001_R2_001_paired.txt2010AC19102_S20_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19102_S20_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19102_S20_L001_R2_001_paired.sam OUTPUT=2010AC19102_S20_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:48:22 PDT 2017] picard.sam.SortSam INPUT=2010AC19102_S20_L001_R2_001_paired.sam OUTPUT=2010AC19102_S20_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:48:22 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:48:38\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:48:47 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.41 minutes.\n",
      "Runtime.totalMemory()=2000158720\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19102_S20_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19102_S20_L001_R2_001_paired.bam.list\n",
      "INFO  13:48:48,933 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:48,935 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:48:48,936 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:48:48,936 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:48:48,936 HelpFormatter - [Tue Oct 24 13:48:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:48:48,936 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:48:48,939 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19102_S20_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19102_S20_L001_R2_001_paired.bam.list \n",
      "INFO  13:48:48,942 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:48:48,943 HelpFormatter - Date/Time: 2017/10/24 13:48:48 \n",
      "INFO  13:48:48,943 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:48,943 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:48,948 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:48:49,106 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:48:49,112 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:48:49,154 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:48:49,312 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:48:49,339 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:48:49,339 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:48:49,340 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:48:49,340 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:48:55,299 ProgressMeter -            done     16100.0     5.0 s       6.2 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:48:55,300 ProgressMeter - Total runtime 5.96 secs, 0.10 min, 0.00 hours \n",
      "INFO  13:48:55,300 MicroScheduler - 86 reads were filtered out during the traversal out of approximately 906599 total reads (0.01%) \n",
      "INFO  13:48:55,301 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:48:55,301 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:48:55,301 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:48:55,302 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:48:55,302 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:48:55,302 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:48:55,302 MicroScheduler -   -> 86 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:48:55,303 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:48:55,303 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:48:55,303 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R2_001_paired.bam -targetIntervals 2010AC19102_S20_L001_R2_001_paired.bam.list -o 2010AC19102_S20_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:48:57,113 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:57,115 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:48:57,115 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:48:57,115 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:48:57,116 HelpFormatter - [Tue Oct 24 13:48:57 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:48:57,116 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:48:57,119 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19102_S20_L001_R2_001_paired.bam -targetIntervals 2010AC19102_S20_L001_R2_001_paired.bam.list -o 2010AC19102_S20_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:48:57,122 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:48:57,123 HelpFormatter - Date/Time: 2017/10/24 13:48:57 \n",
      "INFO  13:48:57,123 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:57,123 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:48:57,208 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:48:57,313 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:48:57,320 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:48:57,367 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:48:57,528 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:48:57,532 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:48:57,533 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:48:57,533 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:48:57,533 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:48:57,557 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:48:57,559 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:48:58,560 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-77 because there are too many reads. \n",
      "INFO  13:49:04,406 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-77 because there are too many reads. \n",
      "INFO  13:49:22,613 ProgressMeter -            done   1100757.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  13:49:22,613 ProgressMeter - Total runtime 24.64 secs, 0.41 min, 0.01 hours \n",
      "INFO  13:49:22,616 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1100757 total reads (0.00%) \n",
      "INFO  13:49:22,616 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:49:22,617 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19105_S24_L001_R2_001_paired.sam 2> 2010AC19105_S24_L001_R2_001_paired.txt2010AC19105_S24_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19105_S24_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19105_S24_L001_R2_001_paired.sam OUTPUT=2010AC19105_S24_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:51:59 PDT 2017] picard.sam.SortSam INPUT=2010AC19105_S24_L001_R2_001_paired.sam OUTPUT=2010AC19105_S24_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:51:59 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:52:28\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:52:32 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.56 minutes.\n",
      "Runtime.totalMemory()=2103443456\n",
      "To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp\n",
      "Exception in thread \"main\" htsjdk.samtools.SAMException: Exception when processing alignment for BAM index M02215:12:000000000-BB9C2:1:1104:9502:23215 76b aligned read.\n",
      "\tat htsjdk.samtools.BAMFileWriter.writeAlignment(BAMFileWriter.java:131)\n",
      "\tat htsjdk.samtools.SAMFileWriterImpl.close(SAMFileWriterImpl.java:214)\n",
      "\tat picard.sam.SortSam.doWork(SortSam.java:107)\n",
      "\tat picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:205)\n",
      "\tat picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:94)\n",
      "\tat picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:104)\n",
      "Caused by: htsjdk.samtools.SAMException: Exception creating BAM index for record M02215:12:000000000-BB9C2:1:1104:9502:23215 76b aligned read.\n",
      "\tat htsjdk.samtools.BAMIndexer.processAlignment(BAMIndexer.java:110)\n",
      "\tat htsjdk.samtools.BAMFileWriter.writeAlignment(BAMFileWriter.java:128)\n",
      "\t... 5 more\n",
      "Caused by: htsjdk.samtools.SAMException: Unexpected reference 0 when constructing index for 22 for record M02215:12:000000000-BB9C2:1:1104:9502:23215 76b aligned read.\n",
      "\tat htsjdk.samtools.BAMIndexer$BAMIndexBuilder.processAlignment(BAMIndexer.java:209)\n",
      "\tat htsjdk.samtools.BAMIndexer.processAlignment(BAMIndexer.java:108)\n",
      "\t... 6 more\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19105_S24_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19105_S24_L001_R2_001_paired.bam.list\n",
      "INFO  13:52:34,795 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:34,797 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:52:34,797 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:52:34,797 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:52:34,798 HelpFormatter - [Tue Oct 24 13:52:34 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:52:34,798 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:52:34,801 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19105_S24_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19105_S24_L001_R2_001_paired.bam.list \n",
      "INFO  13:52:34,805 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:52:34,805 HelpFormatter - Date/Time: 2017/10/24 13:52:34 \n",
      "INFO  13:52:34,806 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:34,806 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:34,811 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:52:34,956 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:52:34,962 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:52:35,002 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:52:35,165 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:52:35,196 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:52:35,196 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:52:35,197 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:52:35,197 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: File /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R2_001_paired.bai is malformed: Premature end-of-file while reading BAM index file 2010AC19105_S24_L001_R2_001_paired.bai. It's likely that this file is truncated or corrupt -- Please try re-indexing the corresponding BAM file.\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R2_001_paired.bam -targetIntervals 2010AC19105_S24_L001_R2_001_paired.bam.list -o 2010AC19105_S24_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:52:40,088 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:40,090 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:52:40,090 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:52:40,090 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:52:40,090 HelpFormatter - [Tue Oct 24 13:52:40 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:52:40,091 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:52:40,094 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19105_S24_L001_R2_001_paired.bam -targetIntervals 2010AC19105_S24_L001_R2_001_paired.bam.list -o 2010AC19105_S24_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:52:40,097 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:52:40,097 HelpFormatter - Date/Time: 2017/10/24 13:52:40 \n",
      "INFO  13:52:40,097 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:40,098 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:52:40,174 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:52:40,358 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:52:40,366 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:52:40,414 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  13:52:40,609 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:52:40,613 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:52:40,613 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:52:40,614 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:52:40,614 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: Could not read file /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19105_S24_L001_R2_001_paired.bam.list because The interval file does not exist.\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19559_S3_L001_R2_001_paired.sam 2> 2010AC19559_S3_L001_R2_001_paired.txt:2010AC19559_S3_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19559_S3_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19559_S3_L001_R2_001_paired.sam OUTPUT=2010AC19559_S3_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:54:05 PDT 2017] picard.sam.SortSam INPUT=2010AC19559_S3_L001_R2_001_paired.sam OUTPUT=2010AC19559_S3_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:54:05 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:54:18\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:54:26 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=1959788544\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19559_S3_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19559_S3_L001_R2_001_paired.bam.list\n",
      "INFO  13:54:28,511 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:28,513 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:54:28,514 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:54:28,514 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:54:28,514 HelpFormatter - [Tue Oct 24 13:54:28 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:54:28,514 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:54:28,518 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19559_S3_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19559_S3_L001_R2_001_paired.bam.list \n",
      "INFO  13:54:28,521 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:54:28,521 HelpFormatter - Date/Time: 2017/10/24 13:54:28 \n",
      "INFO  13:54:28,521 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:28,521 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:28,527 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:54:28,664 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:54:28,670 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:54:28,710 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:54:28,864 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:54:28,892 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:54:28,893 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:54:28,893 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:54:28,893 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:54:34,238 ProgressMeter -            done     16100.0     5.0 s       5.5 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:54:34,239 ProgressMeter - Total runtime 5.35 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:54:34,239 MicroScheduler - 114 reads were filtered out during the traversal out of approximately 811476 total reads (0.01%) \n",
      "INFO  13:54:34,239 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:54:34,240 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:54:34,240 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:54:34,240 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:54:34,240 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:54:34,241 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:54:34,241 MicroScheduler -   -> 114 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:54:34,241 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:54:34,241 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:54:34,242 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R2_001_paired.bam -targetIntervals 2010AC19559_S3_L001_R2_001_paired.bam.list -o 2010AC19559_S3_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:54:35,995 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:35,997 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:54:35,997 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:54:35,997 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:54:35,998 HelpFormatter - [Tue Oct 24 13:54:35 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:54:35,998 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:54:36,001 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19559_S3_L001_R2_001_paired.bam -targetIntervals 2010AC19559_S3_L001_R2_001_paired.bam.list -o 2010AC19559_S3_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:54:36,004 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:54:36,005 HelpFormatter - Date/Time: 2017/10/24 13:54:35 \n",
      "INFO  13:54:36,005 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:36,005 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:54:36,087 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:54:36,173 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:54:36,179 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:54:36,220 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:54:36,378 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:54:36,382 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:54:36,383 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:54:36,383 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:54:36,383 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:54:36,410 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:54:36,411 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:54:37,966 IndelRealigner - Not attempting realignment in interval One_RAD56448:7-61 because there are too many reads. \n",
      "INFO  13:54:43,657 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-77 because there are too many reads. \n",
      "INFO  13:55:00,868 ProgressMeter -            done   1106227.0    24.0 s      22.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  13:55:00,868 ProgressMeter - Total runtime 24.49 secs, 0.41 min, 0.01 hours \n",
      "INFO  13:55:00,871 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1106227 total reads (0.00%) \n",
      "INFO  13:55:00,871 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:55:00,871 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19565_S4_L001_R2_001_paired.sam 2> 2010AC19565_S4_L001_R2_001_paired.txt:2010AC19565_S4_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19565_S4_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19565_S4_L001_R2_001_paired.sam OUTPUT=2010AC19565_S4_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:56:12 PDT 2017] picard.sam.SortSam INPUT=2010AC19565_S4_L001_R2_001_paired.sam OUTPUT=2010AC19565_S4_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:56:12 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:56:22\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:56:22 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.18 minutes.\n",
      "Runtime.totalMemory()=1788870656\n",
      "To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp\n",
      "Exception in thread \"main\" java.lang.NullPointerException\n",
      "\tat htsjdk.samtools.Cigar.numCigarElements(Cigar.java:66)\n",
      "\tat htsjdk.samtools.SAMRecord.getCigarLength(SAMRecord.java:812)\n",
      "\tat htsjdk.samtools.BAMRecordCodec.encode(BAMRecordCodec.java:92)\n",
      "\tat htsjdk.samtools.BAMRecordCodec.encode(BAMRecordCodec.java:37)\n",
      "\tat htsjdk.samtools.util.SortingCollection.spillToDisk(SortingCollection.java:226)\n",
      "\tat htsjdk.samtools.util.SortingCollection.doneAdding(SortingCollection.java:191)\n",
      "\tat htsjdk.samtools.util.SortingCollection.iterator(SortingCollection.java:266)\n",
      "\tat htsjdk.samtools.SAMFileWriterImpl.close(SAMFileWriterImpl.java:213)\n",
      "\tat picard.sam.SortSam.doWork(SortSam.java:107)\n",
      "\tat picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:205)\n",
      "\tat picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:94)\n",
      "\tat picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:104)\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19565_S4_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19565_S4_L001_R2_001_paired.bam.list\n",
      "INFO  13:56:24,774 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:24,776 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:56:24,776 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:56:24,776 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:56:24,776 HelpFormatter - [Tue Oct 24 13:56:24 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:56:24,777 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:56:24,780 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19565_S4_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19565_S4_L001_R2_001_paired.bam.list \n",
      "INFO  13:56:24,783 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:56:24,783 HelpFormatter - Date/Time: 2017/10/24 13:56:24 \n",
      "INFO  13:56:24,784 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:24,784 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:24,789 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:56:24,932 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:56:24,938 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: SAM/BAM/CRAM file 2010AC19565_S4_L001_R2_001_paired.bam is malformed. Please see https://software.broadinstitute.org/gatk/documentation/article?id=1317for more information. Error details: Cannot use index file with textual SAM file\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R2_001_paired.bam -targetIntervals 2010AC19565_S4_L001_R2_001_paired.bam.list -o 2010AC19565_S4_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:56:26,657 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:26,659 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:56:26,660 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:56:26,660 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:56:26,660 HelpFormatter - [Tue Oct 24 13:56:26 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:56:26,660 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:56:26,663 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19565_S4_L001_R2_001_paired.bam -targetIntervals 2010AC19565_S4_L001_R2_001_paired.bam.list -o 2010AC19565_S4_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:56:26,666 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:56:26,666 HelpFormatter - Date/Time: 2017/10/24 13:56:26 \n",
      "INFO  13:56:26,667 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:26,667 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:56:26,751 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:56:26,836 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:56:26,842 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      "##### ERROR A USER ERROR has occurred (version 3.7-0-gcfedb67): \n",
      "##### ERROR\n",
      "##### ERROR This means that one or more arguments or inputs in your command are incorrect.\n",
      "##### ERROR The error message below tells you what is the problem.\n",
      "##### ERROR\n",
      "##### ERROR If the problem is an invalid argument, please check the online documentation guide\n",
      "##### ERROR (or rerun your command with --help) to view allowable command-line arguments for this tool.\n",
      "##### ERROR\n",
      "##### ERROR Visit our website and forum for extensive documentation and answers to \n",
      "##### ERROR commonly asked questions https://software.broadinstitute.org/gatk\n",
      "##### ERROR\n",
      "##### ERROR Please do NOT post this error to the GATK forum unless you have really tried to fix it yourself.\n",
      "##### ERROR\n",
      "##### ERROR MESSAGE: SAM/BAM/CRAM file 2010AC19565_S4_L001_R2_001_paired.bam is malformed. Please see https://software.broadinstitute.org/gatk/documentation/article?id=1317for more information. Error details: Cannot use index file with textual SAM file\n",
      "##### ERROR ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19685_S5_L001_R2_001_paired.sam 2> 2010AC19685_S5_L001_R2_001_paired.txt:2010AC19685_S5_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19685_S5_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19685_S5_L001_R2_001_paired.sam OUTPUT=2010AC19685_S5_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:57:44 PDT 2017] picard.sam.SortSam INPUT=2010AC19685_S5_L001_R2_001_paired.sam OUTPUT=2010AC19685_S5_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:57:44 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:57:57\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:58:05 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.35 minutes.\n",
      "Runtime.totalMemory()=2016411648\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19685_S5_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19685_S5_L001_R2_001_paired.bam.list\n",
      "INFO  13:58:07,775 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:07,778 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:58:07,778 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:58:07,778 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:58:07,778 HelpFormatter - [Tue Oct 24 13:58:07 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:58:07,778 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:58:07,782 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19685_S5_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19685_S5_L001_R2_001_paired.bam.list \n",
      "INFO  13:58:07,785 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:58:07,785 HelpFormatter - Date/Time: 2017/10/24 13:58:07 \n",
      "INFO  13:58:07,785 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:07,785 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:07,790 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:58:07,939 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:58:07,945 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:58:07,986 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:58:08,142 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:58:08,167 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:58:08,168 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:58:08,168 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:58:08,168 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:58:13,710 ProgressMeter -            done     16100.0     5.0 s       5.7 m       99.5%     5.0 s       0.0 s \n",
      "INFO  13:58:13,710 ProgressMeter - Total runtime 5.54 secs, 0.09 min, 0.00 hours \n",
      "INFO  13:58:13,711 MicroScheduler - 69 reads were filtered out during the traversal out of approximately 764887 total reads (0.01%) \n",
      "INFO  13:58:13,711 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:58:13,712 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:58:13,712 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:58:13,712 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:58:13,712 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:58:13,713 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:58:13,713 MicroScheduler -   -> 69 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:58:13,713 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:58:13,713 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:58:13,714 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R2_001_paired.bam -targetIntervals 2010AC19685_S5_L001_R2_001_paired.bam.list -o 2010AC19685_S5_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:58:15,492 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:15,494 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:58:15,494 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:58:15,494 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:58:15,494 HelpFormatter - [Tue Oct 24 13:58:15 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:58:15,495 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:58:15,498 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19685_S5_L001_R2_001_paired.bam -targetIntervals 2010AC19685_S5_L001_R2_001_paired.bam.list -o 2010AC19685_S5_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:58:15,501 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:58:15,501 HelpFormatter - Date/Time: 2017/10/24 13:58:15 \n",
      "INFO  13:58:15,501 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:15,501 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:58:15,579 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:58:15,666 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:58:15,672 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:58:15,718 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:58:15,868 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:58:15,872 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:58:15,873 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:58:15,873 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:58:15,873 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:58:15,899 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:58:15,900 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:58:17,471 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-65 because there are too many reads. \n",
      "INFO  13:58:23,973 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  13:58:40,019 ProgressMeter -            done   1053330.0    23.0 s      22.0 s      100.0%    23.0 s       0.0 s \n",
      "INFO  13:58:40,019 ProgressMeter - Total runtime 23.67 secs, 0.39 min, 0.01 hours \n",
      "INFO  13:58:40,022 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1053330 total reads (0.00%) \n",
      "INFO  13:58:40,022 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:58:40,023 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19689_S6_L001_R2_001_paired.sam 2> 2010AC19689_S6_L001_R2_001_paired.txt:2010AC19689_S6_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19689_S6_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19689_S6_L001_R2_001_paired.sam OUTPUT=2010AC19689_S6_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 13:59:02 PDT 2017] picard.sam.SortSam INPUT=2010AC19689_S6_L001_R2_001_paired.sam OUTPUT=2010AC19689_S6_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 13:59:02 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 13:59:05\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 13:59:08 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.09 minutes.\n",
      "Runtime.totalMemory()=1005060096\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19689_S6_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19689_S6_L001_R2_001_paired.bam.list\n",
      "INFO  13:59:09,957 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:09,959 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:59:09,959 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:59:09,959 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:59:09,959 HelpFormatter - [Tue Oct 24 13:59:09 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:59:09,960 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:59:09,963 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19689_S6_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19689_S6_L001_R2_001_paired.bam.list \n",
      "INFO  13:59:09,966 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:59:09,966 HelpFormatter - Date/Time: 2017/10/24 13:59:09 \n",
      "INFO  13:59:09,966 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:09,966 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:09,971 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:59:10,107 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  13:59:10,113 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:59:10,152 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:59:10,321 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:59:10,349 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:59:10,349 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:59:10,349 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:59:10,350 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  13:59:12,773 ProgressMeter -            done     16100.0     2.0 s       2.5 m       99.5%     2.0 s       0.0 s \n",
      "INFO  13:59:12,773 ProgressMeter - Total runtime 2.42 secs, 0.04 min, 0.00 hours \n",
      "INFO  13:59:12,774 MicroScheduler - 25 reads were filtered out during the traversal out of approximately 194922 total reads (0.01%) \n",
      "INFO  13:59:12,774 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:59:12,775 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  13:59:12,775 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  13:59:12,775 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  13:59:12,775 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  13:59:12,776 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  13:59:12,776 MicroScheduler -   -> 25 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  13:59:12,776 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  13:59:12,776 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  13:59:12,777 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R2_001_paired.bam -targetIntervals 2010AC19689_S6_L001_R2_001_paired.bam.list -o 2010AC19689_S6_L001_R2_001_paired.realigned.bam\n",
      "INFO  13:59:14,517 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:14,519 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  13:59:14,519 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  13:59:14,520 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  13:59:14,520 HelpFormatter - [Tue Oct 24 13:59:14 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  13:59:14,520 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  13:59:14,523 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19689_S6_L001_R2_001_paired.bam -targetIntervals 2010AC19689_S6_L001_R2_001_paired.bam.list -o 2010AC19689_S6_L001_R2_001_paired.realigned.bam \n",
      "INFO  13:59:14,526 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  13:59:14,526 HelpFormatter - Date/Time: 2017/10/24 13:59:14 \n",
      "INFO  13:59:14,526 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:14,527 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  13:59:14,609 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  13:59:14,696 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  13:59:14,701 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  13:59:14,742 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  13:59:14,895 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  13:59:14,898 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  13:59:14,899 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  13:59:14,899 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  13:59:14,899 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  13:59:14,923 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  13:59:14,925 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  13:59:15,419 IndelRealigner - Not attempting realignment in interval One_RAD56448:9-82 because there are too many reads. \n",
      "INFO  13:59:17,640 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-77 because there are too many reads. \n",
      "INFO  13:59:21,470 ProgressMeter -            done    307045.0     6.0 s      21.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  13:59:21,471 ProgressMeter - Total runtime 6.57 secs, 0.11 min, 0.00 hours \n",
      "INFO  13:59:21,474 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 307045 total reads (0.00%) \n",
      "INFO  13:59:21,475 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  13:59:21,475 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19909_S1_L001_R2_001_paired.sam 2> 2010AC19909_S1_L001_R2_001_paired.txt:2010AC19909_S1_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19909_S1_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19909_S1_L001_R2_001_paired.sam OUTPUT=2010AC19909_S1_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 14:01:15 PDT 2017] picard.sam.SortSam INPUT=2010AC19909_S1_L001_R2_001_paired.sam OUTPUT=2010AC19909_S1_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 14:01:15 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 14:01:34\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 14:01:46 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.51 minutes.\n",
      "Runtime.totalMemory()=1696071680\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19909_S1_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19909_S1_L001_R2_001_paired.bam.list\n",
      "INFO  14:01:48,027 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:48,029 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:01:48,029 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:01:48,029 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:01:48,029 HelpFormatter - [Tue Oct 24 14:01:48 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:01:48,030 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:01:48,033 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19909_S1_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19909_S1_L001_R2_001_paired.bam.list \n",
      "INFO  14:01:48,036 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:01:48,036 HelpFormatter - Date/Time: 2017/10/24 14:01:48 \n",
      "INFO  14:01:48,036 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:48,037 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:48,042 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:01:48,188 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  14:01:48,194 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:01:48,251 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.06 \n",
      "INFO  14:01:48,424 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:01:48,449 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:01:48,449 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:01:48,449 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:01:48,450 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  14:01:55,428 ProgressMeter -            done     16100.0     6.0 s       7.2 m       99.5%     6.0 s       0.0 s \n",
      "INFO  14:01:55,429 ProgressMeter - Total runtime 6.98 secs, 0.12 min, 0.00 hours \n",
      "INFO  14:01:55,429 MicroScheduler - 95 reads were filtered out during the traversal out of approximately 1109061 total reads (0.01%) \n",
      "INFO  14:01:55,430 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:01:55,430 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  14:01:55,430 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  14:01:55,430 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  14:01:55,431 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  14:01:55,431 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  14:01:55,431 MicroScheduler -   -> 95 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  14:01:55,432 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  14:01:55,432 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  14:01:55,432 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R2_001_paired.bam -targetIntervals 2010AC19909_S1_L001_R2_001_paired.bam.list -o 2010AC19909_S1_L001_R2_001_paired.realigned.bam\n",
      "INFO  14:01:57,226 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:57,228 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:01:57,228 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:01:57,228 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:01:57,229 HelpFormatter - [Tue Oct 24 14:01:57 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:01:57,229 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:01:57,232 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19909_S1_L001_R2_001_paired.bam -targetIntervals 2010AC19909_S1_L001_R2_001_paired.bam.list -o 2010AC19909_S1_L001_R2_001_paired.realigned.bam \n",
      "INFO  14:01:57,235 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:01:57,236 HelpFormatter - Date/Time: 2017/10/24 14:01:57 \n",
      "INFO  14:01:57,236 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:57,236 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:01:57,318 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:01:57,404 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  14:01:57,410 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:01:57,456 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  14:01:57,607 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:01:57,611 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:01:57,611 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:01:57,611 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:01:57,612 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  14:01:57,636 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  14:01:57,638 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  14:01:59,470 IndelRealigner - Not attempting realignment in interval One_RAD56448:27-59 because there are too many reads. \n",
      "INFO  14:02:07,682 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:21-78 because there are too many reads. \n",
      "INFO  14:02:16,640 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:6-69 because there are too many reads. \n",
      "INFO  14:02:24,887 ProgressMeter -            done   1560253.0    26.0 s      17.0 s      100.0%    26.0 s       0.0 s \n",
      "INFO  14:02:24,888 ProgressMeter - Total runtime 26.86 secs, 0.45 min, 0.01 hours \n",
      "INFO  14:02:24,890 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1560253 total reads (0.00%) \n",
      "INFO  14:02:24,891 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:02:24,891 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19938_S7_L001_R2_001_paired.sam 2> 2010AC19938_S7_L001_R2_001_paired.txt:2010AC19938_S7_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19938_S7_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19938_S7_L001_R2_001_paired.sam OUTPUT=2010AC19938_S7_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 14:03:49 PDT 2017] picard.sam.SortSam INPUT=2010AC19938_S7_L001_R2_001_paired.sam OUTPUT=2010AC19938_S7_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 14:03:49 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 14:04:02\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 14:04:09 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.34 minutes.\n",
      "Runtime.totalMemory()=2017460224\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19938_S7_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19938_S7_L001_R2_001_paired.bam.list\n",
      "INFO  14:04:11,648 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:11,650 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:04:11,650 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:04:11,650 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:04:11,651 HelpFormatter - [Tue Oct 24 14:04:11 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:04:11,651 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:04:11,654 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19938_S7_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19938_S7_L001_R2_001_paired.bam.list \n",
      "INFO  14:04:11,657 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:04:11,657 HelpFormatter - Date/Time: 2017/10/24 14:04:11 \n",
      "INFO  14:04:11,657 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:11,658 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:11,663 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:04:11,815 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  14:04:11,822 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:04:11,862 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  14:04:12,013 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:04:12,039 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:04:12,039 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:04:12,040 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:04:12,040 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  14:04:17,588 ProgressMeter -            done     16100.0     5.0 s       5.7 m       99.5%     5.0 s       0.0 s \n",
      "INFO  14:04:17,588 ProgressMeter - Total runtime 5.55 secs, 0.09 min, 0.00 hours \n",
      "INFO  14:04:17,589 MicroScheduler - 102 reads were filtered out during the traversal out of approximately 817987 total reads (0.01%) \n",
      "INFO  14:04:17,589 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:04:17,589 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  14:04:17,590 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  14:04:17,590 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  14:04:17,590 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  14:04:17,590 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  14:04:17,591 MicroScheduler -   -> 102 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  14:04:17,591 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  14:04:17,591 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  14:04:17,591 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R2_001_paired.bam -targetIntervals 2010AC19938_S7_L001_R2_001_paired.bam.list -o 2010AC19938_S7_L001_R2_001_paired.realigned.bam\n",
      "INFO  14:04:19,414 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:19,416 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:04:19,417 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:04:19,417 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:04:19,417 HelpFormatter - [Tue Oct 24 14:04:19 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:04:19,417 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:04:19,421 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19938_S7_L001_R2_001_paired.bam -targetIntervals 2010AC19938_S7_L001_R2_001_paired.bam.list -o 2010AC19938_S7_L001_R2_001_paired.realigned.bam \n",
      "INFO  14:04:19,425 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:04:19,426 HelpFormatter - Date/Time: 2017/10/24 14:04:19 \n",
      "INFO  14:04:19,426 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:19,426 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:04:19,515 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:04:19,606 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  14:04:19,613 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:04:19,665 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.05 \n",
      "INFO  14:04:19,867 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:04:19,872 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:04:19,873 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:04:19,873 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:04:19,873 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  14:04:19,904 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  14:04:19,905 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  14:04:21,619 IndelRealigner - Not attempting realignment in interval One_RAD56448:18-77 because there are too many reads. \n",
      "INFO  14:04:27,747 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  14:04:44,909 ProgressMeter -            done   1025650.0    24.0 s      23.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  14:04:44,910 ProgressMeter - Total runtime 24.58 secs, 0.41 min, 0.01 hours \n",
      "INFO  14:04:44,913 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1025650 total reads (0.00%) \n",
      "INFO  14:04:44,913 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:04:44,914 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      " -S 2010AC19940_S8_L001_R2_001_paired.sam 2> 2010AC19940_S8_L001_R2_001_paired.txt:2010AC19940_S8_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC19940_S8_L001_R2_001_paired.fastq\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC19940_S8_L001_R2_001_paired.sam OUTPUT=2010AC19940_S8_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 14:05:42 PDT 2017] picard.sam.SortSam INPUT=2010AC19940_S8_L001_R2_001_paired.sam OUTPUT=2010AC19940_S8_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 14:05:42 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 14:05:49\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 14:05:56 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.22 minutes.\n",
      "Runtime.totalMemory()=1475870720\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC19940_S8_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC19940_S8_L001_R2_001_paired.bam.list\n",
      "INFO  14:05:57,940 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:05:57,943 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:05:57,943 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:05:57,943 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:05:57,943 HelpFormatter - [Tue Oct 24 14:05:57 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:05:57,944 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:05:57,947 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC19940_S8_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC19940_S8_L001_R2_001_paired.bam.list \n",
      "INFO  14:05:57,950 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:05:57,950 HelpFormatter - Date/Time: 2017/10/24 14:05:57 \n",
      "INFO  14:05:57,951 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:05:57,951 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:05:57,956 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:05:58,105 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  14:05:58,111 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:05:58,152 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  14:05:58,325 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:05:58,350 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:05:58,351 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:05:58,351 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:05:58,351 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  14:06:02,305 ProgressMeter -            done     16100.0     3.0 s       4.1 m       99.5%     3.0 s       0.0 s \n",
      "INFO  14:06:02,306 ProgressMeter - Total runtime 3.95 secs, 0.07 min, 0.00 hours \n",
      "INFO  14:06:02,306 MicroScheduler - 62 reads were filtered out during the traversal out of approximately 539320 total reads (0.01%) \n",
      "INFO  14:06:02,306 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:06:02,307 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  14:06:02,307 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  14:06:02,307 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  14:06:02,308 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  14:06:02,308 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  14:06:02,308 MicroScheduler -   -> 62 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  14:06:02,308 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  14:06:02,308 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  14:06:02,309 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R2_001_paired.bam -targetIntervals 2010AC19940_S8_L001_R2_001_paired.bam.list -o 2010AC19940_S8_L001_R2_001_paired.realigned.bam\n",
      "INFO  14:06:04,037 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:06:04,039 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:06:04,039 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:06:04,040 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:06:04,040 HelpFormatter - [Tue Oct 24 14:06:04 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:06:04,040 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:06:04,043 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC19940_S8_L001_R2_001_paired.bam -targetIntervals 2010AC19940_S8_L001_R2_001_paired.bam.list -o 2010AC19940_S8_L001_R2_001_paired.realigned.bam \n",
      "INFO  14:06:04,047 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:06:04,047 HelpFormatter - Date/Time: 2017/10/24 14:06:04 \n",
      "INFO  14:06:04,047 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:06:04,048 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:06:04,148 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:06:04,240 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  14:06:04,246 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:06:04,289 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  14:06:04,442 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:06:04,446 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:06:04,446 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:06:04,447 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:06:04,447 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  14:06:04,473 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  14:06:04,474 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  14:06:05,700 IndelRealigner - Not attempting realignment in interval One_RAD56448:23-76 because there are too many reads. \n",
      "INFO  14:06:10,634 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  14:06:20,058 ProgressMeter -            done    701343.0    15.0 s      21.0 s      100.0%    15.0 s       0.0 s \n",
      "INFO  14:06:20,058 ProgressMeter - Total runtime 15.15 secs, 0.25 min, 0.00 hours \n",
      "INFO  14:06:20,061 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 701343 total reads (0.00%) \n",
      "INFO  14:06:20,061 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:06:20,062 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "bowtie2 -x GATKRefTrim --local --rg-id S_2010AC20099_S2_L001_R2_001_paired --rg SM:2010AC20099_S2_L001_R2_001_paired --rg PL:ILLUMINA --rg DS:HiSeq4000 -U 2010AC20099_S2_L001_R2_001_paired.fastq -S 2010AC20099_S2_L001_R2_001_paired.sam 2> 2010AC20099_S2_L001_R2_001_paired.txt\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/Picard/Picard-2.8.2/picard-2.8.2.jar SortSam INPUT=2010AC20099_S2_L001_R2_001_paired.sam OUTPUT=2010AC20099_S2_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true\n",
      "[Tue Oct 24 14:08:03 PDT 2017] picard.sam.SortSam INPUT=2010AC20099_S2_L001_R2_001_paired.sam OUTPUT=2010AC20099_S2_L001_R2_001_paired.bam SORT_ORDER=coordinate VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=true    VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Tue Oct 24 14:08:03 PDT 2017] Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11; Picard version: 2.8.2-SNAPSHOT\n",
      "INFO\t2017-10-24 14:08:19\tSortSam\tFinished reading inputs, merging and writing to output now.\n",
      "[Tue Oct 24 14:08:32 PDT 2017] picard.sam.SortSam done. Elapsed time: 0.49 minutes.\n",
      "Runtime.totalMemory()=2198339584\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T RealignerTargetCreator -I 2010AC20099_S2_L001_R2_001_paired.bam -R 233_Amplicons.fa -o  2010AC20099_S2_L001_R2_001_paired.bam.list\n",
      "INFO  14:08:34,796 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:34,798 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:08:34,798 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:08:34,799 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:08:34,799 HelpFormatter - [Tue Oct 24 14:08:34 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:08:34,799 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:08:34,802 HelpFormatter - Program Args: -T RealignerTargetCreator -I 2010AC20099_S2_L001_R2_001_paired.bam -R 233_Amplicons.fa -o 2010AC20099_S2_L001_R2_001_paired.bam.list \n",
      "INFO  14:08:34,805 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:08:34,805 HelpFormatter - Date/Time: 2017/10/24 14:08:34 \n",
      "INFO  14:08:34,806 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:34,806 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:34,811 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:08:34,964 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  14:08:34,970 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:08:35,011 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  14:08:35,181 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:08:35,207 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:08:35,208 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:08:35,208 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:08:35,208 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  14:08:41,756 ProgressMeter -            done     16100.0     6.0 s       6.8 m       99.5%     6.0 s       0.0 s \n",
      "INFO  14:08:41,757 ProgressMeter - Total runtime 6.55 secs, 0.11 min, 0.00 hours \n",
      "INFO  14:08:41,757 MicroScheduler - 84 reads were filtered out during the traversal out of approximately 980223 total reads (0.01%) \n",
      "INFO  14:08:41,757 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:08:41,758 MicroScheduler -   -> 0 reads (0.00% of total) failing BadMateFilter \n",
      "INFO  14:08:41,758 MicroScheduler -   -> 0 reads (0.00% of total) failing DuplicateReadFilter \n",
      "INFO  14:08:41,758 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  14:08:41,758 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  14:08:41,759 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  14:08:41,759 MicroScheduler -   -> 84 reads (0.01% of total) failing MappingQualityZeroFilter \n",
      "INFO  14:08:41,759 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  14:08:41,759 MicroScheduler -   -> 0 reads (0.00% of total) failing Platform454Filter \n",
      "INFO  14:08:41,760 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n",
      "java -jar /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Programs/GATK/GenomeAnalysisTK-3.7/GenomeAnalysisTK.jar -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R2_001_paired.bam -targetIntervals 2010AC20099_S2_L001_R2_001_paired.bam.list -o 2010AC20099_S2_L001_R2_001_paired.realigned.bam\n",
      "INFO  14:08:43,550 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:43,552 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.7-0-gcfedb67, Compiled 2016/12/12 11:21:18 \n",
      "INFO  14:08:43,552 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:08:43,552 HelpFormatter - For support and documentation go to https://software.broadinstitute.org/gatk \n",
      "INFO  14:08:43,553 HelpFormatter - [Tue Oct 24 14:08:43 PDT 2017] Executing on Linux 4.8.0-59-generic amd64 \n",
      "INFO  14:08:43,553 HelpFormatter - Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 \n",
      "INFO  14:08:43,556 HelpFormatter - Program Args: -T IndelRealigner -R 233_Amplicons.fa -I 2010AC20099_S2_L001_R2_001_paired.bam -targetIntervals 2010AC20099_S2_L001_R2_001_paired.bam.list -o 2010AC20099_S2_L001_R2_001_paired.realigned.bam \n",
      "INFO  14:08:43,559 HelpFormatter - Executing as smay1@ubuntu on Linux 4.8.0-59-generic amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11. \n",
      "INFO  14:08:43,560 HelpFormatter - Date/Time: 2017/10/24 14:08:43 \n",
      "INFO  14:08:43,560 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:43,560 HelpFormatter - -------------------------------------------------------------------------------- \n",
      "INFO  14:08:43,650 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:08:43,748 GenomeAnalysisEngine - Downsampling Settings: No downsampling \n",
      "INFO  14:08:43,754 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  14:08:43,799 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.04 \n",
      "INFO  14:08:43,953 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  14:08:43,957 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:08:43,957 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:08:43,958 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:08:43,958 ProgressMeter -        Location |     reads | elapsed |     reads | completed | runtime |   runtime \n",
      "INFO  14:08:43,985 ReadShardBalancer$1 - Loading BAM index data \n",
      "INFO  14:08:43,986 ReadShardBalancer$1 - Done loading BAM index data \n",
      "INFO  14:08:45,943 IndelRealigner - Not attempting realignment in interval One_RAD56448:11-82 because there are too many reads. \n",
      "INFO  14:08:53,793 IndelRealigner - Not attempting realignment in interval 36611_x1_FemaleLG_9_FemalePos_33.66_MaleLG_NA_MalePos_NA_duplicated_no:22-78 because there are too many reads. \n",
      "INFO  14:09:00,399 IndelRealigner - Not attempting realignment in interval 63267_x1_FemaleLG_23_FemalePos_74.69_MaleLG_23_MalePos_33.76_duplicated_no:6-58 because there are too many reads. \n",
      "INFO  14:09:08,403 ProgressMeter -            done   1434928.0    24.0 s      17.0 s      100.0%    24.0 s       0.0 s \n",
      "INFO  14:09:08,403 ProgressMeter - Total runtime 24.45 secs, 0.41 min, 0.01 hours \n",
      "INFO  14:09:08,406 MicroScheduler - 0 reads were filtered out during the traversal out of approximately 1434928 total reads (0.00%) \n",
      "INFO  14:09:08,406 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  14:09:08,406 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. ------------------------------------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "!perl GATK_pipeline_SM.pl SampleFile.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Moved this output to new directory called GATK_Out"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out\n"
     ]
    }
   ],
   "source": [
    "cd ../Data/GATK_Pipeline_Out/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC20099_S2_L001_R1_001_paired.sam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "still getting a fish-eye network plot"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "generate .sai files by using bwa aln for fwd and reverse trimmed sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 3.41 sec\n",
      "[bwa_aln_core] write to the disk... 0.12 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.42 sec\n",
      "[bwa_aln_core] write to the disk... 0.11 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.44 sec\n",
      "[bwa_aln_core] write to the disk... 0.11 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.38 sec\n",
      "[bwa_aln_core] write to the disk... 0.10 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.35 sec\n",
      "[bwa_aln_core] write to the disk... 0.12 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 1.60 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 1434928 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq\n",
      "[main] Real time: 25.309 sec; CPU: 21.904 sec\n",
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 3.79 sec\n",
      "[bwa_aln_core] write to the disk... 0.12 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.90 sec\n",
      "[bwa_aln_core] write to the disk... 0.12 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.89 sec\n",
      "[bwa_aln_core] write to the disk... 0.09 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.93 sec\n",
      "[bwa_aln_core] write to the disk... 0.10 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.81 sec\n",
      "[bwa_aln_core] write to the disk... 0.11 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 1.79 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 1434928 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq\n",
      "[main] Real time: 27.490 sec; CPU: 24.164 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R1_001_paired.sai\n",
    "\n",
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R2_001_paired.sai"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "have to combine .sai files using bwa sampe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 85)\n",
      "[infer_isize] low and high boundaries: 76 and 95 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 43169 pairs: 83.153 +/- 4.517\n",
      "[infer_isize] skewness: 0.858; kurtosis: 0.447; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 107 (5.28 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.33 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 275 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 3578 out of 17995 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 95 out of 5554 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.18 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.21 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 6.51 sec\n",
      "[bwa_sai2sam_pe_core] 262144 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 86)\n",
      "[infer_isize] low and high boundaries: 76 and 98 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 44216 pairs: 83.675 +/- 5.119\n",
      "[infer_isize] skewness: 0.994; kurtosis: 0.539; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 111 (5.27 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.29 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 316 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 3654 out of 17850 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 89 out of 5410 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.18 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.21 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 6.61 sec\n",
      "[bwa_sai2sam_pe_core] 524288 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 86)\n",
      "[infer_isize] low and high boundaries: 76 and 98 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 44530 pairs: 83.640 +/- 5.121\n",
      "[infer_isize] skewness: 0.991; kurtosis: 0.535; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 111 (5.27 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.32 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 274 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 3845 out of 18081 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 105 out of 5389 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.19 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.21 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 6.75 sec\n",
      "[bwa_sai2sam_pe_core] 786432 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 85)\n",
      "[infer_isize] low and high boundaries: 76 and 95 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 43477 pairs: 83.203 +/- 4.489\n",
      "[infer_isize] skewness: 0.868; kurtosis: 0.495; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 107 (5.28 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.35 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 293 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 4002 out of 19739 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 104 out of 5163 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.19 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.22 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 6.10 sec\n",
      "[bwa_sai2sam_pe_core] 1048576 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 85)\n",
      "[infer_isize] low and high boundaries: 76 and 95 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 43233 pairs: 83.190 +/- 4.508\n",
      "[infer_isize] skewness: 0.850; kurtosis: 0.438; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 107 (5.28 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.44 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 309 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 4081 out of 19394 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 79 out of 5338 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.19 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.22 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 7.27 sec\n",
      "[bwa_sai2sam_pe_core] 1310720 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (80, 83, 86)\n",
      "[infer_isize] low and high boundaries: 76 and 98 for estimating avg and std\n",
      "[infer_isize] inferred external isize from 21004 pairs: 83.681 +/- 5.174\n",
      "[infer_isize] skewness: 0.972; kurtosis: 0.452; ap_prior: 1.00e-05\n",
      "[infer_isize] inferred maximum insert size: 111 (5.27 sigma)\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.15 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 155 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_paired_sw] 1990 out of 8871 Q17 singletons are mated.\n",
      "[bwa_paired_sw] 60 out of 2669 Q17 discordant pairs are fixed.\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.09 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.11 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 3.07 sec\n",
      "[bwa_sai2sam_pe_core] 1434928 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa sampe -r @RG\\tID:Opt1\\tSM:2010AC20099 GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R1_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R2_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq\n",
      "[main] Real time: 129.575 sec; CPU: 44.832 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa sampe -r '@RG\\tID:Opt1\\tSM:2010AC20099' \\\n",
    "GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R1_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_R2_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R1_001_paired.fastq \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC20099_S2_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC20099_S2_L001_PE.sam"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out\n"
     ]
    }
   ],
   "source": [
    "cd /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC20099_S2_L001_PE.sam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "That was a very large (500mb) file (with one locus that over-amplified 350k reads) going to repeat on a medium and small file:\n",
    "Medium: (250mb sam files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 3.55 sec\n",
      "[bwa_aln_core] write to the disk... 0.17 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 3.53 sec\n",
      "[bwa_aln_core] write to the disk... 0.16 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 2.27 sec\n",
      "[bwa_aln_core] write to the disk... 0.09 sec\n",
      "[bwa_aln_core] 694333 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq\n",
      "[main] Real time: 13.589 sec; CPU: 12.204 sec\n",
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 4.35 sec\n",
      "[bwa_aln_core] write to the disk... 0.14 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 4.31 sec\n",
      "[bwa_aln_core] write to the disk... 0.16 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 2.75 sec\n",
      "[bwa_aln_core] write to the disk... 0.09 sec\n",
      "[bwa_aln_core] 694333 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq\n",
      "[main] Real time: 15.572 sec; CPU: 13.548 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R1_001_paired.sai\n",
    "\n",
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R2_001_paired.sai"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (83, 83, 83)\n",
      "[infer_isize] fail to infer insert size: weird pairing\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.46 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 625 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.00 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.26 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 7.75 sec\n",
      "[bwa_sai2sam_pe_core] 262144 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (83, 83, 83)\n",
      "[infer_isize] fail to infer insert size: weird pairing\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.55 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 678 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.00 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.27 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 7.97 sec\n",
      "[bwa_sai2sam_pe_core] 524288 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (83, 83, 83)\n",
      "[infer_isize] fail to infer insert size: weird pairing\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.19 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 436 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.00 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.17 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 4.44 sec\n",
      "[bwa_sai2sam_pe_core] 694333 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa sampe -r @RG\\tID:Opt1\\tSM:2010AC19057 GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R1_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R2_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq\n",
      "[main] Real time: 67.394 sec; CPU: 24.804 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa sampe -r '@RG\\tID:Opt1\\tSM:2010AC19057' \\\n",
    "GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R1_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_R2_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R1_001_paired.fastq \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19057_S16_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19057_S16_L001_PE.sam"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC19057_S16_L001_PE.sam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Small file: (100mb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 3.28 sec\n",
      "[bwa_aln_core] write to the disk... 0.13 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 0.55 sec\n",
      "[bwa_aln_core] write to the disk... 0.02 sec\n",
      "[bwa_aln_core] 307045 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq\n",
      "[main] Real time: 5.033 sec; CPU: 4.340 sec\n",
      "[bwa_aln] 17bp reads: max_diff = 2\n",
      "[bwa_aln] 38bp reads: max_diff = 3\n",
      "[bwa_aln] 64bp reads: max_diff = 4\n",
      "[bwa_aln] 93bp reads: max_diff = 5\n",
      "[bwa_aln] 124bp reads: max_diff = 6\n",
      "[bwa_aln] 157bp reads: max_diff = 7\n",
      "[bwa_aln] 190bp reads: max_diff = 8\n",
      "[bwa_aln] 225bp reads: max_diff = 9\n",
      "[bwa_aln_core] calculate SA coordinate... 3.81 sec\n",
      "[bwa_aln_core] write to the disk... 0.11 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 0.65 sec\n",
      "[bwa_aln_core] write to the disk... 0.02 sec\n",
      "[bwa_aln_core] 307045 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa aln GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq\n",
      "[main] Real time: 6.213 sec; CPU: 5.740 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R1_001_paired.sai\n",
    "\n",
    "!bwa aln GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R2_001_paired.sai"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (83, 83, 83)\n",
      "[infer_isize] fail to infer insert size: weird pairing\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.74 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 438 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.00 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.22 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 7.03 sec\n",
      "[bwa_sai2sam_pe_core] 262144 sequences have been processed.\n",
      "[bwa_sai2sam_pe_core] convert to sequence coordinate... \n",
      "[infer_isize] (25, 50, 75) percentile: (83, 83, 83)\n",
      "[infer_isize] fail to infer insert size: weird pairing\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.05 sec\n",
      "[bwa_sai2sam_pe_core] changing coordinates of 84 alignments.\n",
      "[bwa_sai2sam_pe_core] align unmapped mate...\n",
      "[bwa_sai2sam_pe_core] time elapses: 0.00 sec\n",
      "[bwa_sai2sam_pe_core] refine gapped alignments... 0.04 sec\n",
      "[bwa_sai2sam_pe_core] print alignments... 1.36 sec\n",
      "[bwa_sai2sam_pe_core] 307045 sequences have been processed.\n",
      "[main] Version: 0.7.15-r1140\n",
      "[main] CMD: bwa sampe -r @RG\\tID:Opt1\\tSM:2010AC19057 GATKRefTrim /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R1_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R2_001_paired.sai /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq\n",
      "[main] Real time: 30.142 sec; CPU: 11.644 sec\n"
     ]
    }
   ],
   "source": [
    "!bwa sampe -r '@RG\\tID:Opt1\\tSM:2010AC19057' \\\n",
    "GATKRefTrim \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R1_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_R2_001_paired.sai \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R1_001_paired.fastq \\\n",
    "/mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/Trimmomatic_Out/2010AC19689_S6_L001_R2_001_paired.fastq \\\n",
    "> /mnt/hgfs/Ubuntu_Shared/OptimizationRun1/Data/GATK_Pipeline_Out/2010AC19689_S6_L001_PE.sam"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl getSamMateMismatch.pl 2010AC19689_S6_L001_PE.sam"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
